(function(e){function t(t){for(var n,r,s=t[0],l=t[1],c=t[2],d=0,u=[];d<s.length;d++)r=s[d],Object.prototype.hasOwnProperty.call(o,r)&&o[r]&&u.push(o[r][0]),o[r]=0;for(n in l)Object.prototype.hasOwnProperty.call(l,n)&&(e[n]=l[n]);h&&h(t);while(u.length)u.shift()();return i.push.apply(i,c||[]),a()}function a(){for(var e,t=0;t<i.length;t++){for(var a=i[t],n=!0,r=1;r<a.length;r++){var s=a[r];0!==o[s]&&(n=!1)}n&&(i.splice(t--,1),e=l(l.s=a[0]))}return e}var n={},r={app:0},o={app:0},i=[];function s(e){return l.p+"js/"+({"analyze-phenotypes":"analyze-phenotypes","data-sources":"data-sources",licensing:"licensing","mon-analytics":"mon-analytics",routeerror:"routeerror",search:"search","text-annotate":"text-annotate"}[e]||e)+"."+{"analyze-phenotypes":"057a459b","data-sources":"7762b972",licensing:"0c18b541","mon-analytics":"02893b8c",routeerror:"7e18aa82",search:"347b982c","text-annotate":"b6da5b3b"}[e]+".js"}function l(t){if(n[t])return n[t].exports;var a=n[t]={i:t,l:!1,exports:{}};return e[t].call(a.exports,a,a.exports,l),a.l=!0,a.exports}l.e=function(e){var t=[],a={"analyze-phenotypes":1,"data-sources":1,"mon-analytics":1,routeerror:1,search:1,"text-annotate":1};r[e]?t.push(r[e]):0!==r[e]&&a[e]&&t.push(r[e]=new Promise((function(t,a){for(var n="css/"+({"analyze-phenotypes":"analyze-phenotypes","data-sources":"data-sources",licensing:"licensing","mon-analytics":"mon-analytics",routeerror:"routeerror",search:"search","text-annotate":"text-annotate"}[e]||e)+"."+{"analyze-phenotypes":"5452c421","data-sources":"779e8a1b",licensing:"31d6cfe0","mon-analytics":"09ffa1c4",routeerror:"de150e7a",search:"d1dd7c6c","text-annotate":"e84660d2"}[e]+".css",o=l.p+n,i=document.getElementsByTagName("link"),s=0;s<i.length;s++){var c=i[s],d=c.getAttribute("data-href")||c.getAttribute("href");if("stylesheet"===c.rel&&(d===n||d===o))return t()}var u=document.getElementsByTagName("style");for(s=0;s<u.length;s++){c=u[s],d=c.getAttribute("data-href");if(d===n||d===o)return t()}var h=document.createElement("link");h.rel="stylesheet",h.type="text/css",h.onload=t,h.onerror=function(t){var n=t&&t.target&&t.target.src||o,i=new Error("Loading CSS chunk "+e+" failed.\n("+n+")");i.code="CSS_CHUNK_LOAD_FAILED",i.request=n,delete r[e],h.parentNode.removeChild(h),a(i)},h.href=o;var p=document.getElementsByTagName("head")[0];p.appendChild(h)})).then((function(){r[e]=0})));var n=o[e];if(0!==n)if(n)t.push(n[2]);else{var i=new Promise((function(t,a){n=o[e]=[t,a]}));t.push(n[2]=i);var c,d=document.createElement("script");d.charset="utf-8",d.timeout=120,l.nc&&d.setAttribute("nonce",l.nc),d.src=s(e);var u=new Error;c=function(t){d.onerror=d.onload=null,clearTimeout(h);var a=o[e];if(0!==a){if(a){var n=t&&("load"===t.type?"missing":t.type),r=t&&t.target&&t.target.src;u.message="Loading chunk "+e+" failed.\n("+n+": "+r+")",u.name="ChunkLoadError",u.type=n,u.request=r,a[1](u)}o[e]=void 0}};var h=setTimeout((function(){c({type:"timeout",target:d})}),12e4);d.onerror=d.onload=c,document.head.appendChild(d)}return Promise.all(t)},l.m=e,l.c=n,l.d=function(e,t,a){l.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:a})},l.r=function(e){"undefined"!==typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},l.t=function(e,t){if(1&t&&(e=l(e)),8&t)return e;if(4&t&&"object"===typeof e&&e&&e.__esModule)return e;var a=Object.create(null);if(l.r(a),Object.defineProperty(a,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var n in e)l.d(a,n,function(t){return e[t]}.bind(null,n));return a},l.n=function(e){var t=e&&e.__esModule?function(){return e["default"]}:function(){return e};return l.d(t,"a",t),t},l.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},l.p="/",l.oe=function(e){throw console.error(e),e};var c=window["webpackJsonp"]=window["webpackJsonp"]||[],d=c.push.bind(c);c.push=t,c=c.slice();for(var u=0;u<c.length;u++)t(c[u]);var h=d;i.push([4,"chunk-vendors"]),a()})({0:function(e,t){},"006b":function(e,t,a){},"0150":function(e,t,a){e.exports=a.p+"img/ebi.2c6a5935.png"},"01f7":function(e,t,a){},"034f":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_ORTHOLOG_DISEASE.76865641.png"},"03a1":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_PATHWAYS.123cb200.png"},"04b5":function(e,t,a){"use strict";a("eba9")},"09b1":function(e,t,a){"use strict";a.d(t,"a",(function(){return r})),a.d(t,"e",(function(){return o})),a.d(t,"d",(function(){return s})),a.d(t,"c",(function(){return l})),a.d(t,"b",(function(){return d}));var n={"NCBITaxon:28377":"Anolis carolinensis","NCBITaxon:3702":"Arabidopsis thaliana","NCBITaxon:9913":"Bos taurus","NCBITaxon:9788":"Equidae","NCBITaxon:6237":"Caenorhabditis","NCBITaxon:6239":"Caenorhabditis elegans","NCBITaxon:7955":"Danio rerio","NCBITaxon:7227":"Drosophila melanogaster","NCBITaxon:9796":"Equus caballus","NCBITaxon:9031":"Gallus gallus","NCBITaxon:9606":"Homo sapiens","NCBITaxon:9544":"Macaca mulatta","NCBITaxon:40674":"Mammalia","NCBITaxon:13616":"Monodelphis domestica","NCBITaxon:10090":"Mus musculus","NCBITaxon:10088":"Mus <mouse, genus>","NCBITaxon:39442":"Mus musculus musculus","NCBITaxon:10092":"Mus musculus domesticus","NCBITaxon:10091":"Mus musculus castaneus","NCBITaxon:57486":"Mus musculus molossinus","NCBITaxon:10102":"Mus setulosus","NCBITaxon:477815":"Mus musculus musculus x M. m. domesticus","NCBITaxon:10096":"Mus spretus","NCBITaxon:35531":"Mus musculus bactrianus","NCBITaxon:9258":"Ornithorhynchus anatinus","NCBITaxon:9598":"Pan troglodytes","NCBITaxon:10116":"Rattus norvegicus","NCBITaxon:4932":"Saccharomyces cerevisiae","NCBITaxon:559292":"Saccharomyces cerevisiae S288C","NCBITaxon:9823":"Sus scrofa","NCBITaxon:8364":"Xenopus (Silurana) tropicalis","NCBITaxon:9845":"Ruminantia","NCBITaxon:9615":"Canis lupus familiaris","NCBITaxon:9681":"Felidae","NCBITaxon:9685":"Felis catus","NCBITaxon:33554":"Carnivora","NCBITaxon:9822":"Sus","NCBITaxon:9821":"Suidae","NCBITaxon:9895":"Bovidae","NCBITaxon:33090":"Viridiplantae","NCBITaxon:9783":"Elephas maximus","NCBITaxon:9940":"Ovis aries","NCBITaxon:9545":"Macaca nemestrina","NCBITaxon:10029":"Cricetulus griseus","NCBITaxon:54600":"Macaca nigra","NCBITaxon:9538":"Erythrocebus patas","NCBITaxon:9986":"Oryctolagus cuniculus","NCBITaxon:78454":"Saguinus labiatus","NCBITaxon:9487":"Saguinus fuscicollis","NCBITaxon:9519":"Lagothrix lagotricha","NCBITaxon:9521":"Saimiri sciureus","NCBITaxon:9523":"Plecturocebus moloch","NCBITaxon:9888":"Muntiacus muntjak","OBI:0100026":"Organism"};function r(e){return n[e]}function o(e){var t=Object.entries(n).find((function(t){return t[1]===e}));return t?t[0]:null}var i=["gene","genotype","model","variant","homolog","ortholog-phenotype","ortholog-disease"];function s(e){return i.indexOf(e)>=0}function l(e){var t=["ortholog-phenotype","ortholog-disease","variant"];return t.includes(e)}var c=[6239,7955,7227,9606,10090,10116,559292];function d(e){return c.indexOf(e)>=0}},"09ce":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view"},[a("h2",{staticClass:"page-title"},[e._v(" Citing The Monarch Initiative")]),a("p",[e._v("Please acknowledge the use of Monarch with the following paragraph:")]),a("blockquote",{staticClass:"blockquote"},[a("p",[e._v("{YOUR resource} uses and displays data and algorithms from the Monarch Initiative. The Monarch Initiative ("),a("a",{attrs:{href:"https://monarchinitiative.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://monarchinitiative.org")]),e._v(") makes biomedical knowledge exploration more efficient and effective by providing tools for genotype-phenotype analysis, genomic diagnostics, and precision medicine across broad areas of disease.")])]),a("p",[e._v("To cite the use of Monarch in publications, please use these two references:")]),a("ul",[a("li",[e._v("Navigating the Phenotype Frontier: The Monarch Initiative. McMurry et al. GENETICS August 1, 2016 vol. 203 no. 4 1491-1495; DOI: 10.1534/genetics.116.188870")]),a("li",[e._v("The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species. Shefchek et al. Nucleic Acids Res. 2020 Jan 8; 48(Database issue): D704–D715. Published online 2019 Nov 08. doi: 10.1093/nar/gkz997")])]),a("p",[e._v("To cite a specific data page or result in Monarch:")]),a("blockquote",{staticClass:"blockquote"},[a("p",[e._v("Because most of the association content in Monarch comes from external sources, we kindly ask that you please cite the original database records (e.g. "),a("a",{attrs:{href:"http://www.informatics.jax.org/accession/MGI:1915196",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://www.informatics.jax.org/accession/MGI:1915196")]),e._v(") or the original publications (e.g. PMID:25605924), as appropriate.")])]),a("p",[e._v('If you wish to link from your site to a specific data page from the Monarch Initiative website, or link to algorithmic results as presented in Monarch, please do so using our permalinks (a.k.a. "HTTP URI"). The permalink for each data page starts with the Monarch URL '),a("a",{attrs:{href:"https://monarchinitiative.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://monarchinitiative.org/")]),e._v(" followed by the entity's prefixed accession (e.g. OMIM:154700); in this example, the permalink would be "),a("a",{attrs:{href:"https://monarchinitiative.org/OMIM:154700",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://monarchinitiative.org/OMIM:154700")]),e._v(".")]),a("p",[e._v("For example, the suggested citation for this record from OMIM at Monarch Initiative would be as follows: Prefixed Identifier: OMIM:154700 http URI/permalink: "),a("a",{attrs:{href:"https://monarchinitiative.org/OMIM:154700",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://monarchinitiative.org/OMIM:154700")]),e._v(" Suggested citation: Monarch Consortium. Accessed 2016-09-22. Marfan Syndrome. "),a("a",{attrs:{href:"https://monarchinitiative.org/OMIM:154700",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://monarchinitiative.org/OMIM:154700")])]),a("p",[e._v("The complete set of transformed data for a given source database can be found at "),a("a",{attrs:{href:"https://archive.monarchinitiative.org/latest/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://archive.monarchinitiative.org/latest/")])]),a("p",[e._v("Please note that the data in Monarch is constantly expanding and improving. The data is versioned with each database release, and archived database releases are available upon request. The evolution of individual records in Monarch is not currently tracked. More documentation about our identifiers can be found in GitHub, at "),a("a",{attrs:{href:"https://github.com/monarch-initiative/dipper/blob/master/README.md#identifiers",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/monarch-initiative/dipper/blob/master/README.md#identifiers")])])])])])])}],o={name:"AboutMonarch",components:{}},i=o,s=a("2877"),l=Object(s["a"])(i,n,r,!1,null,null,null);t["default"]=l.exports},"0c1b":function(e,t,a){e.exports=a.p+"img/cross-species-comparison.d9f4e671.png"},"0fc2":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_GENOTYPE.cbe4bbfd.png"},"0fdb":function(e,t,a){},1:function(e,t){},"10e5":function(e,t,a){"use strict";a("7b3b")},1231:function(e,t,a){e.exports=a.p+"img/renci.9996e65d.png"},"16a3":function(e,t,a){"use strict";a("cd8c")},"17ca":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view contact"},[a("h2",{staticClass:"page-title"},[e._v("Contacting The Monarch Initiative")]),a("div",{staticClass:"contact"},[a("p",[e._v(" If you have a specific question about Monarch, its data, usage, software, or services, you can create a ticket on our helpdesk "),a("a",{attrs:{href:"https://github.com/monarch-initiative/helpdesk/issues",target:"__blank"}},[e._v("issue tracker")]),e._v(". We invite you to join the rest of the Monarch Initiative community on "),a("a",{attrs:{href:"https://github.com/monarch-initiative",target:"__blank"}},[e._v("GitHub!")])]),a("p",[e._v(" If you are having trouble using or connecting to any of our public services, please check our "),a("a",{attrs:{href:"https://stats.uptimerobot.com/XPRo9s4BJ5",target:"__blank"}},[e._v("status page")]),e._v(". ")]),a("p",[e._v(" For more information on using or contributing to the Human Phenotype Ontology (HPO), please visit "),a("a",{attrs:{href:"https://hpo.jax.org/",target:"__blank"}},[e._v("https://hpo.jax.org/")]),e._v(". ")]),a("p",[e._v(" To learn more about using or contributing to the Mondo Disease Ontology, please visit "),a("a",{attrs:{href:"https://mondo.monarchinitiative.org/",target:"__blank"}},[e._v("https://mondo.monarchinitiative.org/")]),e._v(" for details. ")]),a("p",[e._v(" Join our mailing list and stay informed! Use our mailing list to follow general developments and discussions about the Monarch Initiative, and to reach out to the community built around Monarch: "),a("br"),a("br"),e._v(" Join the "),a("a",{attrs:{href:"https://groups.google.com/forum/#!forumsearch/Monarch$20Initiative$20Friends"}},[e._v("Monarch Initiative Friends mailing list")]),e._v("."),a("br"),e._v(" Learn more aboout "),a("a",{attrs:{href:"https://support.google.com/groups/answer/1067205?hl=en"}},[e._v(' "Finding and joining a Google group"')]),e._v("."),a("br")]),a("p",[e._v(" For all other general questions or comments, please send us an "),a("a",{attrs:{href:"mailto:info@monarchinitiative.org"}},[e._v("email")]),e._v(". We will do our best to respond as soon as possible. "),a("br"),a("br"),e._v(" Thank you! "),a("br"),a("br"),e._v(" The Monarch Initiative Helpdesk. ")])])])])])])}],o=(a("9efd"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},"1e02":function(e,t,a){},2:function(e,t){},"21bb":function(e,t,a){"use strict";a("2dad")},2368:function(e,t,a){"use strict";a("6aa4")},2849:function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("p",[e._v("Glossary of terms and acronyms we often need to reference; this is a work in progress. For convenience, curation of the living document is not done here but rather "),a("a",{attrs:{href:"https://docs.google.com/spreadsheets/d/1B0f1rE5SAkPfOsE0M22QWTDx-z3_gkLvJn6AyVltco0/edit#gid=1364170751",target:"_blank",rel:"noopener noreferrer"}},[e._v("here")]),e._v(".")]),a("table",[a("thead",[a("tr",[a("th",[e._v("Abbreviation")]),a("th",[e._v("Full name")]),a("th",[e._v("Link")])])]),a("tbody",[a("tr",[a("td",[e._v("AACR")]),a("td",[e._v("American Association for Cancer Research")]),a("td")]),a("tr",[a("td",[e._v("ACMG")]),a("td"),a("td")]),a("tr",[a("td",[e._v("ACT")]),a("td",[e._v("Accrual to Clinical Trials")]),a("td")]),a("tr",[a("td",[e._v("AMIA")]),a("td",[e._v("The American Medical Informatics Association")]),a("td")]),a("tr",[a("td",[e._v("Animal QTLdb")]),a("td"),a("td")]),a("tr",[a("td",[e._v("API")]),a("td"),a("td")]),a("tr",[a("td",[e._v("BD2K")]),a("td",[e._v("Big Data to Knowledge")]),a("td")]),a("tr",[a("td",[e._v("BgeeDB")]),a("td"),a("td",[a("a",{attrs:{href:"http://bgee.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://bgee.org/")])])]),a("tr",[a("td",[e._v("BioCADDIE")]),a("td",[e._v("Biomedical and Healthcare Data Discovery Index Ecosystem")]),a("td")]),a("tr",[a("td",[e._v("BioGRID")]),a("td",[e._v("Biological General Repository for Interaction Datasets.")]),a("td",[a("a",{attrs:{href:"https://thebiogrid.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://thebiogrid.org/")])])]),a("tr",[a("td",[e._v("caDSR")]),a("td",[e._v("Cancer Data Standards Registry and Repository")]),a("td")]),a("tr",[a("td",[e._v("CCH")]),a("td",[e._v("Center for Community Health")]),a("td")]),a("tr",[a("td",[e._v("CCR")]),a("td",[e._v("Center for Clinical Research")]),a("td")]),a("tr",[a("td",[e._v("CDE")]),a("td",[e._v("Common Data Elements")]),a("td")]),a("tr",[a("td",[e._v("CDM")]),a("td",[e._v("Common Data Model")]),a("td")]),a("tr",[a("td",[e._v("CDSI")]),a("td",[e._v("Center for Data Science and Informatics,")]),a("td")]),a("tr",[a("td",[e._v("CECD")]),a("td",[e._v("Center for Education and Career Development,")]),a("td")]),a("tr",[a("td",[e._v("CEDAR")]),a("td",[e._v("Center for Expanded Data Annotation and Retrieval")]),a("td")]),a("tr",[a("td",[e._v("ChEBI")]),a("td"),a("td")]),a("tr",[a("td",[e._v("Chiro")]),a("td",[e._v("ChEBl Integrated Role Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/obophenotype/chiro",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/obophenotype/chiro")])])]),a("tr",[a("td",[e._v("CI")]),a("td",[e._v("Collaborative Innovation")]),a("td")]),a("tr",[a("td",[e._v("CIELO")]),a("td",[e._v("Collaborative Informatics Environment for Learning on healthcare Outcomes")]),a("td")]),a("tr",[a("td",[e._v("CIOs")]),a("td",[e._v("Chief Information Officers")]),a("td")]),a("tr",[a("td",[e._v("CL")]),a("td",[e._v("Cell Ontology")]),a("td",[a("a",{attrs:{href:"http://obofoundry.org/ontology/cl.html",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://obofoundry.org/ontology/cl.html")])])]),a("tr",[a("td",[e._v("ClinVar")]),a("td",[e._v("ClinVar")]),a("td",[a("a",{attrs:{href:"https://www.ncbi.nlm.nih.gov/clinvar/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://www.ncbi.nlm.nih.gov/clinvar/")])])]),a("tr",[a("td",[e._v("CNVs")]),a("td"),a("td")]),a("tr",[a("td",[e._v("CQI")]),a("td",[e._v("Continuous Quality Improvement")]),a("td")]),a("tr",[a("td",[e._v("CRediT")]),a("td",[e._v("Contributor Roles Taxonomy")]),a("td")]),a("tr",[a("td",[e._v("CRO")]),a("td",[e._v("Contributor Role Ontology")]),a("td")]),a("tr",[a("td",[e._v("CRU")]),a("td",[e._v("Clinical Research Unit,")]),a("td")]),a("tr",[a("td",[e._v("CSCW")]),a("td",[e._v("Computer Supported Collaborative Work")]),a("td")]),a("tr",[a("td",[e._v("CSER")]),a("td",[e._v("Clinical Sequencing Exploratory Research")]),a("td")]),a("tr",[a("td",[e._v("CTD")]),a("td",[e._v("Clinical Toxicology Database")]),a("td",[a("a",{attrs:{href:"http://ctdbase.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://ctdbase.org/")])])]),a("tr",[a("td",[e._v("CTS")]),a("td",[e._v("Clinical and Translational Science")]),a("td")]),a("tr",[a("td",[e._v("CTSA")]),a("td",[e._v("Clinical and Translational Science Awards")]),a("td")]),a("tr",[a("td",[e._v("D&I")]),a("td",[e._v("Diversity and Inclusion")]),a("td")]),a("tr",[a("td",[e._v("DREAM")]),a("td",[e._v("Dialogue for Reverse Engineering Assessments and Methods")]),a("td")]),a("tr",[a("td",[e._v("DSD")]),a("td"),a("td")]),a("tr",[a("td",[e._v("DTF")]),a("td",[e._v("Domain Task Force")]),a("td")]),a("tr",[a("td",[e._v("EAC")]),a("td",[e._v("External Advisory Committee")]),a("td")]),a("tr",[a("td",[e._v("EC")]),a("td",[e._v("Entrepreneurship and Commercialization")]),a("td")]),a("tr",[a("td",[e._v("ECO")]),a("td",[e._v("Evidence and Conclusions Ontology")]),a("td",[a("a",{attrs:{href:"http://obofoundry.org/ontology/eco.html",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://obofoundry.org/ontology/eco.html")])])]),a("tr",[a("td",[e._v("ECTO")]),a("td",[e._v("Environmental Conditions, Treatments and Exposures Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/EnvironmentOntology/environmental-exposure-ontology",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/EnvironmentOntology/environmental-exposure-ontology")]),e._v(")")])]),a("tr",[a("td",[e._v("EDM")]),a("td",[e._v("Electronic Data Methods")]),a("td")]),a("tr",[a("td",[e._v("EdX")]),a("td"),a("td")]),a("tr",[a("td",[e._v("EFO")]),a("td",[e._v("Experimental Factor Ontology")]),a("td")]),a("tr",[a("td",[e._v("EGEMS")]),a("td",[e._v("Generating Evidence & Methods to improve patient outcomes")]),a("td")]),a("tr",[a("td",[e._v("EHR")]),a("td",[e._v("electronic health record")]),a("td")]),a("tr",[a("td",[e._v("eMERGE")]),a("td",[e._v("Electronic Medical Records and Genomics Network")]),a("td")]),a("tr",[a("td",[e._v("EMRAM")]),a("td",[e._v("Electronic Medical Record Adoption Model")]),a("td")]),a("tr",[a("td",[e._v("ePHI")]),a("td",[e._v("electronic protected health information")]),a("td")]),a("tr",[a("td",[e._v("ESC")]),a("td",[e._v("External Scientific Consultants")]),a("td")]),a("tr",[a("td",[e._v("ETL")]),a("td",[e._v("Extract, Transform, Load")]),a("td")]),a("tr",[a("td",[e._v("ExAC")]),a("td",[e._v("Exome Aggregation Consortium")]),a("td",[a("a",{attrs:{href:"http://exac.broadinstitute.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://exac.broadinstitute.org/")])])]),a("tr",[a("td",[e._v("FAIR")]),a("td",[e._v("Findable, Accessible, Interoperable, Reusable")]),a("td")]),a("tr",[a("td",[e._v("FAIR-TLC")]),a("td",[e._v("Findable, Accessible, Interoperable, Reusable - Traceable, Licensed, Connected")]),a("td")]),a("tr",[a("td",[e._v("FDA")]),a("td",[e._v("Food and Drug Administration")]),a("td")]),a("tr",[a("td",[e._v("FHIR")]),a("td",[e._v("Fast Healthcare Interoperability Resource")]),a("td")]),a("tr",[a("td",[e._v("FISMA")]),a("td",[e._v("Federal Information Security Management Act")]),a("td")]),a("tr",[a("td",[e._v("FlyBase")]),a("td",[e._v("FlyBase")]),a("td",[a("a",{attrs:{href:"http://flybase.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://flybase.org")])])]),a("tr",[a("td",[e._v("FORCE11")]),a("td",[e._v("The Future of Research Communications and e-Scholarship")]),a("td")]),a("tr",[a("td",[e._v("ForSTI")]),a("td",[e._v("Foresight for Science, Technology and Innovation")]),a("td")]),a("tr",[a("td",[e._v("FSM")]),a("td",[e._v("Feinberg School of Medicine")]),a("td")]),a("tr",[a("td",[e._v("GA4GH")]),a("td",[e._v("Global Alliance for Genomics and Health")]),a("td",[a("a",{attrs:{href:"https://www.ga4gh.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://www.ga4gh.org/")])])]),a("tr",[a("td",[e._v("GARD")]),a("td",[e._v("Genetics and Rare Diseases Information Center")]),a("td")]),a("tr",[a("td",[e._v("GeneNetwork")]),a("td",[e._v("Gene Network")]),a("td",[a("a",{attrs:{href:"http://genenetwork.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://genenetwork.org")])])]),a("tr",[a("td",[e._v("GENIE")]),a("td",[e._v("Genomics, Evidence, Neoplasia, Information, Exchange")]),a("td")]),a("tr",[a("td",[e._v("GENO")]),a("td",[e._v("Genotype Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/monarch-initiative/GENO-ontology/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/monarch-initiative/GENO-ontology/")])])]),a("tr",[a("td",[e._v("GERA")]),a("td",[e._v("Genetic Epidemiology Research on Adult Health and Aging")]),a("td")]),a("tr",[a("td",[e._v("GO")]),a("td",[e._v("Gene Ontology")]),a("td",[a("a",{attrs:{href:"http://geneontology.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://geneontology.org")])])]),a("tr",[a("td",[e._v("GRID")]),a("td",[e._v("Global Research Identifier Database")]),a("td")]),a("tr",[a("td",[e._v("GWAS Catalog")]),a("td",[e._v("genome-wide association studies")]),a("td",[a("a",{attrs:{href:"https://www.ebi.ac.uk/gwas/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://www.ebi.ac.uk/gwas/")])])]),a("tr",[a("td",[e._v("HCLS")]),a("td"),a("td",[a("a",{attrs:{href:"https://www.w3.org/TR/2015/NOTE-hcls-dataset-20150514/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://www.w3.org/TR/2015/NOTE-hcls-dataset-20150514/")])])]),a("tr",[a("td",[e._v("HGNC")]),a("td"),a("td")]),a("tr",[a("td",[e._v("HGVS")]),a("td"),a("td")]),a("tr",[a("td",[e._v("HIMSS")]),a("td",[e._v("Healthcare Information and Management Systems Society")]),a("td")]),a("tr",[a("td",[e._v("HIPAA")]),a("td",[e._v("Health Insurance Portability and Accountability Act")]),a("td")]),a("tr",[a("td",[e._v("HIPPO")]),a("td",[e._v("(Does it stand for anything?)")]),a("td",[a("a",{attrs:{href:"http://hippo.monarchinitiative.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("hippo.monarchinitiative.org")])])]),a("tr",[a("td",[e._v("HL7")]),a("td",[e._v("Health Level Seven International")]),a("td")]),a("tr",[a("td",[e._v("HPO")]),a("td",[e._v("Human Phenotype Ontology")]),a("td",[a("a",{attrs:{href:"http://human-phenotype-ontology.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://human-phenotype-ontology.org/")])])]),a("tr",[a("td",[e._v("I2b2")]),a("td",[e._v("Informatics for Integrating Biology & the Bedside")]),a("td")]),a("tr",[a("td",[e._v("I2I")]),a("td",[e._v("Idea to Implementation")]),a("td")]),a("tr",[a("td",[e._v("ICBO")]),a("td"),a("td")]),a("tr",[a("td",[e._v("ICD11")]),a("td",[e._v("International Classification of Diseases, Eleventh Revision")]),a("td")]),a("tr",[a("td",[e._v("IEEE")]),a("td",[e._v("Institute of Electrical and Electronics Engineers")]),a("td")]),a("tr",[a("td",[e._v("IMPC")]),a("td"),a("td")]),a("tr",[a("td",[e._v("IP")]),a("td",[e._v("Intellectual Property")]),a("td")]),a("tr",[a("td",[e._v("IRB")]),a("td",[e._v("Institutional Review Board")]),a("td")]),a("tr",[a("td",[e._v("IT")]),a("td",[e._v("Information Technology")]),a("td")]),a("tr",[a("td",[e._v("JSON")]),a("td"),a("td")]),a("tr",[a("td",[e._v("JSON-LD")]),a("td",[e._v("JavaScript Object Notation for Linked Data")]),a("td")]),a("tr",[a("td",[e._v("KEGG")]),a("td",[e._v("Kyoto Encyclopedia of Genes and Genomes")]),a("td",[a("a",{attrs:{href:"http://www.kegg.jp/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://www.kegg.jp/")])])]),a("tr",[a("td",[e._v("KL2")]),a("td",[e._v("Transition to Independence")]),a("td")]),a("tr",[a("td",[e._v("LCH")]),a("td",[e._v("Ann and Robert H. Lurie Children’s Hospital of Chicago")]),a("td")]),a("tr",[a("td",[e._v("LoS")]),a("td",[e._v("Letter of Support")]),a("td")]),a("tr",[a("td",[e._v("MAxO")]),a("td",[e._v("Medical Action Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/monarch-initiative/MAxO",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/monarch-initiative/MAxO")])])]),a("tr",[a("td",[e._v("MESA")]),a("td"),a("td")]),a("tr",[a("td",[e._v("MGI")]),a("td",[e._v("Mouse Genome Informatics")]),a("td",[a("a",{attrs:{href:"http://www.informatics.jax.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://www.informatics.jax.org/")])])]),a("tr",[a("td",[e._v("MIM")]),a("td",[e._v("Mendelian Inheritance in Man")]),a("td")]),a("tr",[a("td",[e._v("MME")]),a("td",[e._v("Matchmaker Exchange")]),a("td")]),a("tr",[a("td",[e._v("MMRRC")]),a("td",[e._v("Mutant Mouse Resource and Research Centers")]),a("td")]),a("tr",[a("td",[e._v("MonDO")]),a("td",[e._v("Monarch Merged Disease Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/monarch-initiative/monarch-disease-ontology/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/monarch-initiative/monarch-disease-ontology/")])])]),a("tr",[a("td",[e._v("MP")]),a("td",[e._v("Mammalian Phenotype Ontology")]),a("td",[a("a",{attrs:{href:"http://obofoundry.org/ontology/mp.html",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://obofoundry.org/ontology/mp.html")])])]),a("tr",[a("td",[e._v("MPD")]),a("td",[e._v("Mouse phenome database")]),a("td",[a("a",{attrs:{href:"http://phenome.jax.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://phenome.jax.org/")])])]),a("tr",[a("td",[e._v("MyGene")]),a("td",[e._v("MyGene")]),a("td",[a("a",{attrs:{href:"http://mygene.info",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://mygene.info")])])]),a("tr",[a("td",[e._v("NASA/JPL")]),a("td",[e._v("National Aeronautics and Space Administration/Jet Propulsion Laboratory")]),a("td")]),a("tr",[a("td",[e._v("NBO")]),a("td"),a("td")]),a("tr",[a("td",[e._v("NCATS")]),a("td"),a("td")]),a("tr",[a("td",[e._v("NCIt")]),a("td"),a("td")]),a("tr",[a("td",[e._v("Neo4J")]),a("td"),a("td")]),a("tr",[a("td",[e._v("NGS")]),a("td"),a("td")]),a("tr",[a("td",[e._v("NHLBI")]),a("td",[e._v("National Heart Lung and Blood Institute")]),a("td")]),a("tr",[a("td",[e._v("NIH")]),a("td",[e._v("National Institutes of Health")]),a("td")]),a("tr",[a("td",[e._v("NITRO")]),a("td",[e._v("Northwestern Integrated Translational Research Optimizer")]),a("td")]),a("tr",[a("td",[e._v("NLM")]),a("td",[e._v("National Library of Medicine")]),a("td")]),a("tr",[a("td",[e._v("NLP")]),a("td",[e._v("Natural Language Processing")]),a("td")]),a("tr",[a("td",[e._v("NMHC")]),a("td",[e._v("Northwestern Memorial HealthCare")]),a("td")]),a("tr",[a("td",[e._v("NSIGHT")]),a("td",[e._v("Newborn Sequencing in Genomic Medicine and Public Health")]),a("td")]),a("tr",[a("td",[e._v("NU")]),a("td",[e._v("Northwestern University")]),a("td")]),a("tr",[a("td",[e._v("NUCATS")]),a("td",[e._v("Northwestern University Clinical and Translational Sciences Institute")]),a("td")]),a("tr",[a("td",[e._v("OBO")]),a("td"),a("td")]),a("tr",[a("td",[e._v("ODK")]),a("td",[e._v("Ontology Development Kit")]),a("td")]),a("tr",[a("td",[e._v("OHDSI")]),a("td",[e._v("Observational Health Data Sciences and Informatics")]),a("td")]),a("tr",[a("td",[e._v("OMIA")]),a("td",[e._v("Online Mendelian Inheritance in Animals")]),a("td",[a("a",{attrs:{href:"http://omia.angis.org.au/home/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://omia.angis.org.au/home/")])])]),a("tr",[a("td",[e._v("OMIM")]),a("td",[e._v("Online Mendelian Inheritance in Man")]),a("td",[a("a",{attrs:{href:"http://omim.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://omim.org")])])]),a("tr",[a("td",[e._v("ONC")]),a("td",[e._v("Office of the National Coordinator")]),a("td")]),a("tr",[a("td",[e._v("ORCiD")]),a("td",[e._v("Open Researcher and Contributor ID")]),a("td")]),a("tr",[a("td",[e._v("Orphanet")]),a("td",[e._v("Portal for rare diseases and orphan drugs")]),a("td",[a("a",{attrs:{href:"http://www.orpha.net",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://www.orpha.net")])])]),a("tr",[a("td",[e._v("OSI")]),a("td",[e._v("Open Source Initiative")]),a("td")]),a("tr",[a("td",[e._v("OWL")]),a("td",[e._v("Web Ontology Language")]),a("td")]),a("tr",[a("td",[e._v("Panther")]),a("td",[e._v("PantherDB")]),a("td",[a("a",{attrs:{href:"http://pantherdb.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://pantherdb.org")])])]),a("tr",[a("td",[e._v("PATO")]),a("td",[e._v("Phenotype And Trait Ontology")]),a("td")]),a("tr",[a("td",[e._v("PCORI")]),a("td",[e._v("Patient-Centered Outcomes Research Institute")]),a("td")]),a("tr",[a("td",[e._v("PCORnet")]),a("td",[e._v("National Patient-Centered Clinical Research Network")]),a("td")]),a("tr",[a("td",[e._v("PDs")]),a("td",[e._v("Program Directors")]),a("td")]),a("tr",[a("td",[e._v("PhenX")]),a("td",[e._v("Phenotypes and eXposures")]),a("td")]),a("tr",[a("td",[e._v("PHI")]),a("td",[e._v("Protected Health Information")]),a("td")]),a("tr",[a("td",[e._v("PI")]),a("td",[e._v("Principal Investigator")]),a("td")]),a("tr",[a("td",[e._v("POTATO")]),a("td",[e._v("Phenotype Ontologies: Traversing All The Organisms")]),a("td")]),a("tr",[a("td",[e._v("PROMIS")]),a("td",[e._v("Patient-Reported Outcomes Measurement Information System")]),a("td")]),a("tr",[a("td",[e._v("QA")]),a("td",[e._v("Quality assurance")]),a("td")]),a("tr",[a("td",[e._v("QC")]),a("td",[e._v("Quality control")]),a("td")]),a("tr",[a("td",[e._v("QTLs")]),a("td"),a("td")]),a("tr",[a("td",[e._v("RBA")]),a("td",[e._v("Results-Based Accountability")]),a("td")]),a("tr",[a("td",[e._v("RCS")]),a("td",[e._v("Revision Control System")]),a("td")]),a("tr",[a("td",[e._v("RDF")]),a("td",[e._v("Resource Description Framework")]),a("td")]),a("tr",[a("td",[e._v("REDCap")]),a("td",[e._v("Research Electronic Data Capture")]),a("td")]),a("tr",[a("td",[e._v("RFA")]),a("td",[e._v("Request for Applications")]),a("td")]),a("tr",[a("td",[e._v("RGD")]),a("td",[e._v("Rat Genome Database")]),a("td")]),a("tr",[a("td",[e._v("RIC")]),a("td",[e._v("Recruitment Innovation Center")]),a("td")]),a("tr",[a("td",[e._v("RO")]),a("td",[e._v("Relation Ontology")]),a("td",[a("a",{attrs:{href:"http://obofoundry.org/ontology/ro.html",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://obofoundry.org/ontology/ro.html")])])]),a("tr",[a("td",[e._v("ROO")]),a("td",[e._v("Research Outputs Ontology")]),a("td")]),a("tr",[a("td",[e._v("SCCS")]),a("td",[e._v("Source Code Control System")]),a("td")]),a("tr",[a("td",[e._v("SciENcv")]),a("td",[e._v("Science Experts Network Curriculum Vitae")]),a("td")]),a("tr",[a("td",[e._v("SciTS")]),a("td",[e._v("Science of Team Science")]),a("td")]),a("tr",[a("td",[e._v("SDK")]),a("td",[e._v("Software Development Kit")]),a("td")]),a("tr",[a("td",[e._v("SEPIO")]),a("td",[e._v("Scientific Evidence and Provenance Information Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/monarch-initiative/SEPIO-ontology/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/monarch-initiative/SEPIO-ontology/")])])]),a("tr",[a("td",[e._v("SGD")]),a("td",[e._v("Saccharomyces Genome Database")]),a("td")]),a("tr",[a("td",[e._v("SHARPn")]),a("td",[e._v("Strategic Health IT Advanced Research Projects")]),a("td")]),a("tr",[a("td",[e._v("SIO")]),a("td"),a("td")]),a("tr",[a("td",[e._v("SMC-DNA")]),a("td",[e._v("Somatic Mutation Calling-Deoxyribonucleic Acid")]),a("td")]),a("tr",[a("td",[e._v("SNOMED-CT")]),a("td",[e._v("Systematized Nomenclature of Medicine -- Clinical Terms")]),a("td")]),a("tr",[a("td",[e._v("SNPs")]),a("td"),a("td")]),a("tr",[a("td",[e._v("SNVs")]),a("td"),a("td")]),a("tr",[a("td",[e._v("SO")]),a("td",[e._v("Sequence Ontology")]),a("td",[a("a",{attrs:{href:"http://www.sequenceontology.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://www.sequenceontology.org/")])])]),a("tr",[a("td",[e._v("Solr")]),a("td"),a("td")]),a("tr",[a("td",[e._v("SOLVE-RD")]),a("td"),a("td")]),a("tr",[a("td",[e._v("SPARQL")]),a("td",[e._v("Protocol and RDF Query Language")]),a("td")]),a("tr",[a("td",[e._v("STRING")]),a("td"),a("td")]),a("tr",[a("td",[e._v("TIC")]),a("td",[e._v("Trial Innovation Center")]),a("td")]),a("tr",[a("td",[e._v("TIN")]),a("td",[e._v("Trials Innovation Network")]),a("td")]),a("tr",[a("td",[e._v("TNF")]),a("td",[e._v("Tumor Necrosis Factor")]),a("td")]),a("tr",[a("td",[e._v("TOADS")]),a("td",[e._v("Tools Ontologies, Algorithms, Data, and Standards")]),a("td")]),a("tr",[a("td",[e._v("TSB")]),a("td",[e._v("Translational Science Base")]),a("td")]),a("tr",[a("td",[e._v("TSC")]),a("td",[e._v("Team Science and Collaboration")]),a("td")]),a("tr",[a("td",[e._v("TSV")]),a("td"),a("td")]),a("tr",[a("td",[e._v("Uberon")]),a("td",[e._v("Uber-anatomy ontology")]),a("td",[a("a",{attrs:{href:"http://uberon.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://uberon.org")])])]),a("tr",[a("td",[e._v("UCSF")]),a("td",[e._v("University of California San Francisco")]),a("td")]),a("tr",[a("td",[e._v("UDN")]),a("td",[e._v("Undiagnosed Diseases Network")]),a("td")]),a("tr",[a("td",[e._v("UMLS")]),a("td"),a("td")]),a("tr",[a("td",[e._v("uPheno")]),a("td",[e._v("Unified Phenotype Ontology")]),a("td",[a("a",{attrs:{href:"https://github.com/obophenotype/upheno/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://github.com/obophenotype/upheno/")])])]),a("tr",[a("td",[e._v("UW")]),a("td",[e._v("University of Washington")]),a("td")]),a("tr",[a("td",[e._v("VICC")]),a("td"),a("td",[a("a",{attrs:{href:"https://cancervariants.org/",target:"_blank",rel:"noopener noreferrer"}},[e._v("https://cancervariants.org/")])])]),a("tr",[a("td",[e._v("WormBase")]),a("td",[e._v("WormBase")]),a("td",[a("a",{attrs:{href:"http://wormbase.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://wormbase.org")])])]),a("tr",[a("td",[e._v("XPO")]),a("td",[e._v("Xenopus Phenotype Ontology")]),a("td")]),a("tr",[a("td",[e._v("ZFIN")]),a("td",[e._v("Zebrafish Information Resource")]),a("td",[a("a",{attrs:{href:"http://zfin.org",target:"_blank",rel:"noopener noreferrer"}},[e._v("http://zfin.org")])])])])])])])])}],o=a("2877"),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},"2d78":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view publications"},[a("div",{staticClass:"content"},[a("h2",{staticClass:"text-center"},[e._v("Selected Monarch Publications")]),a("div",{staticClass:"card"},[a("h3",[e._v("2021")]),a("div",{staticClass:"text"},[a("p",[e._v(" Hannestad LM, Dančík V, Godden M, Suen IW, Huellas-Bruskiewicz KC, Good BM, Mungall CJ, Bruskiewich RM. (2021) "),a("b",{staticClass:"title"},[e._v("Knowledge Beacons: Web services for data harvesting of distributed biomedical knowledge")]),e._v(". "),a("i",[e._v("PLoS One.")]),e._v(" "),a("b",[e._v("16(3)")]),e._v(":e0231916. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33755673/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Köhler S, Gargano M, Matentzoglu N, Carmody LC, Lewis-Smith D, Vasilevsky NA, Danis D, Balagura G, Baynam G, Brower AM, Callahan TJ, Chute CG, Est JL, Galer PD, Ganesan S, Griese M, Haimel M, Pazmandi J, Hanauer M, Harris NL, Hartnett MJ, Hastreiter M, Hauck F, He Y, Jeske T, Kearney H, Kindle G, Klein C, Knoflach K, Krause R, Lagorce D, McMurry JA, Miller JA, Munoz-Torres MC, Peters RL, Rapp CK, Rath AM, Rind SA, Rosenberg AZ, Segal MM, Seidel MG, Smedley D, Talmy T, Thomas Y, Wiafe SA, Xian J, Yüksel Z, Helbig I, Mungall CJ, Haendel MA, Robinson PN. (2021) "),a("b",{staticClass:"title"},[e._v("The Human Phenotype Ontology in 2021")]),e._v(". "),a("i",[e._v("Nucleic Acids Research")]),e._v(" "),a("b",[e._v("49(D1)")]),e._v(":D1207-D1217. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33264411/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Luo L, Yan S, Lai PT, Veltri D, Oler A, Xirasagar S, Ghosh R, Similuk M, Robinson PN, Lu Z. (2021) "),a("b",{staticClass:"title"},[e._v("PhenoTagger: A Hybrid Method for Phenotype Concept Recognition using Human Phenotype Ontology")]),a("i",[e._v("Bioinformatics.")]),e._v(" btab019. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33471061/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Reese JT, Unni D, Callahan TJ, Cappelletti L, Ravanmehr V, Carbon S, Shefchek KA, Good BM, Balhoff JP, Fontana T, Blau H, Matentzoglu N, Harris NL, Munoz-Torres MC, Haendel MA, Robinson PN, Joachimiak MP, Mungall CJ. (2021) "),a("b",{staticClass:"title"},[e._v("KG-COVID-19: A Framework to Produce Customized Knowledge Graphs for COVID-19 Response")]),e._v(". "),a("i",[e._v("Patterns (N Y)")]),e._v(" "),a("b",[e._v("2(1)")]),e._v(":100155. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33196056/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Vasilevsky NA, Hosseini M, Teplitzky S, Ilik V, Mohammadi E, Schneider J, Kern B, Colomb J, Edmunds SC, Gutzman K, Himmelstein DS, White M, Smith B, O'Keefe L, Haendel M, Holmes KL. (2021) "),a("b",{staticClass:"title"},[e._v("Is authorship sufficient for today's collaborative research? A call for contributor roles")]),e._v(". "),a("i",[e._v("Account Res.")]),e._v(" "),a("b",[e._v("28(1)")]),e._v(":23-43. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32602379/"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2020")]),a("div",{staticClass:"text"},[a("p",[e._v(" Carmody LC, Blau H, Danis D, Zhang XA, Gourdine JP, Vasilevsky N, Krawitz P, Thompson MD, Robinson PN. (2020) "),a("b",{staticClass:"title"},[e._v("Significantly different clinical phenotypes associated with mutations in synthesis and transamidase+remodeling glycosylphosphatidylinositol (GPI)-anchor biosynthesis genes")]),e._v(". "),a("i",[e._v("Orphanet J Rare Dis.")]),e._v(" "),a("b",[e._v("15(1)")]),e._v(":40. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32019583/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Chang WH, Mashouri P, Lozano AX, Johnstone B, Husić M, Olry A, Maiella S, Balci TB, Sawyer SL, Robinson PN, Rath A, Brudno M. (2020) "),a("b",{staticClass:"title"},[e._v("Phenotate: crowdsourcing phenotype annotations as exercises in undergraduate classes")]),e._v(". "),a("i",[e._v("Genet Med.")]),e._v(" "),a("b",[e._v("22(8)")]),e._v(":1391-1400. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32366968/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"title"},[a("p",[e._v(" Cipriani V, Pontikos N, Arno G, Sergouniotis PI, Lenassi E, Thawong P, Danis D, Michaelides M, Webster AR, Moore AT, Robinson PN, Jacobsen JOB, Smedley D. (2020) "),a("b",{staticClass:"title"},[e._v("An Improved Phenotype-Driven Tool for Rare Mendelian Variant Prioritization: Benchmarking Exomiser on Real Patient Whole-Exome Data")]),a("i",[e._v("Genes (Basel).")]),e._v(" "),a("b",[e._v("11(4)")]),e._v(":460. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32340307/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Duncan WD, Thyvalikakath T, Haendel M, Torniai C, Hernandez P, Song M, Acharya A, Caplan DJ, Schleyer T, Ruttenberg A. (2020) "),a("b",{staticClass:"title"},[e._v("Structuring, reuse and analysis of electronic dental data using the Oral Health and Disease Ontology")]),e._v(". "),a("i",[e._v("J Biomed Semantics.")]),e._v(" "),a("b",[e._v("11(1)")]),e._v(":8. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32819435/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Gasteiger LM, Robinson PN, Pazmandi J, Boztug K, Seppänen MRJ, Seidel MG; Registry Working Party of the European Society for Immunodeficiencies (ESID). (2020) "),a("b",{staticClass:"title"},[e._v("Supplementation of the ESID registry working definitions for the clinical diagnosis of inborn errors of immunity with encoded human phenotype ontology (HPO) terms")]),e._v(". "),a("i",[e._v("J Allergy Clin Immunol Pract.")]),e._v(" "),a("b",[e._v("8(5)")]),e._v(":1778. "),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32389282/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Haendel M, Vasilevsky N, Unni D, Bologa C, Harris N, Rehm H, Hamosh A, Baynam G, Groza T, McMurry J, Dawkins H, Rath A, Thaxon C, Bocci G, Joachimiak MP, Köhler S, Robinson PN, Mungall C, Oprea TI. (2020) "),a("b",{staticClass:"title"},[e._v("How many rare diseases are there?")]),a("i",[e._v("Nat Rev Drug Discov.")]),e._v(" "),a("b",[e._v("19(2)")]),e._v(":77-78. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32020066/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Karlebach G, Hansen P, Veiga DF, Steinhaus R, Danis D, Li S, Anczukow O, Robinson PN. (2020) "),a("b",{staticClass:"title"},[e._v("HBA-DEALS: accurate and simultaneous identification of differential expression and splicing using hierarchical Bayesian analysis")]),e._v(". "),a("i",[e._v("Genome Biol.")]),e._v(" "),a("b",[e._v("21(1)")]),e._v(":171. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32660516/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Ong E, Wang LL, Schaub J, O'Toole JF, Steck B, Rosenberg AZ, Dowd F, Hansen J, Barisoni L, Jain S, de Boer IH, Valerius MT, Waikar SS, Park C, Crawford DC, Alexandrov T, Anderton CR, Stoeckert C, Weng C, Diehl AD, Mungall CJ, Haendel M, Robinson PN, Himmelfarb J, Iyengar R, Kretzler M, Mooney S, He Y. (2020) "),a("b",{staticClass:"title"},[e._v("Modelling kidney disease using ontology: insights from the Kidney Precision Medicine Project")]),e._v(". "),a("i",[e._v("Nat Rev Nephrol.")]),e._v(" "),a("b",[e._v("16(11)")]),e._v(":686-696. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32939051/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Petrini A, Mesiti M, Schubach M, Frasca M, Danis D, Re M, Grossi G, Cappelletti L, Castrignanò T, Robinson PN, Valentini G. (2020) "),a("b",{staticClass:"title"},[e._v("parSMURF, a high-performance computing tool for the genome-wide detection of pathogenic variants")]),e._v(". "),a("i",[e._v("Gigascience.")]),e._v(" "),a("b",[e._v("9(5)")]),e._v(":giaa052. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32444882/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Robinson PN, Haendel MA. (2020) "),a("b",{staticClass:"title"},[e._v("Ontologies, Knowledge Representation, and Machine Learning for Translational Research: Recent Contributions")]),e._v(". "),a("i",[e._v("Yearb Med Inform.")]),e._v(" "),a("b",[e._v("29(1)")]),e._v(":159-162. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32823310/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Robinson PN, Ravanmehr V, Jacobsen JOB, Danis D, Zhang XA, Carmody LC, Gargano MA, Thaxton CL, Karlebach G, Reese J, Holtgrewe M, Köhler S, McMurry JA, Haendel MA, Smedley D. (2020) "),a("b",{staticClass:"title"},[e._v("Interpretable Clinical Genomics with a Likelihood Ratio Paradigm")]),e._v(". "),a("i",[e._v("Am J Hum Genet.")]),e._v(" "),a("b",[e._v("107(3)")]),e._v(":403-417. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32755546/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Rubinstein YR, Robinson PN, Gahl WA, Avillach P, Baynam G, Cederroth H, Goodwin RM, Groft SC, Hansson MG, Harris NL, Huser V, Mascalzoni D, McMurry JA, Might M, Nellaker C, Mons B, Paltoo DN, Pevsner J, Posada M, Rockett-Frase AP, Roos M, Rubinstein TB, Taruscio D, van Enckevort E, Haendel MA. (2020) "),a("b",{staticClass:"title"},[e._v("The case for open science: rare diseases")]),e._v(". "),a("i",[e._v("JAMIA Open.")]),e._v(" "),a("b",[e._v("3(3)")]),e._v(":472-486. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33426479/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Shefchek KA, Harris NL, Gargano M, Matentzoglu N, Unni D, Brush M, Keith D, Conlin T, Vasilevsky N, Zhang XA, Balhoff JP, Babb L, Bello SM, Blau H, Bradford Y, Carbon S, Carmody L, Chan LE, Cipriani V, Cuzick A, Della Rocca M, Dunn N, Essaid S, Fey P, Grove C, Gourdine JP, Hamosh A, Harris M, Helbig I, Hoatlin M, Joachimiak M, Jupp S, Lett KB, Lewis SE, McNamara C, Pendlington ZM, Pilgrim C, Putman T, Ravanmehr V, Reese J, Riggs E, Robb S, Roncaglia P, Seager J, Segerdell E, Similuk M, Storm AL, Thaxon C, Thessen A, Jacobsen JOB, McMurry JA, Groza T, Köhler S, Smedley D, Robinson PN, Mungall CJ, Haendel MA, Munoz-Torres MC, Osumi-Sutherland D. (2020) "),a("b",{staticClass:"title"},[e._v("The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species")]),e._v(". "),a("i",[e._v("Nucleic Acids Res.")]),e._v(" "),a("b",[e._v("48(D1)")]),e._v(":D704-D715. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31701156"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Tcheandjieu C, Aguirre M, Gustafsson S, Saha P, Potiny P, Haendel M, Ingelsson E, Rivas MA, Priest JR. (2020) "),a("b",{staticClass:"title"},[e._v("A phenome-wide association study of 26 mendelian genes reveals phenotypic expressivity of common and rare variants within the general population")]),e._v(". "),a("i",[e._v("PLoS Genet")]),e._v(". "),a("b",[e._v("16(11)")]),e._v(":e1008802. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33226994/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Thessen AE, Walls RL, Vogt L, Singer J, Warren R, Buttigieg PL, Balhoff JP, Mungall CJ, McGuinness DL, Stucky BJ, Yoder MJ, Haendel MA. (2020) "),a("b",{staticClass:"title"},[e._v("Transforming the study of organisms: Phenomic data models and knowledge bases")]),e._v(". "),a("i",[e._v("PLoS Comput Biol.")]),e._v(" "),a("b",[e._v("16(11)")]),e._v(":e1008376. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/33232313/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Wagner AH, Walsh B, Mayfield G, Tamborero D, Sonkin D, Krysiak K, Deu-Pons J, Duren RP, Gao J, McMurry J, Patterson S, Del Vecchio Fitz C, Pitel BA, Sezerman OU, Ellrott K, Warner JL, Rieke DT, Aittokallio T, Cerami E, Ritter DI, Schriml LM, Freimuth RR, Haendel M, Raca G, Madhavan S, Baudis M, Beckmann JS, Dienstmann R, Chakravarty D, Li XS, Mockus S, Elemento O, Schultz N, Lopez-Bigas N, Lawler M, Goecks J, Griffith M, Griffith OL, Margolin AA. (2020) "),a("b",{staticClass:"title"},[e._v("A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer")]),e._v(". "),a("i",[e._v("Nat Genet.")]),e._v(" "),a("b",[e._v("52(4)")]),e._v(":448-457. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/32246132/"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2019")]),a("div",{staticClass:"text"},[a("p",[e._v(" Ahalt SC, Chute CG, Fecho K, Glusman G, Hadlock J, Taylor CO, Pfaff ER, Robinson PN, Solbrig H, Ta C, Tatonetti N, Weng C; Biomedical Data Translator Consortium. (2019) "),a("b",{staticClass:"title"},[e._v("Clinical Data: Sources and Types, Regulatory Constraints, Applications")]),e._v(". "),a("i",[e._v("Clin Transl Sci.")]),e._v(" "),a("b",[e._v("12(4)")]),e._v(":329-333. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/31074176/"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Altinbas L, Bormann N, Lehmann D, Jeuthe S, Wulsten D, Kornak U, Robinson PN, Wildemann B, Kararigas G. (2019) "),a("b",{staticClass:"title"},[e._v("Assessment of Bones Deficient in Fibrillin-1 Microfibrils Reveals Pronounced Sex Differences")]),e._v(". "),a("i",[e._v("Int J Mol Sci.")]),e._v(" "),a("b",[e._v("20(23)")]),e._v(":6059. ["),a("a",{attrs:{target:"__blank",href:"https://pubmed.ncbi.nlm.nih.gov/31805661"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Cacheiro P, Haendel MA, Smedley D; International Mouse Phenotyping Consortium and the Monarch Initiative (2019) "),a("b",{staticClass:"title"},[e._v("New models for human disease from the International Mouse Phenotyping Consortium")]),e._v(". "),a("i",[e._v("Mamm Genome ")]),e._v(" "),a("b",[e._v("30(5-6)")]),e._v(":143-150. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31127358"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Carbon S, Champieux R, McMurry JA, Winfree L, Wyatt LR, Haendel MA. (2019) "),a("b",{staticClass:"title"},[e._v("An analysis and metric of reusable data licensing practices for biomedical resources")]),e._v(". "),a("i",[e._v("PLoS One")]),e._v(". "),a("b",[e._v("14(3)")]),e._v(":e0213090. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30917137"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Gourdine JF, Brush MH, Vasilevsky NA, Shefchek K, Köhler S, Matentzoglu N, Munoz-Torres MC, McMurry JA, Zhang XA, Robinson PN, Haendel MA. (2019) "),a("b",{staticClass:"title"},[e._v("Representing glycophenotypes: semantic unification of glycobiology resources for disease discovery")]),e._v(". "),a("i",[e._v("Database (Oxford). ")]),e._v(" baz114. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31735951"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Jackson RC, Balhoff JP, Douglass E, Harris NL, Mungall CJ, Overton JA. (2019) "),a("b",{staticClass:"title"},[e._v("ROBOT: A Tool for Automating Ontology Workflows")]),e._v(". "),a("i",[e._v("BMC Bioinformatics.")]),e._v(" "),a("b",[e._v("20(1)")]),e._v(":407. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31357927"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Köhler S, Carmody L, Vasilevsky N, Jacobsen JOB, Danis D, Gourdine JP, Gargano M, Harris NL, Matentzoglu N, McMurry JA, Osumi-Sutherland D, Cipriani V, Balhoff JP, Conlin T, Blau H, Baynam G, Palmer R, Gratian D, Dawkins H, Segal M, Jansen AC, Muaz A, Chang WH, Bergerson J, Laulederkind SJF, Yüksel Z, Beltran S, Freeman AF, Sergouniotis PI, Durkin D, Storm AL, Hanauer M, Brudno M, Bello SM, Sincan M, Rageth K, Wheeler MT, Oegema R, Lourghi H, Della Rocca MG, Thompson R, Castellanos F, Priest J, Cunningham-Rundles C, Hegde A, Lovering RC, Hajek C, Olry A, Notarangelo L, Similuk M, Zhang XA, Gómez-Andrés D, Lochmüller H, Dollfus H, Rosenzweig S, Marwaha S, Rath A, Sullivan K, Smith C, Milner JD, Leroux D, Boerkoel CF, Klion A, Carter MC, Groza T, Smedley D, Haendel MA, Mungall C, Robinson PN (2019) "),a("b",{staticClass:"title"},[e._v("Expansion of the Human Phenotype Ontology (HPO) knowledge base and resources")]),e._v(". "),a("i",[e._v(" Nucleic Acids Res")]),e._v(" "),a("b",[e._v(" 47(D1)")]),e._v(":D1018-D1027. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30476213"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Köhler S, Øien NC, Buske OJ, Groza T, Jacobsen JOB, McNamara C, Vasilevsky N, Carmody LC, Gourdine JP, Gargano M, McMurry JA, Danis D, Mungall CJ, Smedley D, Haendel M, Robinson PN. (2019) "),a("b",{staticClass:"title"},[e._v("Encoding Clinical Data with the Human Phenotype Ontology for Computational Differential Diagnostics")]),e._v(". "),a("i",[e._v("Curr Protoc Hum Gene")]),e._v(". "),a("b",[e._v("103(1)")]),e._v(":e92. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31479590"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Nellåker C, Alkuraya FS, Baynam G, Bernier RA, Bernier FPJ, Boulanger V, Brudno M, Brunner HG, Clayton-Smith J, Cogné B, Dawkins HJS, deVries BBA, Douzgou S, Dudding-Byth T, Eichler EE, Ferlaino M, Fieggen K, Firth HV, FitzPatrick DR, Gration D, Groza T, Haendel M, Hallowell N, Hamosh A, Hehir-Kwa J, Hitz MP, Hughes M, Kini U, Kleefstra T, Kooy RF, Krawitz P, Küry S, Lees M, Lyon GJ, Lyonnet S, Marcadier JL, Meyn S, Moslerová V, Politei JM, Poulton CC, Raymond FL, Reijnders MRF, Robinson PN, Romano C, Rose CM, Sainsbury DCG, Schofield L, Sutton VR, Turnovec M, Van Dijck A, Van Esch H, Wilkie AOM; Minerva Consortium. (2019) "),a("b",{staticClass:"title"},[e._v("Enabling Global Clinical Collaborations on Identifiable Patient Data: The Minerva Initiative")]),e._v(". "),a("i",[e._v("Front Genet.")]),e._v(" "),a("b",[e._v("10")]),e._v(":611. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31417602"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Sergouniotis PI, Maxime E, Leroux D, Olry A, Thompson R, Rath A, Robinson PN, Dollfus H; ERN-EYE Ontology Study Group. (2019) "),a("b",{staticClass:"title"},[e._v("An ontological foundation for ocular phenotypes and rare eye diseases")]),e._v(". "),a("i",[e._v("Orphanet J Rare Dis.")]),e._v(" "),a("b",[e._v("14(1)")]),e._v(":8. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30626441"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Sickle Cell Disease Ontology Working Group. (2019) "),a("b",{staticClass:"title"},[e._v("The Sickle Cell Disease Ontology: enabling universal sickle cell-based knowledge representation")]),a("i",[e._v("Database (Oxford).")]),e._v(" ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31769834"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Taylor DM, Aronow BJ, Tan K, Bernt K, Salomonis N, Greene CS, Frolova A, Henrickson SE, Wells A, Pei L, Jaiswal JK, Whitsett J, Hamilton KE, MacParland SA, Kelsen J, Heuckeroth RO, Potter SS, Vella LA, Terry NA, Ghanem LR, Kennedy BC, Helbig I, Sullivan KE, Castelo-Soccio L, Kreigstein A, Herse F, Nawijn MC, Koppelman GH, Haendel M, Harris NL, Rokita JL, Zhang Y, Regev A, Rozenblatt-Rosen O, Rood JE, Tickle TL, Vento-Tormo R, Alimohamed S, Lek M, Mar JC, Loomes KM, Barrett DM, Uapinyoying P, Beggs AH, Agrawal PB, Chen YW, Muir AB, Garmire LX, Snapper SB, Nazarian J, Seeholzer SH, Fazelinia H, Singh LN, Faryabi RB, Raman P, Dawany N, Xie HM, Devkota B, Diskin SJ, Anderson SA, Rappaport EF, Peranteau W, Wikenheiser-Brokamp KA, Teichmann S, Wallace D, Peng T, Ding YY, Kim MS, Xing Y, Kong SW, Bönnemann CG, Mandl KD, White PS. (2019) "),a("b",{staticClass:"title"},[e._v("The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution")]),e._v(". "),a("i",[e._v("Dev Cell.")]),e._v(" "),a("b",[e._v("49(1)")]),e._v(":10-29. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30930166"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Walls RL, Cooper L, Elser J, Gandolfo MA, Mungall CJ, Smith B, Stevenson DW, Jaiswal P. (2019) "),a("b",{staticClass:"title"},[e._v("The Plant Ontology Facilitates Comparisons of Plant Development Stages Across Species")]),e._v(". "),a("i",[e._v("Front Plant Sci.")]),e._v(" "),a("b",[e._v("10")]),e._v(":631. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31214208"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Zhang XA, Yates A, Vasilevsky N, Gourdine JP, Callahan TJ, Carmody LC, Danis D, Joachimiak MP, Ravanmehr V, Pfaff ER, Champion J, Robasky K, Xu H, Fecho K, Walton NA, Zhu RL, Ramsdill J, Mungall CJ, Köhler S, Haendel MA, McDonald CJ, Vreeman DJ, Peden DB, Bennett TD, Feinstein JA, Martin B, Stefanski AL, Hunter LE, Chute CG, Robinson PN. (2019) "),a("b",{staticClass:"title"},[e._v("Semantic integration of clinical laboratory tests from electronic health records for deep phenotyping and biomarker discovery")]),e._v(". "),a("i",[e._v("NPJ Digit Med.")]),e._v(" "),a("b",[e._v("2")]),e._v(":32. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/31119199"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2018")]),a("div",{staticClass:"text"},[a("p",[e._v(" Haendel MA, Chute CG, Robinson PN 2018. "),a("b",{staticClass:"title"},[e._v("Classification, Ontology, and Precision Medicine (2018)")]),e._v(". "),a("i",[e._v(" N Engl J Med")]),e._v(" "),a("b",[e._v(" 379(15)")]),e._v(":1452-1462. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30304648"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Maiella S, Olry A, Hanauer M, Lanneau V, Lourghi H, Donadille B, Rodwell C, Köhler S, Seelow D, Jupp S, Parkinson H, Groza T, Brudno M, Robinson PN, Rath A (2018) "),a("b",{staticClass:"title"},[e._v("Harmonising phenomics information for a better Interoperability in the rare disease field")]),e._v(". "),a("i",[e._v(" Eur J Med Genet")]),e._v(" "),a("b",[e._v(" 61(11)")]),e._v(":706-714. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30304648"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Muñoz-Fuentes V, Cacheiro P, Meehan TF, Aguilar-Pimentel JA, Brown SDM, Flenniken AM, Flicek P, Galli A, Mashhadi HH, Hrabě de Angelis M, Kim JK, Lloyd KCK, McKerlie C, Morgan H, Murray SA, Nutter LMJ, Reilly PT, Seavitt JR, Seong JK, Simon M, Wardle-Jones H, Mallon AM, Smedley D, Parkinson HE; IMPC consortium (2018) "),a("b",{staticClass:"title"},[e._v("Classification, Ontology, and Precision Medicine")]),e._v(". "),a("i",[e._v("Conserv Genet ")]),e._v(" "),a("b",[e._v("19(4)")]),e._v(":995-1005. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/30100824"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Vasilevsky NA, Foster ED, Engelstad ME, Carmody L, Might M, Chambers C, Dawkins HJS, Lewis J, Della Rocca MG, Snyder M, Boerkoel CF, Rath A, Terry SF, Kent A, Searle B, Baynam G, Jones E, Gavin P, Bamshad M, Chong J, Groza T, Adams D, Resnick AC, Heath AP, Mungall C, Holm IA, Rageth K, Brownstein CA, Shefchek K, McMurry JA, Robinson PN, Köhler S, Haendel MA.(2018) "),a("b",{staticClass:"title"},[e._v("Plain-language medical vocabulary for precision diagnosis")]),e._v(". "),a("i",[e._v("Nat Genet ")]),e._v(" "),a("b",[e._v("50(4)")]),e._v(":474-476. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/29632381"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2017")]),a("div",{staticClass:"text"},[a("p",[e._v(" Mungall CJ, McMurry JA, Köhler S, Balhoff JP, Borromeo C, Brush M, Carbon S, Conlin T, Dunn N, Engelstad M, Foster E, Gourdine JP, Jacobsen JO, Keith D, Laraway B, Lewis SE, NguyenXuan J, Shefchek K, Vasilevsky N, Yuan Z,Washington N, Hochheiser H, Groza T, Smedley D, Robinson PN, Haendel MA (2017) "),a("b",{staticClass:"title"},[e._v("The Monarch Initiative: An Integrative Data and Analytic Platform Connecting Phenotypes to Genotypes across Species")]),e._v(". "),a("i",[e._v(" Nucleic Acids Research")]),e._v(" "),a("b",[e._v(" 45(D1)")]),e._v(":D712–22. ["),a("a",{attrs:{target:"__blank",href:"https://www.ncbi.nlm.nih.gov/pubmed/27899636"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Köhler S, Vasilevsky NA, Engelstad M, Foster E, McMurry J, Aymé S, Baynam G, Bello SM, Boerkoel CF, Boycott KM, Brudno M, Buske OJ, Chinnery PF, Cipriani V, Connell LE, Dawkins HJ, DeMare LE, Devereau AD, de Vries BB, Firth HV, Freson K, Greene D, Hamosh A, Helbig I, Hum C, Jähn JA, James R, Krause R, F Laulederkind SJ, Lochmüller H, Lyon GJ, Ogishima S, Olry A, Ouwehand WH, Pontikos N, Rath A, Schaefer F, Scott RH, Segal M, Sergouniotis PI, Sever R, Smith CL, Straub V, Thompson R, Turner C, Turro E, Veltman MW, Vulliamy T, Yu J, von Ziegenweidt J, Zankl A, Züchner S, Zemojtel T, Jacobsen JO, Groza T, Smedley D, Mungall CJ, Haendel M, Robinson PN (2017) "),a("b",{staticClass:"title"},[e._v("The Human Phenotype Ontology in 2017")]),e._v(". "),a("i",[e._v("Nucleic Acids Research")]),e._v(" "),a("b",[e._v(" 45(D1)")]),e._v(": D865–76. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27899602"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" McMurry JA, Juty N, Blomberg N, Burdett T, Conlin T, Conte N, Courtot M, Deck J, Dumontier M, Fellows DK, Gonzalez-Beltran A, Gormanns P, Grethe J, Hastings J, Hériché JK, Hermjakob H, Ison JC, Jimenez RC, Jupp S, Kunze J, Laibe C, Le Novère N, Malone J, Martin MJ, McEntyre JR, Morris C, Muilu J, Müller W, Rocca-Serra P, Sansone SA, Sariyar M, Snoep JL, Soiland-Reyes S, Stanford NJ, Swainston N, Washington N, Williams AR, Wimalaratne SM, Winfree LM, Wolstencroft K, Goble C, Mungall CJ, Haendel MA, Parkinson H. (2017) "),a("b",{staticClass:"title"},[e._v("Identifiers for the 21st Century: How to Design, Provision, and Reuse Persistent Identifiers to Maximize Utility and Impact of Life Science Data")]),e._v(". "),a("i",[e._v("PLoS Biology")]),e._v(" "),a("b",{staticClass:"title"},[e._v(" 15(6)")]),e._v(":e2001414. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28662064"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Meehan TF, Conte N, West DB, Jacobsen JO, Mason J, Warren J, Chen CK, Tudose I, Relac M, Matthews P, Karp N, Santos L, Fiegel T, Ring N, Westerberg H, Greenaway S, Sneddon D, Morgan H, Codner GF, Stewart ME, Brown J, Horner N; International Mouse Phenotyping Consortium, Haendel M, Washington N, Mungall CJ, Reynolds CL, Gallegos J, Gailus-Durner V, Sorg T, Pavlovic G, Bower LR, Moore M, Morse I, Gao X, Tocchini-Valentini GP, Obata Y, Cho SY, Seong JK, Seavitt J, Beaudet AL, Dickinson ME, Herault Y, Wurst W, de Angelis MH, Lloyd KCK, Flenniken AM, Nutter LMJ, Newbigging S, McKerlie C, Justice MJ, Murray SA, Svenson KL, Braun RE, White JK, Bradley A, Flicek P, Wells S, Skarnes WC, Adams DJ, Parkinson H, Mallon AM, Brown SDM, Smedley D (2017) "),a("b",{staticClass:"title"},[e._v("Disease model discovery from 3,328 gene knockouts by The International Mouse Phenotyping Consortium")]),e._v(". "),a("i",[e._v(" Nat Genet")]),e._v(" "),a("b",[e._v(" 49(8)")]),e._v(":1231-1238. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28650483"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Robinson PN (2017) "),a("b",{staticClass:"title"},[e._v("Once Doesn’t Count: Phenotype-Driven Gene Hunting in Cohorts")]),e._v(". "),a("i",[e._v(" Human Mutat")]),e._v(" "),a("b",[e._v(" 38(5)")]),e._v(":469. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28425197"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Boycott KM, Rath A, Chong JX, Hartley T, Alkuraya FS, Baynam G, Brookes AJ, Brudno M, Carracedo A, den Dunnen JT, Dyke SOM, Estivill X, Goldblatt J, Gonthier C, Groft SC, Gut I, Hamosh A, Hieter P, Höhn S, Hurles ME, Kaufmann P, Knoppers BM, Krischer JP, Macek M Jr, Matthijs G, Olry A, Parker S, Paschall J, Philippakis AA, Rehm HL, Robinson PN, Sham PC, Stefanov R, Taruscio D, Unni D, Vanstone MR, Zhang F, Brunner H, Bamshad MJ, Lochmüller H (2017) "),a("b",{staticClass:"title"},[e._v("International Cooperation to Enable the Diagnosis of All Rare Genetic Diseases")]),e._v(". "),a("i",[e._v(" Am J Hum Genet")]),e._v(" "),a("b",[e._v(" 100(5)")]),e._v(":695-705. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28475856"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Gall T, Valkanas E, Bello C, Markello T, Adams C, Bone WP, Brandt AJ, Brazill JM, Carmichael L, Davids M, Davis J, Diaz-Perez Z, Draper D, Elson J, Flynn ED, Godfrey R, Groden C, Hsieh CK, Fischer R, Golas GA, Guzman J, Huang Y, Kane MS, Lee E, Li C, Links AE, Maduro V, Malicdan MCV, Malik FS, Nehrebecky M, Park J, Pemberton P, Schaffer K, Simeonov D, Sincan M, Smedley D, Valivullah Z, Wahl C, Washington N, Wolfe LA, Xu K, Zhu Y, Gahl WA, Tifft CJ, Toro C, Adams DR, He M, Robinson PN, Haendel MA, Zhai RG, Boerkoel CF (2017) "),a("b",{staticClass:"title"},[e._v("Defining Disease, Diagnosis, and Translational Medicine within a Homeostatic Perturbation Paradigm: The National Institutes of Health Undiagnosed Diseases Program Experience")]),e._v(". "),a("i",[e._v("Front Med (Lausanne) ")]),e._v(" "),a("b",[e._v(" 4")]),e._v(":62. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28603714"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Elhanan G, Ochs C, Mejino JLV Jr, Liu H, Mungall CJ, Perl Y. (2017) "),a("b",{staticClass:"title"},[e._v("From SNOMED CT to Uberon: Transferability of Evaluation Methodology between Similarly Structured Ontologies")]),e._v(". "),a("i",[e._v(" Artif Intell Med")]),e._v(" "),a("b",[e._v(" 79")]),e._v(":9-14. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28532962"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Koehler S, Robinson PN, Mungall CJ (2017) "),a("b",{staticClass:"title"},[e._v("Opposite-of’-Information Improves Similarity Calculations in Phenotype Ontologies")]),e._v(". ["),a("a",{attrs:{target:"__blank",href:"https://www.biorxiv.org/content/10.1101/108977v1"}},[e._v("bioRxiv")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Ong E, Xiang Z, Zhao B, Liu Y, Lin Y, Zheng J, Mungall C, Courtot M, Ruttenberg A, He Y (2017) "),a("b",{staticClass:"title"},[e._v("Ontobee: A linked ontology data server to support ontology term dereferencing, linkage, query and integration")]),e._v(". "),a("i",[e._v(" Nucleic Acids Res")]),e._v(" "),a("b",[e._v(" 45(D1)")]),e._v(":D347-D352. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27733503"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2016")]),a("div",{staticClass:"text"},[a("p",[e._v(" McMurry JA, Köhler S, Washington NL, Balhoff JP, Borromeo C, Brush M, Carbon S, Conlin T, Dunn N, Engelstad M, Foster E, Gourdine JP, Jacobsen JO, Keith D, Laraway B, Xuan JN, Shefchek K, Vasilevsky NA, Yuan Z, Lewis SE, Hochheiser H, Groza T, Smedley D, Robinson PN, Mungall CJ, Haendel MA (2016) "),a("b",{staticClass:"title"},[e._v("Navigating the Phenotype Frontier: The Monarch Initiative")]),e._v(". "),a("i",[e._v(" Genetics")]),e._v(" "),a("b",[e._v(" 203(4)")]),e._v(":1491-5. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27516611"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Dumontier M, Gray AJG, Marshall MS, Alexiev V, Ansell P, Bader G, Baran J, Bolleman JT, Callahan A, Cruz-Toledo J, Gaudet P, Gombocz EA, Gonzalez-Beltran AN, Groth P, Haendel M, Ito M, Jupp S, Juty N, Katayama T, Kobayashi N, Krishnaswami K, Laibe C, Le Novère N, Lin S, Malone J, Miller M, Mungall CJ, Rietveld L, Wimalaratne SM, Yamaguchi A (2016) "),a("b",{staticClass:"title"},[e._v("The health care and life sciences community profile for dataset descriptions")]),e._v(". "),a("i",[e._v("PeerJ")]),e._v(" "),a("b",[e._v(" 4")]),e._v(":e2331. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27602295"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Vasilevsky NA, Minnier J, Haendel MA, Champieux RE (2016) "),a("b",{staticClass:"title"},[e._v("Reproducible and reusable research: are journal data sharing policies meeting the mark?")]),a("i",[e._v(" PeerJ")]),e._v(" "),a("b",[e._v(" 5")]),e._v(":e3208. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/28462024"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Links AE, Draper D, Lee E, Guzman J, Valivullah Z, Maduro V, Lebedev V, Didenko M, Tomlin G, Brudno M, Girdea M, Dumitriu S, Haendel MA, Mungall CJ, Smedley D, Hochheiser H, Arnold AM, Coessens B, Verhoeven S, Bone W, Adams D, Boerkoel CF, Gahl WA, Sincan M (2016) "),a("b",{staticClass:"title"},[e._v("Distributed Cognition and Process Management Enabling Individualized Translational Research: The NIH Undiagnosed Diseases Program Experience")]),e._v(". "),a("i",[e._v(" Front Med (Lausanne)")]),e._v(" "),a("b",[e._v(" 3")]),e._v(":39. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27785453"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Mungall CJ, Koehler S, Robinson P, Holmes I, Haendel M (2016) "),a("b",{staticClass:"title"},[e._v("K-BOOM: A Bayesian Approach to Ontology Structure Inference, with Applications in Disease Ontology Construction")]),e._v(". "),a("i",[e._v(" bioRxiv")]),e._v(". doi:10.1101/048843. ["),a("a",{attrs:{href:"https://www.biorxiv.org/content/10.1101/048843v3"}},[e._v("bioRxiv")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Bandrowski A, Brinkman R, Brochhausen M, Brush MH, Bug B, Chibucos MC, Clancy K, Courtot M, Derom D, Dumontier M, Fan L, Fostel J, Fragoso G, Gibson F, Gonzalez-Beltran A, Haendel MA, He Y, Heiskanen M, Hernandez-Boussard T, Jensen M, Lin Y, Lister AL, Lord P, Malone J, Manduchi E, McGee M, Morrison N, Overton JA, Parkinson H, Peters B, Rocca-Serra P, Ruttenberg A, Sansone SA, Scheuermann RH, Schober D, Smith B, Soldatova LN, Stoeckert CJ Jr, Taylor CF, Torniai C, Turner JA, Vita R, Whetzel PL, Zheng J (2016) "),a("b",{staticClass:"title"},[e._v("The Ontology for Biomedical Investigations")]),e._v(". "),a("i",[e._v(" PLoS One")]),e._v(" "),a("b",[e._v(" 11(4)")]),e._v(":e0154556. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27128319"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Diehl AD, Meehan TF, Bradford YM, Brush MH, Dahdul WM, Dougall DS, He Y, Osumi-Sutherland D, Ruttenberg A, Sarntivijai S, Van Slyke CE, Vasilevsky NA, Haendel MA, Blake JA, Mungall CJ (2016) "),a("b",{staticClass:"title"},[e._v("The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability")]),e._v(". "),a("i",[e._v(" J Biomed Semantics")]),e._v(" "),a("b",[e._v(" 7(1)")]),e._v(":44. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27377652"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Oellrich A, Collier N, Groza T, Rebholz-Schuhmann D, Shah N, Bodenreider O, Boland MR, Georgiev I, Liu H, Livingston K, Luna A, Mallon AM, Manda P, Robinson PN, Rustici G, Simon M, Wang L, Winnenburg R, Dumontier M (2016) "),a("b",{staticClass:"title"},[e._v("The digital revolution in phenotyping")]),e._v(". "),a("i",[e._v(" Brief Bioinform")]),e._v(" "),a("b",[e._v(" 17(5)")]),e._v(":819-30. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26420780"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Smedley D, Schubach M, Jacobsen JOB, Köhler S, Zemojtel T, Spielmann M, Jäger M, Hochheiser H, Washington NL, McMurry JA, Haendel MA, Mungall CJ, Lewis SE, Groza T, Valentini G, Robinson PN (2016) "),a("b",{staticClass:"title"},[e._v("A Whole-Genome Analysis Framework for Effective Identification of Pathogenic Regulatory Variants in Mendelian Disease")]),e._v(". "),a("i",[e._v(" Am J Hum Genet")]),e._v(" "),a("b",[e._v(" 99(3)")]),e._v(":595-606. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/27569544"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Bone WP, Washington NL, Buske OJ, Adams DR, Davis J, Draper D, Flynn ED, Girdea M, Godfrey R, Golas G, Groden C, Jacobsen J, Köhler S, Lee EM, Links AE, Markello TC, Mungall CJ, Nehrebecky M, Robinson PN, Sincan M, Soldatos AG, Tifft CJ, Toro C, Trang H, Valkanas E, Vasilevsky N, Wahl C, Wolfe LA, Boerkoel CF, Brudno M, Haendel MA, Gahl WA, Smedley D (2016) "),a("b",{staticClass:"title"},[e._v("Computational evaluation of exome sequence data using human and model organism phenotypes improves diagnostic efficiency")]),e._v(". "),a("i",[e._v(" Genet Med")]),e._v(" "),a("b",[e._v(" 18(6)")]),e._v(":608-17. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26562225"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Smedley D, Jacobsen JO, Jäger M, Köhler S, Holtgrewe M, Schubach M, Siragusa E, Zemojtel T, Buske OJ, Washington NL, Bone WP, Haendel MA, Robinson PN (2016) "),a("b",[e._v("Next-generation diagnostics and disease-gene discovery with the Exomiser")]),e._v(". "),a("i",[e._v(" Nat Protoc")]),e._v(" "),a("b",[e._v(" 10(12)")]),e._v(":2004-15. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26562621"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Bone WP, Washington NL, Buske OJ, Adams DR, Davis J, Draper D, Flynn ED, Girdea M, Godfrey R, Golas G, Groden C, Jacobsen J, Köhler S, Lee EM, Links AE, Markello TC, Mungall CJ, Nehrebecky M, Robinson PN, Sincan M, Soldatos AG, Tifft CJ, Toro C, Trang H, Valkanas E, Vasilevsky N, Wahl C, Wolfe LA, Boerkoel CF, Brudno M, Haendel MA, Gahl WA, Smedley D (2016) "),a("b",{staticClass:"title"},[e._v("Computational evaluation of exome sequence data using human and model organism phenotypes improves diagnostic efficiency")]),e._v(". "),a("i",[e._v(" Genet Med.=")]),e._v(" "),a("b",[e._v(" 18(6)")]),e._v(":608-17. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26562225"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2015")]),a("div",{staticClass:"text"},[a("p",[e._v(" Brookes AJ, Robinson PN (2015) "),a("b",{staticClass:"title"},[e._v("Human genotype-phenotype databases: aims, challenges and opportunities.")]),a("i",[e._v("Nature Rev Genet")]),e._v(" "),a("b",[e._v(" 16(12)")]),e._v(":702-15. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/    26553330"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Oellrich A, Collier N, Groza T, Rebholz-Schuhmann D, Shah N, Bodenreider O, Boland MR, Georgiev I, Liu H, Livingston K, Luna A, Mallon AM, Manda P, Robinson PN, Rustici G, Simon M, Wang L, Winnenburg R, Dumontier M (2015) "),a("b",{staticClass:"title"},[e._v("The digital revolution in phenotyping.")]),a("i",[e._v(" Brief Bioinform")]),e._v(" "),a("b",[e._v(" 17(5)")]),e._v(":819-30 ["),a("a",{attrs:{target:"__blank",href:"http://bib.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=26420780"}},[e._v("abstract")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Smedley D, Robinson PN (2015) "),a("b",{staticClass:"title"},[e._v("Phenotype-driven strategies for exome prioritization of human Mendelian disease genes.")]),a("i",[e._v(" Genome Med")]),e._v(" "),a("b",[e._v(" 7(1)")]),e._v(":81 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26229552"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Yang H, Robinson PN, Wang K (2015) "),a("b",{staticClass:"title"},[e._v("Phenolyzer: phenotype-based prioritization of candidate genes for human diseases.")]),a("i",[e._v(" Nat Methods")]),e._v(" "),a("b",[e._v(" 12(9)")]),e._v(":841-3 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26192085"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Groza T, Köhler S, Moldenhauer D, Vasilevsky N, Baynam G, Zemojtel T, Schriml LM, Kibbe WA, Schofield PN, Beck T, Vasant D, Brookes AJ, Zankl A, Washington NL, Mungall CJ, Lewis SE, Haendel MA, Parkinson H, Robinson PN (2015) "),a("b",{staticClass:"title"},[e._v(" The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease")]),e._v(". "),a("i",[e._v(" Am J Hum Genet")]),e._v(" "),a("b",[e._v(" 97(1)")]),e._v(":111-24. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26119816"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Haendel MA, Vasilevsky N, Brush M, Hochheiser HS, Jacobsen J, Oellrich A, Mungall CJ, Washington N, Köhler S, Lewis SE, Robinson PN, Smedley D (2015) "),a("b",{staticClass:"title"},[e._v(" Disease insights through cross-species phenotype comparisons")]),e._v(". "),a("i",[e._v(" Mamm Genome")]),e._v(" "),a("b",[e._v(" 26(9-10)")]),e._v(":548-55. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/26092691"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Deans AR, Lewis SE, Huala E, Anzaldo SS, Ashburner M, Balhoff JP, Blackburn DC, Blake JA, Burleigh JG, Chanet B, Cooper LD, Courtot M, Csösz S, Cui H, Dahdul W, Das S, Dececchi TA, Dettai A, Diogo R, Druzinsky RE, Dumontier M, Franz NM,Friedrich F, Gkoutos GV, Haendel M, Harmon LJ, Hayamizu TF, He Y, Hines HM, Ibrahim N, Jackson LM, Jaiswal P, James-Zorn C, Köhler S, Lecointre G, Lapp H, Lawrence CJ, Le Novère N, Lundberg JG, Macklin J, Mast AR, Midford PE, Mikó I, Mungall CJ, Oellrich A, Osumi-Sutherland D, Parkinson H, Ramírez MJ, Richter S, Robinson PN, Ruttenberg A, Schulz KS, Segerdell E, Seltmann KC, Sharkey MJ, Smith AD, Smith B, Specht CD, Squires RB, Thacker RW, Thessen A, Fernandez-Triana J, Vihinen M, Vize PD, Vogt L, Wall CE, Walls RL, Westerfeld M, Wharton RA, Wirkner CS, Woolley JB, Yoder MJ, Zorn AM, Mabee P (2015) "),a("b",{staticClass:"title"},[e._v(" Finding our way through phenotypes")]),e._v(". "),a("i",[e._v(" PLoS Biol")]),e._v(" "),a("b",[e._v(" 13(1)")]),e._v(":e1002033. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25562316"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2014")]),a("div",{staticClass:"text"},[a("p",[e._v(" Ibn-Salem J, Köhler S, Love MI, Chung H-R, Huang N, Hurles ME, Haendel M, Washington NL, Smedley D, Mungall CJ, Lewis SE, Ott C-E, Bauer S, Schofield PN, Mundlos S, Spielmann M and Robinson PN (2014) "),a("b",{staticClass:"title"},[e._v("Deletions of chromosomal regulatory boundaries are associated with congenital disease")]),a("i",[e._v(" Genome Biol")]),e._v(" "),a("b",[e._v(" 15(9)")]),e._v(":423 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25315429"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Zemojtel T, Kohler S, Mackenroth L, Jager M, Hecht J, Krawitz P, Graul-Neumann L, Doelken S, Ehmke N, Spielmann M, Oien NC, Schweiger MR, Kruger U, Frommer G, Fischer B, Kornak U, Flottmann R, Ardeshirdavani A, Moreau Y, Lewis SE, Haendel M, Smedley D, Horn D, Mundlos S, Robinson PN (2014) "),a("b",{staticClass:"title"},[e._v("Effective diagnosis of genetic disease by computational phenotype analysis of the disease-associated genome.")]),a("i",[e._v(" Sci Transl Med.")]),e._v(" "),a("b",[e._v(" 6(252)")]),e._v(":252ra123. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25186178"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Dahdul WM, Cui H, Mabee PM, Mungall CJ, Osumi-Sutherland D, Walls RL, Haendel MA (2014) "),a("b",{staticClass:"title"},[e._v("Nose to tail, roots to shoots: spatial descriptors for phenotypic diversity in the Biological Spatial Ontology.")]),a("i",[e._v(" J Biomed Semantics.")]),e._v(" "),a("b",[e._v(" 5")]),e._v(":34. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25140222"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Sarntivijai S., Lin Y, Xiang Z., Meehan TF, Diehl AD, Vempati UD, Schurer SC, Pang C, Malone J, Parkinson H, Liu Y, Takatsuki T, Saijo K, Masuy H, Nakamura Y, Brush MH, Haendel MA, Zheng J, Stoeckert CJ, Peters B, Mungall CJ, Carey TE, States DJ, Athey BD, He Y. (2014). "),a("b",{staticClass:"title"},[e._v("CLO: The cell line ontology.")]),a("i",[e._v("J Biomed Semantics.")]),e._v(" "),a("b",[e._v("5(1)")]),e._v(":37. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25852852"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Smedley D, Kohler S, Czeschik JC, Amberger J, Bocchini C, Hamosh A, Veldboer J, Zemojtel T, Robinson PN (2014) "),a("b",{staticClass:"title"},[e._v("Walking the interactome for candidate prioritization in exome sequencing studies of Mendelian diseases.")]),a("i",[e._v("Bioinformatics.")]),e._v(" "),a("b",[e._v(" 30(22)")]),e._v(":3215-22. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25078397"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Oellrich A, Koehler S, Washington N; Sanger Mouse Genetic Project, Mungall C, Lewis S, Haendel M, Robinson PN, Smedley D (2014) "),a("b",{staticClass:"title"},[e._v("The influence of disease categories on gene candidate predictions from model organism phenotypes.")]),a("i",[e._v("J Biomed Semantics")]),a("b",[e._v(" 5(Suppl 1 Proceedings of the Bio-Ontologies Spec Interest G)")]),e._v(":S4. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/25093073"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Robinson PN, Kohler S, Oellrich A, Sanger Mouse Genetics Project, Wang K, Mungall CJ, Lewis SE, Washington N, Bauer S, Seelow D, Krawitz P, Gilissen C, Haendel M, Smedley D (2014) "),a("b",{staticClass:"title"},[e._v("Improved exome prioritization of disease genes through cross-species phenotype comparison.")]),a("i",[e._v(" Genome Res")]),e._v(" "),a("b",[e._v(" 24(2)")]),e._v(":340-8. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24162188"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Van Slyke CE, Bradford YM, Westerfield M, Haendel MA (2014) "),a("b",{staticClass:"title"},[e._v("The zebrafish anatomy and stage ontologies: representing the anatomy and development of Danio rerio.")]),a("i",[e._v("J Biomed Semantics.")]),e._v(" "),a("b",[e._v(" 5(1)")]),e._v(":12. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24568621"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Köhler S, Doelken SC, Mungall CJ, Bauer S, Firth HV, Bailleul-Forestier I, Black GC, Brown DL, Brudno M, Campbell J, FitzPatrick DR, Eppig JT, Jackson AP, Freson K, Girdea M, Helbig I, Hurst JA, JÃ¤hn J, Jackson LG, Kelly AM, Ledbetter DH, Mansour S, Martin CL, Moss C, Mumford A, Ouwehand WH, Park SM, Riggs ER, Scott RH, Sisodiya S, Van Vooren S, Wapner RJ, Wilkie AO, Wright CF, Vulto-van Silfhout AT, de Leeuw N, de Vries BB, Washingthon NL, Smith CL, Westerfield M, Schofield P, Ruef BJ, Gkoutos GV, Haendel M, Smedley D, Lewis SE, Robinson PN (2014) "),a("b",{staticClass:"title"},[e._v("The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data.")]),a("i",[e._v(" Nucleic Acids Res.")]),e._v(" "),a("b",[e._v(" 42(Database issue)")]),e._v(":D966-74. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24217912"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2013")]),a("div",{staticClass:"text"},[a("p",[e._v(" Arighi CN, Carterette B, Cohen KB, Krallinger M, Wilbur WJ, Fey P, Dodson R, Cooper L, Van Slyke CE, Dahdul W, Mabee P, Li D, Harris B, Gillespie M, Jimenez S, Roberts P, Matthews L, Becker K, Drabkin H, Bello S, Licata L,Chatr-aryamontri A, Schaeffer ML, Park J, Haendel M, Van Auken K, Li Y, Chan J, Muller HM, Cui H, Balhoff JP, Chi-Yang Wu J, Lu Z, Wei CH, Tudor CO, Raja K, Subramani S, Natarajan J, Cejuela JM, Dubey P, Wu C (2013) "),a("b",{staticClass:"title"},[e._v("An overview of the BioCreative 2012 Workshop Track III: interactive text mining task.")]),a("i",[e._v(" Database (Oxford)")]),e._v(" "),a("b",[e._v(" 2013")]),e._v(":bas056 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23327936"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Buttigieg PL, Morrison N, Smith B, Mungall CJ, Lewis SE; ENVO Consortium (2013) "),a("b",{staticClass:"title"},[e._v("The environment ontology: contextualising biological and biomedical entities.")]),a("i",[e._v(" J Biomed Semantics")]),e._v(" "),a("b",[e._v("4(1)")]),e._v(":43 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24330602"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Doelken SC, Kohler S, Mungall CJ, Gkoutos GV, Ruef BJ, Smith C, Smedley D, Bauer S, Klopocki E, Schofield PN, Westerfield M, Robinson PN, Lewis SE (2013) "),a("b",{staticClass:"title"},[e._v("Phenotypic overlap in the contribution of individual genes to CNV pathogenicity revealed by cross-species computational analysis of single-gene mutations in humans, mice and zebrafish.")]),a("i",[e._v(" Dis Model Mech")]),e._v(" "),a("b",[e._v(" 6(2)")]),e._v(":358-72 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23104991"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Druzinsky R, Mungall C, Haendel M, Lapp H, Mabee P. "),a("b",{staticClass:"title"},[e._v("What is an anatomy ontology? (2013)")]),a("i",[e._v(" Anat Rec (Hoboken)")]),e._v(" "),a("b",[e._v(" 296(12)")]),e._v(":1797-9 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24127438"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Heinrich V, Kamphans T, Stange J, Parkhomchuk D, Hecht J, Dickhaus T, Robinson PN, Krawitz PM (2013)"),a("b",{staticClass:"title"},[e._v("Estimating exome genotyping accuracy by comparing to data from large scale sequencing projects.")]),a("i",[e._v(" Genome Med")]),e._v(" "),a("b",[e._v(" 5(7)")]),e._v(":69 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23902830"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Kohler S, Doelken SC, Ruef BJ, Bauer S, Washington N, Westerfield M, Gkoutos G, Schofield P, Smedley D, Lewis SE, Robinson PN, Mungall CJ (2013) "),a("b",{staticClass:"title"},[e._v("Construction and accessibility of a cross-species phenotype ontology along with gene annotations for biomedical research.")]),e._v(" Version 2. "),a("i",[e._v(" F1000Res")]),e._v(" "),a("b",[e._v(" 2")]),e._v(":30 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24358873"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Krawitz PM, Hochsmann B, Murakami Y, Teubner B, Kruger U, Klopocki E, Neitzel H, Hoellein A, Schneider C, Parkhomchuk D, Hecht J, Robinson PN, Mundlos S, Kinoshita T, Schrezenmeier H (2013) "),a("b",{staticClass:"title"},[e._v("A case of paroxysmal nocturnal hemoglobinuria caused by a germline mutation and a somatic mutation in PIGT.")]),a("i",[e._v(" Blood")]),e._v(" "),a("b",[e._v(" 122(7)")]),e._v(":1312-5 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23733340"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Maynard SM, Mungall CJ, Lewis SE, Imam FT, Martone ME. "),a("b",{staticClass:"title"},[e._v("A knowledge based approach to matching human neurodegenerative disease and animal models (2013)")]),a("i",[e._v(" Front Neuroinform")]),e._v(" "),a("b",[e._v(" 7")]),e._v(":7 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23717278"}},[e._v("pubmed.")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Meehan TF, Vasilevsky NA, Mungall CJ, Dougall DS, Haendel MA, Blake JA, Diehl AD. "),a("b",{staticClass:"title"},[e._v("Ontology based molecular signatures for immune cell types via gene expression analysis (2013)")]),a("i",[e._v(" BMC Bioinformatics")]),e._v(" "),a("b",[e._v(" 14")]),e._v(":263 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24004649"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Robinson PN. "),a("b",{staticClass:"title"},[e._v("A systematic large-scale phenotypic analysis of de novo and inherited copy number variation (2013)")]),a("i",[e._v(" Hum Mutat")]),e._v(" "),a("b",[e._v(" 34(12)")]),e._v(":v. ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24227377"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Roncaglia P, Martone ME, Hill DP, Berardini TZ, Foulger RE, Imam FT, Drabkin H, Mungall CJ, Lomax J (2013) "),a("b",{staticClass:"title"},[e._v("The Gene Ontology (GO) Cellular Component Ontology: integration with SAO (Subcellular Anatomy Ontology) and other recent developments.")]),a("i",[e._v(" J Biomed Semantics")]),e._v(" "),a("b",[e._v(" 4(1)")]),e._v(":20 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/24093723"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Smedley D, Oellrich A, Kohler S, Ruef B; Sanger Mouse Genetics Project, Westerfield M, Robinson P, Lewis S, Mungall C (2013) "),a("b",{staticClass:"title"},[e._v("PhenoDigm: analyzing curated annotations to associate animal models with human diseases.")]),a("i",[e._v(" Database (Oxford)")]),e._v(" "),a("b",[e._v(" 2013")]),e._v(":bat025 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23660285"}},[e._v("pubmed")]),e._v("] ")])])]),a("div",{staticClass:"card"},[a("h3",[e._v("2012")]),a("div",{staticClass:"text"},[a("p",[e._v(" Chen CK, Mungall CJ, Gkoutos GV, Doelken SC, Kohler S, Ruef BJ, Smith C, Westerfield M, Robinson PN, Lewis SE, Schofield PN, Smedley D (2012) "),a("b",{staticClass:"title"},[e._v("MouseFinder: Candidate disease genes from mouse phenotype data.")]),a("i",[e._v(" Hum Mutat")]),e._v(" "),a("b",[e._v(" 33(5)")]),e._v(":858-66 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/22331800"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Dahdul WM, Balhoff JP, Blackburn DC, Diehl AD, Haendel MA, Hall BK, Lapp H, Lundberg JG, Mungall CJ, Ringwald M, Segerdell E, Van Slyke CE, Vickaryous MK, Westerfield M, Mabee PM (2012) "),a("b",{staticClass:"title"},[e._v("A unified anatomy ontology of the vertebrate skeletal system.")]),a("i",[e._v(" PLoS One")]),e._v(" "),a("b",[e._v(" 7(12)")]),e._v(":e51070 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/23251424"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Mungall CJ, Torniai C, Gkoutos GV, Lewis SE, Haendel MA. "),a("b",{staticClass:"title"},[e._v("Uberon, an integrative multi-species anatomy ontology (2012)")]),a("i",[e._v(" Genome Biol")]),e._v(" "),a("b",[e._v(" 13(1)")]),e._v(":R5 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/22293552"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Osumi-Sutherland D, Reeve S, Mungall CJ, Neuhaus F, Ruttenberg A, Jefferis GS, Armstrong JD (2012) "),a("b",{staticClass:"title"},[e._v("A strategy for building neuroanatomy ontologies.")]),a("i",[e._v(" Bioinformatics")]),e._v(" "),a("b",[e._v(" 28(9)")]),e._v(":1262-9 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/22402613"}},[e._v("pubmed")]),e._v("] ")])]),a("div",{staticClass:"text"},[a("p",[e._v(" Thomas PD, Wood V, Mungall CJ, Lewis SE, Blake JA, Gene Ontology Consortium (2012) "),a("b",{staticClass:"title"},[e._v("On the Use of Gene Ontology Annotations to Assess Functional Similarity among Orthologs and Paralogs: A Short Report.")]),a("i",[e._v(" PLoS Comput Biol")]),e._v(" "),a("b",[e._v(" 8(2)")]),e._v(":e1002386 ["),a("a",{attrs:{target:"__blank",href:"http://www.ncbi.nlm.nih.gov/pubmed/22359495"}},[e._v("pubmed")]),e._v("] ")])])])])])])])])}],o=(a("b035"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},"2dad":function(e,t,a){},3:function(e,t){},"317c":function(e,t,a){"use strict";a("4b32")},3763:function(e,t,a){},"3d49":function(e,t,a){e.exports=a.p+"img/phenotype-cov-hum-6.f5edcadc.png"},4:function(e,t,a){e.exports=a("56d7")},"443d":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_MODEL.a0fd8e11.png"},"4b21":function(e,t,a){},"4b32":function(e,t,a){},"4e41":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view about-phenotypes"},[a("h2",{attrs:{id:"about-phenotypes"}},[e._v("About Phenotypes "),a("a",{staticClass:"header-anchor",attrs:{href:"#about-phenotypes",target:"_self",rel:""}},[e._v("🔗")])]),a("p",[e._v("A "),a("em",[e._v("phenotype")]),e._v(" is an observable trait, such as height, eye color, or blood type. Phenotypes arise from the complex interplay of genetic and environmental factors.")]),a("p",[e._v("A "),a("em",[e._v("phenotype profile")]),e._v(" is simply a collection of phenotypes, that are specific to an individual organism, and which occur at a given point in time. Diseases can also have a canonical presentation which can be represented as a phenotype profile. In the case of a disease, the corresponding phenotypes are commonly referred to as signs and symptoms such as fever, hypertension, or joint laxity.")]),a("p",[e._v("When clinicians observe a cluster of individuals with similar phenotype profiles that do not match any known disease, they may describe these features in a scientific paper. Monarch curators and others have systematically read such papers and have extracted structured representations of the corresponding phenotype profiles for each known rare disease; the phenotype terms in these gold standard profiles are from the Human Phenotype Ontology.")])])])])])}],o={name:"AboutPhenotypes",components:{}},i=o,s=(a("317c"),a("2877")),l=Object(s["a"])(i,n,r,!1,null,null,null);t["default"]=l.exports},5:function(e,t){},"50dc":function(e,t,a){e.exports=a.p+"img/HPO-figure-for-monarchui.4187299d.png"},"51db":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_VARIANT.f43f1046.png"},"53d7":function(e,t,a){"use strict";a("01f7")},"543d":function(e,t,a){"use strict";a("af03")},"55b4":function(e,t,a){},5631:function(e,t,a){"use strict";var n=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"monarch-autocomplete autocomplete autorootdiv",class:{"home-search":e.homeSearch,open:e.open}},[a("div",{staticClass:"input-group",class:{"input-group-sm":!e.homeSearch}},[a("div",{staticClass:"input-group-prepend"},[a("button",{staticClass:"btn btn-secondary dropdown-toggle",attrs:{type:"button"},on:{click:e.toggleFilterBox}},[e._v(" "+e._s(e.selectDisplay)+" ")]),e.catDropDown?a("div",{directives:[{name:"click-outside",rawName:"v-click-outside",value:e.closeFilterBox,expression:"closeFilterBox"}],staticClass:"dropdown-menu list-group dropCatList px-4"},[a("div",{staticClass:"form-group"},[a("b-form-radio-group",{attrs:{options:e.options,stacked:""},model:{value:e.category,callback:function(t){e.category=t},expression:"category"}})],1)]):e._e()]),a("input",{directives:[{name:"model",rawName:"v-model",value:e.value,expression:"value"},{name:"click-outside",rawName:"v-click-outside",value:e.toggleSuggestions,expression:"toggleSuggestions"}],staticClass:"form-control xform-control-sm test-input-search-text",class:{loading:e.loading},attrs:{placeholder:e.dynamicPlaceholder,type:"text"},domProps:{value:e.value},on:{input:[function(t){t.target.composing||(e.value=t.target.value)},e.debounceInput],keydown:[e.inputChanged,function(t){return!t.type.indexOf("key")&&e._k(t.keyCode,"enter",13,t.key,"Enter")?null:e.enter.apply(null,arguments)},function(t){return!t.type.indexOf("key")&&e._k(t.keyCode,"down",40,t.key,["Down","ArrowDown"])?null:e.down.apply(null,arguments)},function(t){return!t.type.indexOf("key")&&e._k(t.keyCode,"up",38,t.key,["Up","ArrowUp"])?null:e.up.apply(null,arguments)},function(t){return!t.type.indexOf("key")&&e._k(t.keyCode,"esc",27,t.key,["Esc","Escape"])?null:e.clearSearch.apply(null,arguments)}],focus:e.toggleSuggestions}}),e.showSearchButton?a("div",{staticClass:"input-group-append"},[a("button",{directives:[{name:"b-tooltip",rawName:"v-b-tooltip.topright",modifiers:{topright:!0}}],staticClass:"btn xbtn-sm btn-light py-0 test-button-show-all",attrs:{type:"button",title:"Show all matching results"},on:{click:e.showMore}},[a("i",{staticClass:"p-0 m-0 fa xfa-2x fa-search-plus fa-solid"})])]):e._e(),e.open?a("div",{staticClass:"dropdown-menu list-group dropList mx-2 test-input-dropdown",class:{"full-width-search":e.fullWidthSearch}},[e._l(e.suggestions,(function(t,n){return a("div",{key:n,staticClass:"border-bottom m-0 px-1 test-input-dropdown-option",class:{active:e.isActive(n)},on:{mousedown:function(t){return e.suggestionClick(n)},mouseover:function(t){return e.mouseOver(n)}}},[a("div",{staticClass:"row m-0 p-0"},[t.has_hl?a("div",{staticClass:"col-5"},[a("span",{domProps:{innerHTML:e._s(e.$sanitizeText(t.highlight))}})]):a("div",{staticClass:"col-5"},[a("strong",[e._v(e._s(t.match))])]),a("div",{staticClass:"col-4"},[a("i",[e._v(e._s(t.taxon))])]),a("div",{staticClass:"col-3 text-align-right"},[a("small",[e._v(e._s(t.category))])])])])})),a("div",{staticClass:"row mx-3"},[0===e.suggestions.length?a("div",{staticClass:"col border"},[a("b",[e._v("No results for '"+e._s(e.value)+"'")])]):e._e()])],2):e._e()]),e.homeSearch?a("div",{staticClass:"examples"},[e._v(" e.g. "),e._l(e.exampleSearches,(function(t,n){return a("button",{key:n,staticClass:"btn btn-outline-light m-1 py-0 example-button",attrs:{role:"button"},on:{click:function(a){return e.useExample(t.searchString,t.category)}}},[e._v(" "+e._s(t.searchString)+" ")])}))],2):e._e()])},r=[],o=a("a34a"),i=a.n(o),s=a("c28b"),l=a.n(s),c=a("c1be"),d=a("733f");function u(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function h(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){u(o,n,r,i,s,"next",e)}function s(e){u(o,n,r,i,s,"throw",e)}i(void 0)}))}}var p=a("b047"),v=[{searchString:"Marfan Syndrome"},{searchString:"Multicystic kidney dysplasia",category:"phenotype"},{searchString:"SHH",category:"gene"}],m={name:"AutoComplete",filters:{allLower:function(e){return e.toLowerCase()}},directives:{clickOutside:l.a.directive},props:{showSearchButton:{type:Boolean,required:!1,default:!1},autoFocus:{type:Boolean,required:!1,default:!1},homeSearch:{type:Boolean,required:!1,default:!1},fullWidthSearch:{type:Boolean,required:!1,default:!1},definedCategories:{type:Array,required:!1,default:null},allowedPrefixes:{type:Array,required:!1,default:null},searchFilters:{type:Array,required:!1,default:null},dynamicPlaceholder:{type:String,required:!1,default:"Explore Monarch for phenotypes, diseases, genes and more.."}},data:function(){return{destroying:!1,category:"all",selectDisplay:"All",exampleSearches:v,options:[],catDropDown:!1,value:"",suggestions:[],open:!1,current:-1,loading:!1}},watch:{value:function(){this.value||this.clearSearch()},category:function(e){this.selectDisplay=this.options.find((function(t){return t.value===e})).text,this.closeFilterBox(),this.definedCategories||(this.suggestions=[]),this.value.length>0&&this.fetchData()},definedCategories:function(){this.configureOptions()}},mounted:function(){this.configureOptions()},beforeUnmount:function(){this.destroying=!0},methods:{debounceInput:p((function(){this.destroying||this.value&&this.value.length>=3&&this.fetchData()}),500,{leading:!1,trailing:!0}),fetchData:function(){var e=this;return h(i.a.mark((function t(){var a;return i.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:if(t.prev=0,"all-subset"!==e.category){t.next=7;break}return t.next=4,c["l"](e.value,e.definedCategories,e.allowedPrefixes,e.searchFilters);case 4:a=t.sent,t.next=10;break;case 7:return t.next=9,c["l"](e.value,e.category,e.allowedPrefixes,e.searchFilters);case 9:a=t.sent;case 10:e.suggestions=[],e.current=-1,a.docs.forEach((function(t){var a={match:t.match,category:Object(d["a"])(t.category),taxon:e.checkTaxon(t.taxon_label),curie:t.id,highlight:t.highlight,has_hl:t.has_highlight};e.suggestions.push(a)})),e.open=!0,e.loading=!1,t.next=19;break;case 17:t.prev=17,t.t0=t["catch"](0);case 19:case"end":return t.stop()}}),t,null,[[0,17]])})))()},configureOptions:function(){var e=this;this.options=[],this.definedCategories?(this.definedCategories.length>1&&this.options.push({text:"All",value:"all-subset"}),this.definedCategories.forEach((function(t){e.options.push({text:e.firstCap(t),value:t})})),1===this.definedCategories.length?this.category=this.options[0].value:this.category="all-subset"):this.options=[{text:"All",value:"all"},{text:"Gene",value:"gene"},{text:"Genotype",value:"genotype"},{text:"Variant",value:"variant"},{text:"Phenotype",value:"phenotype"},{text:"Disease",value:"disease"}]},enter:function(){var e=this.suggestions[this.current];e?this.definedCategories?this.$emit("interface",e):this.$router.push({path:"/".concat(e.category,"/").concat(e.curie)}):this.showMore(),this.value="",this.open=!1,this.suggestions=[]},up:function(){this.current>0&&(this.current-=1)},toggleSuggestions:function(e){this.open?this.open=!1:this.suggestions.length>0&&"INPUT"===e.target.nodeName&&(this.open=!0)},closeFilterBox:function(){this.catDropDown=!1},toggleFilterBox:function(){this.catDropDown=!this.catDropDown},inputChanged:function(){this.catDropDown=!1},down:function(){this.current<this.suggestions.length-1&&(this.current+=1)},isActive:function(e){return e===this.current},mouseOver:function(e){this.current=e},suggestionClick:function(e){var t=this.suggestions[e];this.definedCategories?this.$emit("interface",this.suggestions[e]):this.$router.push({path:"/".concat(t.category,"/").concat(t.curie)}),this.value="",this.open=!1,this.suggestions=[]},showMore:function(){this.$router.push({path:"/search/".concat(this.value)})},clearSearch:function(){this.suggestions=[],this.open=!1},firstCap:function(e){return e.charAt(0).toUpperCase()+e.substr(1)},checkTaxon:function(e){return"string"===typeof e?e:null},useExample:function(e,t){this.value=e,t?this.category=t:this.fetchData()}}},g=m,f=(a("10e5"),a("2877")),b=Object(f["a"])(g,n,r,!1,null,null,null);t["a"]=b.exports},"56d7":function(e,t,a){"use strict";a.r(t);var n=a("5ee5"),r=a.n(n),o=a("1e18"),i=(a("2dd8"),a("b380"),a("a584")),s=a("8c4f"),l=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"container-fluid monarch-home-view"},[a("vml-home-intro-component"),a("vml-home-about-monarch-component"),a("vml-home-news-component"),a("disease-carousel"),a("vml-home-partners-component")],1)},c=[],d=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",[e.activeCarousel?a("div",{staticClass:"disease-carousel"},[a("h3",{staticClass:"carousel-title"},[e._v("Featured Diseases")]),a("p",{staticClass:"carousel-sub"},[e._v(" We are always updating our corpus of disease - phenotype associations to support current scientific understanding of disease. ")]),e.activeCarousel.id?a("div",{staticClass:"row carousel-row"},[a("div",{staticClass:"col-md disease-information"},[a("h3",{staticClass:"disease-name"},[a("a",{attrs:{href:"/disease/"+e.activeCarousel.id}},[e._v(" "+e._s(e.activeCarousel.label)+" ")]),a("span",{staticClass:"disease-id"},[e._v(" "+e._s(e.activeCarousel.id)+" ")])]),a("p",{staticClass:"disease-description"},[e._v(" "+e._s(e.activeCarousel.description)+" ")]),a("b-button",{staticClass:"linked-data-btn",attrs:{to:"/disease/"+e.activeCarousel.id}},[e._v(" View Linked Data "),a("i",{staticClass:"fa fa-caret-right"})])],1),a("div",{staticClass:"col-md disease-phenotypes"},[a("histo-pheno",{attrs:{"active-item":e.activeCarousel}})],1)]):a("div",{staticClass:"disease-loading"},[a("b-spinner",{attrs:{label:"Loading...",variant:"light"}})],1)]):e._e()])},u=[],h=a("a34a"),p=a.n(h),v=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"histo-pheno"})},m=[],g=a("ba30"),f=a.n(g),b={props:{activeItem:{type:Object,required:!0},colorScheme:{type:String,default:"light"},toolbar:{type:Boolean,default:!1}},data:function(){return{histoPheno:null}},watch:{activeItem:{handler:"updateHistoPheno"}},mounted:function(){this.updateHistoPheno()},methods:{generateHistoPheno:function(e){this.histoPheno=new f.a(document.querySelector(".histo-pheno"),e),this.histoPheno.render()},refreshHistoPheno:function(e){this.histoPheno.updateOptions(e)},getChartOptions:function(e,t){var a="#FFF",n="light";return"dark"===this.colorScheme&&(a="#868686",n="dark"),{series:[{data:e}],colors:["#CCE34C"],chart:{type:"bar",redrawOnParentResize:!0,height:"100%",width:"100%",toolbar:{show:this.toolbar}},plotOptions:{bar:{horizontal:!0}},stroke:{width:0,colors:[a]},xaxis:{showForNullSeries:!1,categories:t,labels:{style:{colors:[a],cssClass:n}},title:{text:"# of Phenotypes",style:{color:a,cssClass:n}},lines:{show:!1}},dataLabels:{enabled:!1},yaxis:{showForNullSeries:!1,title:{text:void 0},labels:{style:{colors:[a],cssClass:n}}},tooltip:{enabled:!1},fill:{opacity:1},grid:{xaxis:{lines:{show:!1}},yaxis:{lines:{show:!1}}}}},updateHistoPheno:function(){var e=this.activeItem.categories;Object.keys(e).forEach((function(t){return 0===e[t]&&delete e[t]})),e=Object.entries(e),e=e.sort((function(e,t){return t[1]-e[1]})),e=Object.fromEntries(e);var t=Object.keys(e),a=Object.values(e),n=this.getChartOptions(a,t);this.histoPheno?this.refreshHistoPheno(n):this.generateHistoPheno(n)}}},_=b,y=(a("6391"),a("2877")),w=Object(y["a"])(_,v,m,!1,null,null,null),C=w.exports,M=a("63e8"),S=a("c1be");function D(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function O(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){D(o,n,r,i,s,"next",e)}function s(e){D(o,n,r,i,s,"throw",e)}i(void 0)}))}}var I={components:{"histo-pheno":C},data:function(){return{carouselData:[],activeCarousel:{},featureDate:""}},mounted:function(){var e=this;return O(p.a.mark((function t(){var a;return p.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:return t.next=2,Object(M["a"])();case 2:return a=t.sent,t.next=5,Promise.all(a.map(function(){var t=O(p.a.mark((function t(a){var n,r;return p.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:return t.next=2,Object(S["j"])(a.mondo);case 2:return n=t.sent,t.next=5,Object(S["d"])(a.mondo);case 5:r=t.sent,Object.values(n).every((function(e){return 0===e}))||e.carouselData.push({categories:n,label:r.label,id:a.mondo,description:r.description,date:a.date});case 7:case"end":return t.stop()}}),t)})));return function(e){return t.apply(this,arguments)}}()));case 5:e.carouselData.length>0&&(e.setDate(e.carouselData[0].date),e.changeActiveCarouselItem(),setInterval(e.changeActiveCarouselItem,6e3));case 6:case"end":return t.stop()}}),t)})))()},methods:{changeActiveCarouselItem:function(){var e=this.carouselData.map((function(e){return e.id})).indexOf(this.activeCarousel.id),t=this.carouselData.length-1,a=e+1;this.activeCarousel=e===t||-1===e?this.carouselData[0]:this.carouselData[a]},isEmpty:function(){return 0===Object.keys(this.activeCarousel).length},setDate:function(e){e=new Date(e);var t=["January","February","March","April","May","June","July","August","September","October","November","December"],a=t[e.getMonth()],n=e.getFullYear();this.featureDate="- "+a+" "+n}}},x=I,k=(a("a9b9"),Object(y["a"])(x,d,u,!1,null,"7dd8bf56",null)),A=k.exports,E={name:"Home",components:{"vml-home-intro-component":a("98eb").default,"vml-home-about-monarch-component":a("5bdf").default,"vml-home-news-component":a("6246").default,"vml-home-partners-component":a("a355").default,"disease-carousel":A}},T=E,R=(a("21bb"),Object(y["a"])(T,l,c,!1,null,null,null)),N=R.exports,P=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"node-wrapper"},[e.node?a("node-sidebar",{ref:"sidebar",attrs:{"node-type":e.nodeType,"node-label":e.node.label,"is-group":e.isGroup,"expanded-card":e.expandedCard,"available-cards":e.availableCards,"cards-to-display":e.nonEmptyCards,"card-counts":e.counts,"parent-node":e.node,"parent-node-id":e.nodeId,"is-neighborhood-showing":e.isNeighborhoodShowing,subclasses:e.subclasses,superclasses:e.superclasses},on:{"expand-card":e.expandCard,"toggle-neighborhood":e.toggleNeighborhood}}):e._e(),e.node?e._e():a("div",{staticClass:"loading"},[e.nodeError?a("div",[a("small",[a("h6",[e._v("Error loading "+e._s(e.labels[e.nodeType])+":  "+e._s(e.nodeId))]),a("div",[e._v(e._s(e.nodeError))])])]):a("div",{staticClass:"text-center spinner-wrapper"},[a("b-spinner",{staticClass:"loading-spinner",attrs:{type:"grow",label:"Spinning"}}),a("h5",{staticClass:"text-center"},[e._v(e._s(e.nodeId))])],1)]),e.node?a("div",{staticClass:"container-cards"},[a("div",{staticClass:"title-bar-wrapper"},[a("div",{staticClass:"overlay",class:{active:e.isNeighborhoodShowing}}),a("div",{ref:"titleBar",staticClass:"title-bar"},[a("h4",{staticClass:"node-label-label"},[a("span",{domProps:{innerHTML:e._s(e.node.label)}}),e._v(" "),a("span",{staticClass:"node-label-id"},[e._v(e._s(e.node.id))]),e.originalId?a("span",{staticClass:"node-label-id"},[e._v(" (Redirected from "+e._s(e.originalId)+") ")]):e._e(),a("br"),e.synonyms["Exact Synonym"].length>1?a("span",{staticClass:"node-label-synonyms"},[e._v(" "+e._s(e.synonyms["Exact Synonym"].join(", ")))]):e._e()]),e._v("   ")])]),e.node?a("div",{staticClass:"container-fluid node-container"},[a("div",{staticClass:"row"},[!e.expandedCard&&e.overviewSection()?a("div",{staticClass:"node-content-section col-12 col-md-6"},[a("div",{staticClass:"node-content-section-content"},[a("h5",[e._v("Overview")]),e.node.description?a("div",{staticClass:"node-sub-section"},[a("div",{staticClass:"node-description"},[a("div",{staticClass:"description"},[a("div",{domProps:{innerHTML:e._s(e.node.description)}})])])]):e._e(),e.entrezResult?a("div",{staticClass:"node-sub-section"},[a("h6",[e._v(e._s(e.entrezResult.pubdate))]),a("h6",[e._v(" "+e._s(e.entrezResult.authors.map((function(e){return e.name})).join(", "))+" ")]),a("h6",{staticClass:"publication-abstract"},[e._v(" "+e._s(e.entrezResult.abstract)+" ")]),a("h6",[e._v("DOI: "+e._s(e.entrezResult.doi))]),a("br"),a("b-button",{attrs:{href:e.entrezResult.pubmedURL,target:"__blank",variant:"outline-primary"}},[e._v(" View on PubMed ")])],1):e._e(),e.inheritance||e.modifiers?a("div",[a("h5",[e._v("Key Features")]),e.inheritance?a("div",{staticClass:"node-sub-section"},[a("b",[e._v("Heritability: ")]),e._v(e._s(e.inheritance)+" ")]):e._e(),e.modifiers?a("div",{staticClass:"node-sub-section"},[a("b",[e._v("Clinical Modifiers: ")]),e._v(" "+e._s(e.modifiers)+" ")]):e._e()]):e._e()])]):e._e(),!e.expandedCard&&e.supportSection()?a("div",{staticClass:"node-content-section col-12 col-md-6"},[a("div",{staticClass:"node-content-section-content"},[a("h5",[e._v("External Resources")]),a("div",{staticClass:"node-sub-section"},[a("div",{staticClass:"linked-references"},["disease"==e.nodeType?a("div",[e.authoritiveXref.synopsis&&e.authoritiveXref.synopsis.label?a("h6",{staticClass:"resource-section"},[a("strong",[e._v("Clinical Synopsis")]),e._v(": "),a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.synopsis.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.synopsis.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])],1):e._e(),e.authoritiveXref.patient&&e.authoritiveXref.patient.label?a("h6",{staticClass:"resource-section"},[a("strong",[e._v("Clinical Information for patients")]),e._v(": "),a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.patient.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.patient.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])],1):e._e()]):e._e(),"gene"==e.nodeType?a("div",[e.authoritiveXref.gene.taxon&&e.authoritiveXref.gene.taxon.label?a("div",{staticClass:"resource-section"},[a("strong",[e._v("Gene Summaries")]),e._v(": "),a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.gene.taxon.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.gene.taxon.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})]),e.authoritiveXref.gene.ensembl?a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.gene.ensembl.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.gene.ensembl.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})]):e._e()],1):e._e(),e.authoritiveXref.gene&&e.authoritiveXref.gene.clinical?a("h6",{staticClass:"resource-section"},[a("strong",[e._v("Clinical Information")]),e._v(": "),e.authoritiveXref.gene&&e.authoritiveXref.gene.clinical.omim?a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.gene.clinical.omim.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.gene.clinical.omim.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})]):e._e(),e.authoritiveXref.gene&&e.authoritiveXref.gene.clinical.varsome?a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.gene.clinical.varsome.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.gene.clinical.varsome.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})]):e._e()],1):e._e(),e.authoritiveXref.gene&&e.authoritiveXref.gene.pathway?a("h6",{staticClass:"resource-section"},[a("strong",[e._v("Pathway Analysis")]),e._v(": "),a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.gene.pathway.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.gene.pathway.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])],1):e._e()]):e._e(),"phenotype"===e.nodeType?a("div",[e.authoritiveXref.phenotype&&e.authoritiveXref.phenotype.label?a("h6",{staticClass:"resource-section"},[a("strong",[e._v("Ontology Browser")]),e._v(": "),a("b-button",{attrs:{size:"sm",variant:"outline-info",href:e.authoritiveXref.phenotype.uri,target:"_blank"}},[e._v(" "+e._s(e.authoritiveXref.phenotype.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])],1):e._e()]):e._e()]),e.references.linked.length>0?a("div",{staticClass:"linked-references"},["disease"===e.nodeType?a("h6",{directives:[{name:"b-toggle",rawName:"v-b-toggle.collapse-1",modifiers:{"collapse-1":!0}}],staticClass:"heading-toggle"},[e._m(0)]):a("h6",{directives:[{name:"b-toggle",rawName:"v-b-toggle.collapse-1",modifiers:{"collapse-1":!0}}],staticClass:"heading-toggle"},[e._m(1)]),a("b-collapse",{attrs:{id:"collapse-1"}},e._l(e.references.linked,(function(t,n){return a("span",{key:n,staticClass:"synonym"},[a("span",{staticClass:"reference-external"},[a("a",{staticClass:"node-label-id",attrs:{href:t.uri,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.id)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])])])})),0)],1):e._e()])])]):e._e(),!e.expandedCard&&e.hasGeneExac&&e.showExac?a("div",{staticClass:"node-content-section col-12 col-md-6"},[a("div",{staticClass:"node-content-section-content"},[a("h5",[e._v("ExAC Population Frequencies")]),a("exac-gene",{attrs:{"node-id":e.nodeId},on:{"show-exac":function(t){return e.showExacSection(t)}}})],1)]):e._e(),!e.expandedCard&&e.node.geneInfo&&e.node.geneInfo.externalURL?a("div",{staticClass:"node-content-section col-12 col-md-8"},[a("div",{staticClass:"node-content-section-content"},[a("h5",[e._v("Genome Features")]),a("genome-feature",{attrs:{"mygene-data":e.node.geneInfo}})],1)]):e._e(),!e.expandedCard&&e.histoPhenoData.categories&&e.counts["phenotype"]>0?a("div",{staticClass:"node-content-section col-12 col-md-6",on:{click:function(t){return e.expandCard("phenotype")}}},[a("div",{staticClass:"node-content-section-content associated-phenotypes"},[a("h5",[e._v("Associated Phenotypes")]),a("div",{staticClass:"histo-pheno-wrapper"},[a("histo-pheno",{attrs:{"active-item":e.histoPhenoData,"color-scheme":"dark"}})],1)])]):e._e()]),!e.expandedCard&&e.reactomeId?a("div",{staticClass:"row py-0"},[a("reactome-viewer",{attrs:{"reactome-id":e.reactomeId}})],1):e._e(),e.expandedCard||"variant"!==e.nodeType?e._e():a("div",[a("exac-variant",{attrs:{"node-id":e.nodeId}})],1),e.expandedCard?a("div",{staticClass:"expanded-card-view node-content-section"},[a("div",{staticClass:"node-content-section-content"},[a("assoc-table",{attrs:{"taxon-counts":e.taxonCounts,"node-type":e.nodeType,"card-type":e.expandedCard,"node-id":e.nodeId,"node-label":e.node.label,"is-group":e.isGroup,"card-counts":e.counts}})],1)]):e._e()]):e._e()]):e._e(),a("script",{attrs:{type:"application/ld+json"},domProps:{innerHTML:e._s(e.jsonld)}})],1)},H=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("strong",[a("i",{staticClass:"fa fa-angle-right",attrs:{"aria-hidden":"true"}}),a("i",{staticClass:"fa fa-angle-down",attrs:{"aria-hidden":"true"}}),e._v(" Other Mappings ")])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("strong",[a("i",{staticClass:"fa fa-angle-right",attrs:{"aria-hidden":"true"}}),a("i",{staticClass:"fa fa-angle-down",attrs:{"aria-hidden":"true"}}),e._v(" Other External Resources ")])}],L=a("c46f"),G=a("bc3a"),B=a.n(G),F=a("09b1");function J(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function j(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){J(o,n,r,i,s,"next",e)}function s(e){J(o,n,r,i,s,"throw",e)}i(void 0)}))}}function U(e){return W.apply(this,arguments)}function W(){return W=j(p.a.mark((function e(t){var a,n,r,o,i,s,l,c,d,u,h,v,m,g,f,b,_,y,w,C,M,S;return p.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return a="https://mygene.info/v3/query",n="",r="all",t.match(/^NCBIGene/)?n=t.replace(/\S+:(\d+)/,"$1"):t.match(/^OMIM/)?(n=t.replace(/\S+:(\d+)/,"mim:$1"),r=9606):t.match(/^MGI/)?(n=t.replace(/\S+:(\d+)/,"mgi:MGI\\:$1"),r=10090):t.match(/^FlyBase/)?(n=t.replace(/\S+:(\d+)/,"flybase:$1"),r=7227):t.match(/^Wormbase/)?(n=t.replace(/\S+:(\d+)/,"wormbase:$1"),r=6239):t.match(/^ZFIN/)?(n=t.replace(/\S+:(\d+)/,"zfin:$1"),r=7955):t.match(/^RGD/)?(n=t.replace(/\S+:(\d+)/,"rgd:$1"),r=10116):n=t,o={q:n,fields:"summary,genomic_pos,name,symbol,taxid",species:r},e.prev=5,e.next=8,B.a.get(a,{params:o});case 8:if(s=e.sent,i=s.data,!(i.hits.length>0)){e.next=41;break}if(l=i.hits[0],c=l.symbol,l.genomic_pos){e.next=17;break}i.hits=[],e.next=41;break;case 17:if(d=l.genomic_pos,Array.isArray(d)&&(u=d[0],h=u.start,v=u.end,h>v?i.hits=[]:(m=!1,d.forEach((function(e){h>e.start&&(h=e.start),v<e.end&&(v=e.end),u.chr!==e.chr&&(m=!0)})),m?i.hits=[]:(u.start=h,u.end=v,d=u,l.genomic_pos=d))),g=l.taxid,g||(g=d.taxid),Object(F["b"])(g)){e.next=24;break}e.next=41;break;case 24:return f=Object(F["a"])("NCBITaxon:".concat(g)),b="All Genes",_=d.chr+":"+d.start+".."+d.end,y="https://agr-apollo.berkeleybop.io/apollo/",w="http://www.alliancegenome.org/jbrowse/index.html?data=data/"+encodeURI(f)+"&loc="+encodeURI(_)+"&tracks="+b,C=y+"track/"+encodeURI(f)+"/"+b+"/"+encodeURI(_)+".json",M=C+"?name="+c,e.prev=31,e.next=34,B.a.get(M);case 34:S=e.sent,i.trackResponse=S.data,i.externalURL=w,e.next=41;break;case 39:e.prev=39,e.t0=e["catch"](31);case 41:e.next=45;break;case 43:e.prev=43,e.t1=e["catch"](5);case 45:return e.abrupt("return",i);case 46:case"end":return e.stop()}}),e,null,[[5,43],[31,39]])}))),W.apply(this,arguments)}function z(e,t){var a="undefined"!==typeof Symbol&&e[Symbol.iterator]||e["@@iterator"];if(!a){if(Array.isArray(e)||(a=K(e))||t&&e&&"number"===typeof e.length){a&&(e=a);var n=0,r=function(){};return{s:r,n:function(){return n>=e.length?{done:!0}:{done:!1,value:e[n++]}},e:function(e){throw e},f:r}}throw new TypeError("Invalid attempt to iterate non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.")}var o,i=!0,s=!1;return{s:function(){a=a.call(e)},n:function(){var e=a.next();return i=e.done,e},e:function(e){s=!0,o=e},f:function(){try{i||null==a.return||a.return()}finally{if(s)throw o}}}}function K(e,t){if(e){if("string"===typeof e)return q(e,t);var a=Object.prototype.toString.call(e).slice(8,-1);return"Object"===a&&e.constructor&&(a=e.constructor.name),"Map"===a||"Set"===a?Array.from(e):"Arguments"===a||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(a)?q(e,t):void 0}}function q(e,t){(null==t||t>e.length)&&(t=e.length);for(var a=0,n=new Array(t);a<t;a++)n[a]=e[a];return n}function V(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function $(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){V(o,n,r,i,s,"next",e)}function s(e){V(o,n,r,i,s,"throw",e)}i(void 0)}))}}function X(e){return Y.apply(this,arguments)}function Y(){return Y=$(p.a.mark((function e(t){var a,n,r,o,i,s;return p.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:if(a=/^PMID:(\d+)$/,t=t.map((function(e){return(a.exec(e)||[])[1]})),t.length){e.next=4;break}return e.abrupt("return",null);case 4:return n="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?",r={db:"pubmed",retmode:"json",id:t.join(",")},o="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?",i={db:"pubmed",retmode:"xml",rettype:"abstract",id:t.join(",")},s=[],e.next=11,B.a.all([B.a.get(n,{params:r}),B.a.get(o,{params:i})]).then(B.a.spread((function(e,a){var n,r=Array.from((new DOMParser).parseFromString((null===a||void 0===a?void 0:a.data)||"","text/xml").getElementsByTagName("PubmedArticle")),o=z(t);try{var i=function(){var t,o,i=n.value;if(!i)return s.push({}),"continue";var l=((null===e||void 0===e||null===(t=e.data)||void 0===t?void 0:t.result)||[])[i]||{},c=r.find((function(e){var t,a;return null===e||void 0===e||null===(t=e.getElementsByTagName("PMID")[0])||void 0===t||null===(a=t.textContent)||void 0===a?void 0:a.includes(i)})),d=null===(o=(l.articleids||[]).find((function(e){var t=e.idtype;return"doi"===t})))||void 0===o?void 0:o.value,u=(Array.from((null===c||void 0===c?void 0:c.getElementsByTagName("ArticleId"))||[]).find((function(e){return"doi"===(null===e||void 0===e?void 0:e.getAttribute("IdType"))}))||{}).textContent;l.doi=d||u||"";var h=Array.from((null===c||void 0===c?void 0:c.getElementsByTagName("AbstractText"))||[]).map((function(e){return null===e||void 0===e?void 0:e.textContent})).join("\n\n");l.abstract=h,l.pubmedURL="https://www.ncbi.nlm.nih.gov/pubmed/".concat(i),l.summaryURL=e.request.responseURL,l.abstractURL=a.request.responseURL,s.push(l)};for(o.s();!(n=o.n()).done;)i()}catch(l){o.e(l)}finally{o.f()}})));case 11:return e.abrupt("return",s);case 12:case"end":return e.stop()}}),e)}))),Y.apply(this,arguments)}var Z=a("b279"),Q=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{directives:[{name:"click-outside",rawName:"v-click-outside",value:e.hideNeighborhoodOrFacets,expression:"hideNeighborhoodOrFacets"}],staticClass:"node-sidebar"},[a("node-sidebar-neighborhood",{attrs:{"is-visible":e.isNeighborhoodShowing,"node-type":e.nodeType,"node-label":e.nodeLabel,subclasses:e.subclasses,superclasses:e.superclasses},on:{toggleNeighborhood:function(t){return e.toggleNeighborhood()}}}),a("div",[e.nodeType?a("ul",{staticClass:"list-group"},[a("li",{staticClass:"list-group-item title"},[e._v(" "+e._s(e.$parent.labels[e.nodeType])+" ")]),a("li",{staticClass:"list-group-item list-group-item-squat",class:{active:!e.expandedCard}},[a("b-link",{on:{click:function(t){return e.expandCard(null)}}},[a("i",{staticClass:"fa fa-fw fa-th-large"}),a("span",{staticClass:"list-group-item-value"},[e._v("Overview")])])],1),a("li",{staticClass:"list-group-item list-group-item-squat"},[a("b-link",{attrs:{disabled:e.neighborhoodDisabled},on:{click:function(t){return e.toggleNeighborhood()}}},[a("i",{staticClass:"fa fa-fw fa-share-alt neighbors"}),a("span",{staticClass:"list-group-item-value"},[e._v("Neighbors")])])],1),e._l(e.cardsToDisplay,(function(t){return a("li",{key:t,staticClass:"list-group-item",class:{active:e.expandedCard===t}},[a("a",{attrs:{href:"#"+t},on:{click:function(a){return e.expandCard(t)}}},[a("img",{staticClass:"entity-type-icon",attrs:{src:e.$parent.icons[t]}}),a("span",{staticClass:"list-group-item-value"},[e._v(" "+e._s(e.$parent.labels[t])+" "),a("span",{staticClass:"count"},[e._v(e._s(e.cardCounts[t]))])])])])}))],2):e._e()])],1)},ee=[],te=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("nav",{class:{active:e.isVisible},attrs:{id:"neighborhood"}},[a("i",{staticClass:"close fa fa-times",attrs:{"aria-hidden":"true"},on:{click:function(t){return e.closeNeighborhood()}}}),a("div",{staticClass:"neighborhood"},[e._l(e.superclasses,(function(t){return a("div",{key:t.id,staticClass:"row superclass"},[a("div",{staticClass:"col-12"},[a("router-link",{attrs:{to:"/"+e.nodeType+"/"+t.id}},[e._v(" "+e._s(e.getLabel(t))+" ")])],1)])})),a("div",{staticClass:"row currentclass"},[a("div",{staticClass:"col-12"},[e._v(" "+e._s(e.nodeLabel)+" "),a("i",{staticClass:"fa fa-flag",staticStyle:{float:"right"},attrs:{"aria-hidden":"true"}})])]),e._l(e.subclasses,(function(t){return a("div",{key:t.id,staticClass:"row subclass"},[a("div",{staticClass:"col-12"},[a("router-link",{attrs:{to:"/"+e.nodeType+"/"+t.id}},[e._v(" "+e._s(e.getLabel(t))+" ")])],1)])}))],2)])},ae=[],ne={name:"NodeSidebarNeighborhood",props:{nodeType:{type:String,required:!1,default:""},nodeLabel:{type:String,required:!1,default:""},superclasses:{type:Array,required:!1,default:null},subclasses:{type:Array,required:!1,default:null},isVisible:{type:Boolean,required:!1,default:!1}},methods:{getLabel:function(e){return e.label||e.id},closeNeighborhood:function(){this.$emit("toggleNeighborhood")}}},re=ne,oe=(a("6271"),Object(y["a"])(re,te,ae,!1,null,null,null)),ie=oe.exports,se=a("c28b"),le=a.n(se),ce={name:"NodeSidebar",components:{"node-sidebar-neighborhood":ie},directives:{clickOutside:le.a.directive},props:{cardsToDisplay:{type:Array,required:!1,default:null},expandedCard:{type:String,required:!1,default:""},cardCounts:{type:Object,required:!1,default:null},nodeType:{type:String,required:!1,default:""},nodeLabel:{type:String,required:!1,default:""},superclasses:{type:Array,required:!1,default:null},subclasses:{type:Array,required:!1,default:null},isNeighborhoodShowing:{type:Boolean,required:!1,default:!1},isGroup:{type:Boolean,required:!1,default:!1}},data:function(){return{}},computed:{neighborhoodDisabled:function(){return(!this.superclasses||0===this.superclasses.length)&&(!this.subclasses||0===this.subclasses.length)},facetsDisabled:function(){return!1}},created:function(){},updated:function(){},unmounted:function(){},mounted:function(){},methods:{expandCard:function(e){this.$emit("expand-card",e)},toggleNeighborhood:function(){this.$emit("toggle-neighborhood")},hideNeighborhoodOrFacets:function(){this.isNeighborhoodShowing&&this.toggleNeighborhood()}}},de=ce,ue=(a("6df1"),Object(y["a"])(de,Q,ee,!1,null,null,null)),he=ue.exports,pe=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"assoc-table"},[e.dataError?a("div",{staticClass:"border p-2 m-2"},[a("h3",[e._v("BioLink Error")]),a("div",{staticClass:"col-xs-12"},[e._v(" "+e._s(e.dataError)+" ")])]):e._e(),a("div",{directives:[{name:"show",rawName:"v-show",value:!e.initialLoad&&!e.dataError,expression:"!initialLoad && !dataError"}],staticClass:"loading-div"},[a("b-spinner",{staticClass:"loading-spinner",attrs:{type:"grow",label:"Spinning"}})],1),a("div",{directives:[{name:"show",rawName:"v-show",value:!e.dataError&&e.initialLoad,expression:"!dataError && initialLoad"}]},[a("taxon-filter",{attrs:{"is-visible":e.isTaxonShowing},on:{"toggle-filter":function(t){return e.toggleTaxonFilter(t)}},model:{value:e.taxonFilter,callback:function(t){e.taxonFilter=t},expression:"taxonFilter"}}),a("div",[a("h5",["phenotype"===e.cardType&&e.totalAssociations>=0||e.totalAssociations>0?a("strong",[e._v(e._s(e.totalAssociations))]):e._e(),e._v(" "+e._s(e.cardType)+" associations. ")]),"gene"===e.nodeType&&e.hasTaxon?a("b-button",{staticClass:"taxon-filter",attrs:{size:"sm",variant:"primary"},on:{click:function(t){return e.toggleTaxonFilter()}}},[a("i",{staticClass:"fa fa-filter",class:{"filter-active":e.hasFalseFilter()},attrs:{"aria-hidden":"true"}}),e._v(" Taxon Filter ")]):e._e(),a("br")],1),"phenotype"===e.cardType&&0==e.totalAssociations?a("div",["gene"===e.nodeType?[e._v(" We are not aware of any authoritative sources of observed human phenotypes "),0==e.altDisplayGene?a("span",[e._v(" or diseases ")]):e._e(),e._v(" for this gene. "),e.altDisplayGene>0?a("span",[e._v(" However, we encourage you to view the available ")]):e._e(),1==e.altDisplayGene?a("span",[a("a",{attrs:{href:this.$route.path+"#ortholog-phenotype"}},[e._v(e._s(e.cardCounts["ortholog-phenotype"])+" ortholog phenotype(s)")]),e._v(" in other species. ")]):e._e(),2==e.altDisplayGene?a("span",[a("a",{attrs:{href:this.$route.path+"#causal-disease"}},[e._v(e._s(e.cardCounts["causal-disease"])+" associated disease(s)")]),e._v(". ")]):e._e(),3==e.altDisplayGene?a("span",[a("a",{attrs:{href:this.$route.path+"#correlated-disease"}},[e._v(e._s(e.cardCounts["correlated-disease"])+" associated disease(s)")]),e._v(". ")]):e._e(),4==e.altDisplayGene?a("span",[a("a",{attrs:{href:this.$route.path+"#ortholog-disease"}},[e._v(e._s(e.cardCounts["ortholog-disease"])+" associated disease(s)")]),e._v(". ")]):e._e()]:"disease"===e.nodeType?[e._v(" We are not aware of any publicly available source data for phenotypes "),0==e.altDisplayDisease?a("span",[e._v(" or genes ")]):e._e(),e._v(" for this disease. "),e.altDisplayDisease>0?a("span",[e._v(" However, we encourage you to view the available ")]):e._e(),1==e.altDisplayDisease?a("span",[a("a",{attrs:{href:this.$route.path+"#causal-gene"}},[e._v(e._s(e.cardCounts["causal-gene"])+" causal gene(s)")]),e._v(". ")]):e._e(),2==e.altDisplayDisease?a("span",[a("a",{attrs:{href:this.$route.path+"#correlated-gene"}},[e._v(e._s(e.cardCounts["correlated-gene"])+" correlated gene(s)")]),e._v(". ")]):e._e(),3==e.altDisplayDisease?a("span",[a("a",{attrs:{href:this.$route.path+"#gene"}},[e._v(e._s(e.cardCounts["gene"])+" associated gene(s)")]),e._v(". ")]):e._e()]:"variant"===e.nodeType?[e._v(" We are not aware of any phenotypes "),0==e.cardCounts["disease"]?a("span",[e._v(" or diseases ")]):e._e(),e._v(" specifically associated with this variant. "),e.cardCounts["disease"]>0?a("span",[e._v(" However, we encourage you to view the available "),a("a",{attrs:{href:this.$route.path+"#disease"}},[e._v(e._s(e.cardCounts["disease"])+" associated disease(s)")]),e._v(". ")]):e._e()]:e._e(),e._v(" We are always improving our knowledgebase; to suggest a new source, "),a("a",{attrs:{href:"https://github.com/monarch-initiative/helpdesk/issues",target:"_blank"}},[e._v("please submit a ticket")]),e._v(". ")],2):e._e(),a("div",{directives:[{name:"show",rawName:"v-show",value:!("phenotype"===e.cardType&&0==e.totalAssociations),expression:"!(cardType === 'phenotype' && totalAssociations == 0)"}]},[a("b-table",{ref:"tableRef",staticClass:"table-sm",attrs:{items:e.rowsProvider,busy:e.tableBusy,fields:e.fields,"current-page":e.currentPage,"per-page":e.rowsPerPage,responsive:"true"},scopedSlots:e._u([e.hasTaxon?{key:"cell(taxon)",fn:function(t){return[a("i",[e._v(e._s(t.item.taxonLabel))])]}}:null,{key:"cell(relation)",fn:function(t){return[a("small",[a("a",{attrs:{href:t.item.relation.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.item.relation.label)+" ")])])]}},{key:"cell(assocObject)",fn:function(t){return[t.item.objectLink?[a("strong",[a("router-link",{attrs:{to:t.item.objectLink},domProps:{innerHTML:e._s(e.$sanitizeText(t.item.assocObject))}})],1)]:[a("strong",[e._v(" "+e._s(t.item.assocObject)+" ")])]]}},e.isGroup||e.cardType.includes("ortholog")?{key:"cell(assocSubject)",fn:function(t){return[t.item.subjectLink?[a("strong",[a("router-link",{attrs:{to:t.item.subjectLink},domProps:{innerHTML:e._s(e.$sanitizeText(t.item.assocSubject))}},[e._v(" "+e._s(t.item.assocSubject)+" ")])],1)]:[a("strong",[e._v(" "+e._s(t.item.assocSubject)+" ")])]]}}:null,e.hasFrequencyOnset?{key:"cell(frequency)",fn:function(t){return[t.item.frequency&&e.nodeId===t.item.subjectCurie?a("a",{attrs:{href:t.item.frequency.url,target:"_blank",rel:"noopener noreferrer"}},[a("small",[e._v(" "+e._s(t.item.frequency.label)+" ")])]):e._e()]}}:null,e.hasFrequencyOnset?{key:"cell(onset)",fn:function(t){return[t.item.onset&&e.nodeId===t.item.subjectCurie?a("a",{attrs:{href:t.item.onset.url,target:"_blank",rel:"noopener noreferrer"}},[a("small",[e._v(" "+e._s(t.item.onset.label)+" ")])]):e._e()]}}:null,{key:"cell(support)",fn:function(t){return[a("b-button",{staticClass:"btn btn-xs px-1 py-0 m-0",attrs:{pressed:t.item._showDetails,size:"small",variant:"outline-info"},on:{click:function(e){t.item._showDetails=!t.item._showDetails}}},[t.item._showDetails?a("span",[e._v(" ▾  ")]):a("span",[e._v(" ▸  ")]),e._l(t.item.supportIcons,(function(e,t){return a("span",{key:t},[a("i",{staticClass:"fa fa-fw",class:e})])})),a("small",[e._v(e._s(t.item.supportLength))])],2)]}},{key:"row-details",fn:function(t){return[a("EvidenceViewer",{attrs:{evidence:t.item.evidence,"evidence-cache":e.evidenceCache,"node-id":e.nodeId,"node-label":e.nodeLabel,"node-type":e.nodeType,"subject-id":t.item.subjectCurie,"subject-label":t.item.assocSubject,"card-type":e.cardType}})]}}],null,!0)}),e.totalAssociations>e.rowsPerPage?a("b-pagination",{staticClass:"pag-width my-1",attrs:{"per-page":e.rowsPerPage,"total-rows":e.totalAssociations,align:"center",size:"md"},model:{value:e.currentPage,callback:function(t){e.currentPage=t},expression:"currentPage"}}):e._e()],1)],1)])},ve=[],me=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"container-fluid evidence-section py-0"},[a("div",{staticClass:"summary"},[e._v(" Summary ("+e._s(e.evidence.evidence_types.length+e.evidence.publications.length+e.evidence.provided_by.length)+") ")]),e.nodeId!==e.subjectId?[a("i",{staticClass:"fa fa-fw fa-share-alt neighbors"}),e.cardType.includes("ortholog")?a("span",{staticClass:"subclass-of"},[e._v(" "+e._s(e.subjectLabel)+" is an ortholog of "+e._s(e.nodeLabel)+" ")]):a("span",{staticClass:"subclass-of"},[e._v(" "+e._s(e.subjectLabel)+" is a subclass of "+e._s(e.nodeLabel)+" ")])]:e._e(),e.evidence.evidence_types.length?[a("div",[a("b-button",{directives:[{name:"b-toggle",rawName:"v-b-toggle",value:"collapse-1"+e.evidence.id,expression:"'collapse-1' + evidence.id"}],staticClass:"btn btn-xs px-1 py-0 m-0 summary-btn",attrs:{size:"small",variant:"light"}},[a("span",{staticClass:"when-opened"},[e._v("▾ ")]),a("span",{staticClass:"when-closed"},[e._v("▸ ")]),a("i",{staticClass:"fa fa-fw fa-flask text-info"}),e._v(" Evidence Codes ("+e._s(e.evidence.evidence_types.length)+") ")]),a("b-collapse",{attrs:{id:"collapse-1"+e.evidence.id}},e._l(e.evidence.evidence_types,(function(t,n){return a("div",{key:n,staticClass:"row"},[a("a",{attrs:{href:t.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.label)+"  ")])])})),0)],1)]:e._e(),e.evidence.publications.length?[a("div",[a("b-button",{directives:[{name:"b-toggle",rawName:"v-b-toggle",value:"collapse-2"+e.evidence.id,expression:"'collapse-2' + evidence.id"}],staticClass:"btn btn-xs px-1 py-0 m-0 summary-btn",attrs:{size:"small",variant:"light"}},[a("span",{staticClass:"when-opened"},[e._v("▾ ")]),a("span",{staticClass:"when-closed"},[e._v("▸ ")]),a("i",{staticClass:"fa fa-fw fa-book text-info"}),e._v(" Publications ("+e._s(e.evidence.publications.length)+") ")]),a("b-collapse",{attrs:{id:"collapse-2"+e.evidence.id}},e._l(e.evidence.publications,(function(t,n){return a("div",{key:n,staticClass:"row"},[t.id.startsWith("PMID")?a("span",[a("a",{attrs:{href:t.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])]):a("span",[a("router-link",{attrs:{to:t.url}},[e._v(e._s(t.label))])],1)])})),0)],1)]:e._e(),e.evidence.provided_by.length?[a("div",[a("b-button",{directives:[{name:"b-toggle",rawName:"v-b-toggle",value:"collapse-3"+e.evidence.id,expression:"'collapse-3' + evidence.id"}],staticClass:"btn btn-xs px-1 py-0 m-0 summary-btn",attrs:{size:"small",variant:"light"}},[a("span",{staticClass:"when-opened"},[e._v("▾ ")]),a("span",{staticClass:"when-closed"},[e._v("▸ ")]),a("i",{staticClass:"fa fa-fw fa-database text-info"}),e._v(" Sources ("+e._s(e.evidence.provided_by.length)+") ")]),a("b-collapse",{attrs:{id:"collapse-3"+e.evidence.id}},[a("strong",[e._v(e._s(e.evidence.provided_by.join(", ")))])])],1)]:e._e(),e.evidenceError?a("div",{staticClass:"border p-2 m-2"},[e._v(" Error fetching evidence ")]):e._e(),a("div",{directives:[{name:"show",rawName:"v-show",value:!e.evidenceFetched&&!e.evidenceError,expression:"!evidenceFetched && !evidenceError"}],staticClass:"evidence-ajax-msg"},[a("b-spinner",{staticClass:"loading-spinner",attrs:{type:"grow",label:"Spinning"}}),a("span",[e._v("  Fetching Evidence")])],1),a("div",{directives:[{name:"show",rawName:"v-show",value:e.evidenceFetched&&!e.evidenceError,expression:"evidenceFetched && !evidenceError"}]},[a("div",{staticClass:"statements"},[e._v(" Supporting Statements ("+e._s(e.totalStatements)+") ")]),e.totalStatements>e.rowsPerPage?a("div",[a("b-pagination",{staticClass:"pag-width my-1",attrs:{"per-page":e.rowsPerPage,"total-rows":e.totalStatements,align:"left",size:"md"},model:{value:e.currentPage,callback:function(t){e.currentPage=t},expression:"currentPage"}})],1):e._e(),a("b-table",{staticClass:"table-sm",attrs:{items:e.rowsProvider,fields:e.fields,"current-page":e.currentPage,"per-page":e.rowsPerPage,"no-provider-paging":e.noProviderPaging,responsive:"true"},scopedSlots:e._u([{key:"cell(subject)",fn:function(t){return[t.item.subject.url?[a("router-link",{attrs:{to:t.item.subject.url},domProps:{innerHTML:e._s(e.$sanitizeText(t.item.subject.label))}})]:[a("span",{domProps:{innerHTML:e._s(e.$sanitizeText(t.item.subject.label))}})]]}},{key:"cell(relation)",fn:function(t){return[e._v(" "+e._s(t.item.relation.label)+" ")]}},{key:"cell(object)",fn:function(t){return[t.item.object.url?[a("router-link",{attrs:{to:t.item.object.url},domProps:{innerHTML:e._s(e.$sanitizeText(t.item.object.label))}})]:[a("span",{domProps:{innerHTML:e._s(e.$sanitizeText(t.item.object.label))}})]]}},{key:"cell(publications)",fn:function(t){return[t.item.publications.length<3?e._l(t.item.publications,(function(t,n){return a("div",{key:n,staticClass:"row"},[t.id.startsWith("PMID")?a("span",{staticClass:"nowrap"},[a("a",{attrs:{href:t.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])]):a("span",{staticClass:"nowrap"},[a("router-link",{attrs:{to:t.url}},[e._v(e._s(t.label))])],1)])})):[e._l(t.item.publications.slice(0,2),(function(t,n){return a("div",{key:n,staticClass:"row"},[t.id.startsWith("PMID")?a("span",{staticClass:"nowrap"},[a("a",{attrs:{href:t.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])]):a("span",{staticClass:"nowrap"},[a("router-link",{attrs:{to:t.url}},[e._v(e._s(t.label))])],1)])})),a("div",[a("b-collapse",{attrs:{id:"collapse-pubs"+e.evidence.id+t.item.rowNum}},e._l(t.item.publications.slice(2),(function(t,n){return a("div",{key:n,staticClass:"row final-row"},[t.id.startsWith("PMID")?a("span",{staticClass:"nowrap"},[a("a",{attrs:{href:t.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])]):a("span",{staticClass:"nowrap"},[a("router-link",{attrs:{to:t.url}},[e._v(e._s(t.label))])],1)])})),0),a("b-button",{directives:[{name:"b-toggle",rawName:"v-b-toggle",value:"collapse-pubs"+e.evidence.id+t.item.rowNum,expression:"'collapse-pubs' + evidence.id + data.item.rowNum"}],staticClass:"btn btn-xs px-1 py-0 m-0 pub-btn",attrs:{size:"sm",variant:"primary"}},[a("span",{staticClass:"when-closed"},[e._v("more...")]),a("span",{staticClass:"when-opened"},[e._v("less...")])])],1)]]}},{key:"cell(sources)",fn:function(t){return e._l(t.item.provided_by,(function(t,n){return a("div",{key:n,staticClass:"row"},[e._v(" "+e._s(t)+" ")])}))}},{key:"cell(references)",fn:function(t){return e._l(t.item.references,(function(t,n){return a("div",{key:n,staticClass:"row final-row"},[a("span",{staticClass:"nowrap"},[a("a",{attrs:{href:t.url,target:"_blank",rel:"noopener noreferrer"}},[e._v(" "+e._s(t.label)+" "),a("i",{staticClass:"fa fa-external-link",attrs:{"aria-hidden":"true"}})])])])}))}}])})],1)],2)},ge=[];function fe(e){var t="";return null!==e&&""!==e&&(t=e.toUpperCase(),e.match(/biogrid/i)?t="BioGRID":e.match(/clinvar/i)?t="ClinVar":e.match(/coriell/)?t="Coriell":e.match(/decipher/i)?t="Decipher":e.match(/ensembl/)?t="Ensembl":e.match(/fb|flybase/i)?t="FlyBase":e.match(/genereviews/i)?t="Gene Reviews":e.match(/hpo/)?t="HPO":e.match(/mygene/i)?t="MyGene":e.match(/orphanet/i)?t="Orphanet":e.match(/panther/i)?t="Panther":e.match(/pharmgkb/i)?t="PharmGKB":e.match(/uniprot/)?t="UniProt":e.match(/vega/i)?t="Vega":e.match(/wb|wormbase/i)?t="Wormbase":e.match(/gwascatalog/)?t="GWAS Catalog":e.match(/animalqtldb/)?t="AnimalQTLdb":e.match(/go/)?t="Gene Ontology":e.match(/gene reviews/)?t="Gene Reviews":e.match(/reactome/)?t="Reactome":e.match(/bgee/)&&(t="Bgee")),t}function be(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function _e(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){be(o,n,r,i,s,"next",e)}function s(e){be(o,n,r,i,s,"throw",e)}i(void 0)}))}}var ye={components:{},props:{evidence:{type:Object,required:!0},evidenceCache:{type:Object,required:!0},cardType:{type:String,required:!0},nodeId:{type:String,required:!0},nodeLabel:{type:String,required:!1,default:null},nodeType:{type:String,required:!1,default:null},subjectId:{type:String,required:!1,default:null},subjectLabel:{type:String,required:!1,default:null}},data:function(){return{currentPage:1,rowsPerPage:5,totalStatements:0,noProviderPaging:!0,evidenceFetched:!1,evidenceError:!1,rows:[],fields:[{key:"subject",label:"Subject",class:"subject"},{key:"relation",label:"Relation",class:"relation"},{key:"object",label:"Object",class:"object"},{key:"publications",label:"Publications",class:"publications"},{key:"sources",label:"Sources",class:"sources"},{key:"references",label:"References",class:"references"}]}},methods:{fetchEvidence:function(){var e=this;return _e(p.a.mark((function t(){var a,n;return p.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:if(a=e,a.evidence.id in a.evidenceCache||a.evidenceFetched){t.next=19;break}return a.evidenceError=!1,t.prev=3,t.next=6,Object(S["e"])(a.evidence.id,e.nodeType);case 6:if(n=t.sent,n.data&&n.data.associations){t.next=9;break}throw new Error("getEvidence() returned no data");case 9:a.evidenceFetched=!0,a.evidenceCache[a.evidence.id]=a.processEvidence(n.data.associations),a.$emit("evidenceCache",a.evidenceCache),t.next=17;break;case 14:t.prev=14,t.t0=t["catch"](3),a.evidenceError=t.t0;case 17:t.next=21;break;case 19:a.evidenceError=!1,a.evidenceFetched=!0;case 21:a.rows=a.evidenceCache[a.evidence.id],a.totalStatements=a.rows.length;case 23:case"end":return t.stop()}}),t,null,[[3,14]])})))()},rowsProvider:function(e,t){var a=this;this.fetchEvidence().then((function(){t(a.rows)})).catch((function(){t([])}))},processEvidence:function(e){var t=this,a=0;return e.forEach((function(e){[e.subject,e.object].forEach((function(e){e.label=e.label||e.id,e.label!==Object(Z["e"])(e.label)&&(e.label=Object(Z["d"])(e.label)),e.id.startsWith("BNODE")||e.id.startsWith("MONARCH")||e.id===t.nodeId?e.url=null:e.url="/".concat(e.id)})),e.publications=e.publications.filter((function(e){return!e.id.startsWith("MONDO")})).map((function(e){var t="";if(e.id.startsWith("PMID")){var a=e.id.split(":")[1];t="http://www.ncbi.nlm.nih.gov/pubmed/".concat(a)}else t="/publication/".concat(e.id);return{id:e.id,label:e.id,url:t}})),e.provided_by=Object(Z["c"])(e.provided_by);var n=[e.subject.id].concat(e.subject_eq),r=[e.object.id].concat(e.object_eq);e.references=[],e.provided_by.forEach((function(t){n.forEach((function(a){var n=Object(Z["a"])(t,a,e.subject.label);null!==n&&("impc"===t&&a.startsWith("MGI")&&"RO:0002200"===e.relation.id&&e.references.push({url:n,label:a.replace("MGI:","IMPC:")}),"impc"!==t&&e.references.push({url:n,label:a}))})),r.forEach((function(a){var n=Object(Z["a"])(t,a,e.object.label);null!==n&&("impc"===t&&a.startsWith("MGI")&&"GENO:0000408"===e.relation.id&&e.references.push({url:n,label:a.replace("MGI:","IMPC:")}),"impc"!==t&&e.references.push({url:n,label:a}))}))})),e.provided_by=e.provided_by.map((function(e){return fe(e)})),e.rowNum=++a})),e=e.sort((function(e,t){return t.provided_by.join().includes("OMIM")||t.provided_by.join().includes("Orphanet")?1:-1})),e=e.sort((function(e,t){return"has phenotype"!==e.relation.label&&"has phenotype"===t.relation.label?1:-1})),e}}},we=ye,Ce=(a("9e25"),Object(y["a"])(we,me,ge,!1,null,null,null)),Me=Ce.exports,Se=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"container-fluid",class:{active:e.isVisible},attrs:{id:"filter"}},[a("div",{staticClass:"top-row"},[a("b-button",{attrs:{size:"sm"},on:{click:e.toggleSelectedFilter}},[e._v(" Select All ")]),a("b-button",{staticClass:"exit",attrs:{size:"sm"},on:{click:e.hideFilter}},[e._v(" Apply & Close ")]),e.showMessage?a("p",{staticClass:"warning"},[e._v(" You must select at least 1 taxon. ")]):e._e()],1),e._l(e.taxonFilter.taxons,(function(t,n){return a("div",{key:n,staticClass:"facet-item row"},[a("div",{staticClass:"col-lg-10"},[a("b-form-checkbox",{on:{input:e.updateTaxonFilter},model:{value:e.taxonFilter.taxons[n],callback:function(t){e.$set(e.taxonFilter.taxons,n,t)},expression:"taxonFilter.taxons[key]"}},[a("i",[e._v(e._s(e.idToLabel(n)))]),e._v(" ("+e._s(n)+") ")])],1),a("div",{staticClass:"col-lg-2"},[e._v(" "+e._s(e.idToCount(n))+" ")])])}))],2)},De=[],Oe={name:"TaxonFilter",model:{prop:"taxonFilter"},props:{taxonFilter:{type:Object,required:!1,default:null},isVisible:{type:Boolean,required:!1,default:!1}},data:function(){return{selectedAll:!0,localCopy:{},showMessage:!1,initialDataFlag:!1}},watch:{isVisible:function(){this.localCopy=JSON.parse(JSON.stringify(this.taxonFilter))}},mounted:function(){},beforeUpdate:function(){this.initialDataFlag||(this.initialDataFlag=!0)},methods:{updateTaxonFilter:function(){this.$forceUpdate()},idToLabel:function(e){return Object(F["a"])(e)},idToCount:function(e){return this.taxonFilter.counts[e]},hideFilter:function(){var e=this;if(Object.values(this.taxonFilter.taxons).includes(!0)){this.showMessage=!1;var t=!1,a=Object.keys(this.taxonFilter.taxons),n=Object.keys(this.localCopy.taxons);n.length!==a.length?t=!0:a.forEach((function(a){e.taxonFilter.taxons[a]!==e.localCopy.taxons[a]&&(t=!0)})),t?this.$emit("toggle-filter",!0):this.$emit("toggle-filter",!1)}else this.showMessage=!0},toggleSelectedFilter:function(){var e=this;this.selectedAll=!this.selectedAll,Object.keys(this.taxonFilter.taxons).forEach((function(t){e.taxonFilter.taxons[t]=e.selectedAll})),this.updateTaxonFilter()}}},Ie=Oe,xe=(a("5f20"),Object(y["a"])(Ie,Se,De,!1,null,null,null)),ke=xe.exports;function Ae(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function Ee(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){Ae(o,n,r,i,s,"next",e)}function s(e){Ae(o,n,r,i,s,"throw",e)}i(void 0)}))}}var Te=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1?arguments[1]:void 0;return e.length>t?"".concat(e.substring(0,t-3),"..."):e},Re={components:{EvidenceViewer:Me,"taxon-filter":ke},props:{nodeId:{type:String,required:!0},nodeLabel:{type:String,required:!0},cardType:{type:String,required:!0},nodeType:{type:String,required:!0},taxonCounts:{type:Object,required:!0},isFacetsShowing:{type:Boolean,required:!1,default:!1},isGroup:{type:Boolean,required:!1,default:!1},cardCounts:{type:Object,required:!1,default:null}},data:function(){return{currentPage:1,rowsPerPage:15,totalAssociations:0,hasTaxon:!1,hasFrequencyOnset:!1,associationData:"",tableBusy:!1,dataError:!1,initialLoad:!1,fields:[],rows:[],lastSelection:[],evidenceCache:{},taxonFilter:{counts:{},taxons:{}},filterActive:null,isTaxonShowing:this.isFacetsShowing}},computed:{altDisplayGene:function(){var e=0;return this.cardCounts["ortholog-phenotype"]>0?e=1:this.cardCounts["causal-disease"]>0?e=2:this.cardCounts["correlated-disease"]>0?e=3:this.cardCounts["ortholog-disease"]>0&&(e=4),e},altDisplayDisease:function(){var e=0;return this.cardCounts["causal-gene"]>0?e=1:this.cardCounts["correlated-gene"]>0?e=2:this.cardCounts.gene>0&&(e=3),e}},watch:{cardType:function(){this.taxonFilter={counts:{},taxons:{}},this.initialLoad=!1,this.dataError=!1,this.isTaxonShowing=!1,window.scroll(0,0),this.currentPage=1,this.generateFields(),this.$refs.tableRef.refresh()}},mounted:function(){this.generateFields()},methods:{toggleTaxonFilter:function(e){this.isTaxonShowing&&e&&this.$refs.tableRef.refresh(),this.isTaxonShowing=!this.isTaxonShowing},hasFalseFilter:function(){var e=!1;return Object.entries(this.taxonFilter.taxons).forEach((function(t){t[1]||(e=!0)})),e},trueTaxonFilters:function(){var e=[];return Object.entries(this.taxonFilter.taxons).forEach((function(t){t[1]&&e.push(t[0])})),e},rowsProvider:function(e,t){var a=this;return Ee(p.a.mark((function e(){return p.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:a.fetchData().then((function(){t(a.rows)})).catch((function(){t([])}));case 1:case"end":return e.stop()}}),e)})))()},fetchData:function(e){var t=this;return Ee(p.a.mark((function a(){var n,r,o,i,s,l;return p.a.wrap((function(a){while(1)switch(a.prev=a.next){case 0:return n=t,t.tableBusy=!0,t.dataError=!1,a.prev=3,r={fetch_objects:!0,start:(t.currentPage-1)*t.rowsPerPage,rows:t.rowsPerPage},o=t.hasFalseFilter()?t.trueTaxonFilters():null,a.next=8,S["h"](t.nodeType,t.nodeId,t.cardType,o,r);case 8:if(i=a.sent,i.data&&i.data.associations){a.next=12;break}throw n.associationData=null,new Error("BioLink returned no data");case 12:if(n.associationData=i.data,"publication"!==t.cardType){a.next=19;break}return s=n.associationData.associations.map((function(e){var t=e.object;return t.id})),a.next=17,X(s);case 17:l=a.sent,n.associationData.associations.forEach((function(e,t){e.publicationMeta=l[t],console.log(e.publicationMeta)}));case 19:e&&(t.currentPage=1),n.populateRows(),n.tableBusy=!1,t.initialLoad=!0,a.next=28;break;case 25:a.prev=25,a.t0=a["catch"](3),n.dataError=a.t0;case 28:case"end":return a.stop()}}),a,null,[[3,25]])})))()},fixupRelation:function(e,t,a){var n=e.relation;if(n){var r=!1;!n.label&&n.id?n.label=n.id:n.label||(n.label="",n.id=""),n.label=n.label.replace(/_/g," "),"RO:0003301"===n.id?"model"===t&&(n.id="RO:0002615",n.label="has model"):"RO:0002615"===n.id?"model"!==t&&(n.id="RO:0003301",n.label="is model of"):"RO:0002206"===n.id?"gene"===t&&"anatomy"===a&&(n.id="RO:0002292",n.label="expresses"):"RO:0002607"===n.id?"gene"===t&&"disease"===a&&(r=!0):"RO:0003303"===n.id||"RO:0003304"===n.id?"disease"!==a&&"phenotype"!==a||(r=!0):"GENO:0000841"===n.id?"gene"===t&&"disease"===a&&(r=!0):"GENO:0000408"===n.id?"variant"===t&&"gene"===a&&(n.id="GENO:0000413",n.label="has allele"):"RO:0002331"===n.id?"gene"===t&&"pathway"===a&&(r=!0):"GENO:0000639"===n.id&&("variant"===t&&"gene"===a&&(r=!0),n.inverse=r)}else e.relation={label:"Unknown",id:"RO:Unknown"}},populateRows:function(){var e=this;if(this.rows=[],this.associationData.associations.forEach((function(t){var a,n,r,o,i=t.object,s=t.subject,l=e.parseTaxon(i),c=L["a"].pick(t,["id","provided_by","publications","evidence_types"]);c.publications=Object(Z["b"])(c.publications),c.provided_by=Object(Z["c"])(c.provided_by),c.provided_by=c.provided_by.map((function(e){return fe(e)})),c.evidence_types=c.evidence_types.map((function(t){return{id:t.id,label:t.label,url:e.eviHref(t)}}));var d=[];if(c.evidence_types.length>0){var u="fa-flask";d.push(u)}if(c.publications.length>0){var h="fa-book";d.push(h)}if(c.provided_by.length>0){var p="fa-database";d.push(p)}var v=[c.provided_by.length,c.publications.length,c.evidence_types.length].reduce((function(e,t){return e+t})),m=e.cardType;"interaction"===m?(m="gene",l=e.parseTaxon(s)):"ortholog-phenotype"===m?(m="phenotype",l=e.parseTaxon(s)):"ortholog-disease"===m?(m="disease",l=e.parseTaxon(s)):"homolog"===m?(m="gene",l=e.parseTaxon(i)):"causal-disease"===m||"correlated-disease"===m?m="disease":"causal-gene"!==m&&"correlated-gene"!==m||(m="gene");var g,f="/".concat(m,"/").concat(i.id);("model"===m&&(f="/".concat(i.id)),i.id.startsWith("BNODE")&&"publication"!==m&&(f=null),s.label!==Object(Z["e"])(s.label)&&(s.label=Object(Z["d"])(s.label)),i.label!==Object(Z["e"])(i.label)&&(i.label=Object(Z["d"])(i.label)),"publication"===m)&&(i.label=Te(null===(g=t.publicationMeta)||void 0===g?void 0:g.title,50)||i.label);var b="/".concat(e.nodeType,"/").concat(s.id);e.fixupRelation(t,e.nodeType,e.cardType),e.rows.push({annotationType:e.cardType,evidence:c,supportLength:v,supportIcons:d,objectCurie:i.id,assocObject:i.label,objectLink:f,assocSubject:s.label,subjectCurie:s.id,subjectLink:b,author:(null===(a=((null===(n=t.publicationMeta)||void 0===n?void 0:n.authors)||[])[0])||void 0===a?void 0:a.name)||"",publisher:(null===(r=t.publicationMeta)||void 0===r?void 0:r.source)||"",date:((null===(o=t.publicationMeta)||void 0===o?void 0:o.pubdate)||"").split(/\s/)[0]||"",taxonLabel:l.label,taxonId:l.id,relation:t.relation,frequency:t.frequency,onset:t.onset,_showDetails:!1}),t.relation.url=e.relationHref(t.relation),t.frequency&&(t.frequency.url=e.frequencyHref(t.frequency)),t.onset&&(t.onset.url=e.onsetHref(t.onset))})),Object(F["d"])(this.cardType)){var t=Object.keys(this.associationData.facet_counts)[0];Object.keys(this.associationData.facet_counts[t]).forEach((function(t){e.taxonFilter.taxons[t]=!0})),this.taxonFilter.counts=this.associationData.facet_counts[t]}this.totalAssociations=this.associationData.numFound},getTotalRowCounts:function(){var e=this,t=0;return Object.values(this.trueTaxonFilters()).forEach((function(a){t+=e.taxonFilter.counts[a]})),t},generateFields:function(){this.hasTaxon=!1,this.hasFrequencyOnset=!1;var e=2,t=[{key:"assocObject",label:this.firstCap(this.cardType),class:"assoc-object"},{key:"relation",label:"Relation",class:"relation-column-width"},{key:"support",class:"support-column-width",label:"Support"}];this.isGroup&&(t.splice(e,0,{key:"assocSubject",label:this.firstCap(this.nodeType),class:"assoc-subject"}),e++),this.cardType.includes("ortholog")&&(t.splice(0,0,{key:"assocSubject",label:this.firstCap(this.nodeType),class:"assoc-subject"}),e++),this.cardType.includes("publication")&&(t.splice(1,0,{key:"author",label:"Author"},{key:"date",label:"Date"},{key:"publisher",label:"Publisher"}),e+=3),this.isFrequencyOnsetType(this.nodeType,this.cardType)&&(this.hasFrequencyOnset=!0,t.splice(e,0,{key:"frequency",label:"Frequency",class:"frequency-column-width"}),e++,t.splice(e,0,{key:"onset",label:"Onset",class:"onset-column-width"})),Object(F["d"])(this.cardType)&&(this.hasTaxon=!0,t.splice(0,0,{key:"taxon",label:"Species"})),this.fields=t},parseTaxon:function(e){var t={label:"",id:""};return"taxon"in e&&(t.label=e.taxon.label,t.id=e.taxon.id),t},firstCap:function(e){return e.charAt(0).toUpperCase()+e.slice(1)},pubHref:function(e){return"/publication/".concat(e)},eviHref:function(e){var t=e.id||"",a=t.split(/[:]+/).pop();return"http://purl.obolibrary.org/obo/ECO_".concat(a)},relationHref:function(e){var t=e.id||"",a=t.split(/[:]+/).slice(-2,2).join("_");return"http://purl.obolibrary.org/obo/".concat(a)},frequencyHref:function(e){var t=e.id||"",a=t.split(/[:]+/).slice(-2,2).join("_");return"http://purl.obolibrary.org/obo/".concat(a)},onsetHref:function(e){var t=e.id||"",a=t.split(/[:]+/).slice(-2,2).join("_");return"http://purl.obolibrary.org/obo/".concat(a)},sourceLabel:function(e){var t=e.split(/[/]+/).pop().split(/[.]+/)[0].toUpperCase();return t},isFrequencyOnsetType:function(e,t){return"disease"===e&&"phenotype"===t||"phenotype"===e&&"disease"===t}}},Ne=Re,Pe=(a("ef71"),Object(y["a"])(Ne,pe,ve,!1,null,null,null)),He=Pe.exports,Le=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{directives:[{name:"show",rawName:"v-show",value:e.showGeneExac,expression:"showGeneExac"}],staticClass:"container-fluid",attrs:{id:"exacGene"}},[a("table",{staticClass:"table table-hover"},[e._m(0),a("tbody",[a("tr",[a("th",{attrs:{scope:"row"}},[e._v("Synonymous")]),a("td",[e._v(e._s(e.exacGene.exp_syn))]),a("td",[e._v(e._s(e.exacGene.n_syn))]),a("td",[e._v("z = "+e._s(e.exacGene.syn_z))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("Missense")]),a("td",[e._v(e._s(e.exacGene.exp_mis))]),a("td",[e._v(e._s(e.exacGene.n_mis))]),a("td",[e._v("z = "+e._s(e.exacGene.mis_z))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("LoF")]),a("td",[e._v(e._s(e.exacGene.exp_lof))]),a("td",[e._v(e._s(e.exacGene.n_lof))]),a("td",[e._v("pLI = "+e._s(e.exacGene.p_li))])])])]),a("div",{staticClass:"row"},[a("div",{staticClass:"col-9",attrs:{id:"mgi-link"}},[e._v(" [Retrieved from "),a("a",{attrs:{href:e.exacGene.link,target:"_blank"}},[e._v(" MyGene.info ")]),e._v(" ] ")]),e._m(1)])])},Ge=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("thead",[a("tr",[a("th",[e._v("Constraint from ExAC")]),a("th",[e._v("Expected no. variants")]),a("th",[e._v("Observed no. variants")]),a("th",[e._v("Constraint Metric")])])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"col-3",attrs:{id:"exac-link"}},[a("a",{staticClass:"fa fa-link",attrs:{target:"_blank",href:"http://exac.broadinstitute.org/"}})])}],Be={props:{nodeId:{type:String,required:!0}},data:function(){return{exacGene:"",showGeneExac:!1,curieMap:{HGNC:"hgnc",OMIM:"mim",ENSEMBL:"ensembl.gene",NCBIGene:"entrezgene"}}},computed:{nodePrefix:function(){var e=this.nodeId.split(":");return{prefix:e[0],identifier:e[1]}}},mounted:function(){this.nodePrefix.prefix in this.curieMap?this.hitMyGene():this.$emit("show-exac",!1)},methods:{round:function(e,t){var a="";return a=e<1?e.toPrecision(2):Number(Math.round("".concat(e,"e").concat(t))+"e-".concat(t)),a},hitMyGene:function(){var e=this,t="https://mygene.info/v3/query/",a="".concat(this.curieMap[this.nodePrefix.prefix],":").concat(this.nodePrefix.identifier);B.a.get(t,{params:{q:a,fields:"exac"}}).then((function(t){var a=t.data.hits[0];a&&a.exac?(e.showGeneExac=!0,e.$emit("show-exac",!0),e.exacGene={exp_syn:e.round(a.exac.all.exp_syn,1),n_syn:e.round(a.exac.all.n_syn,1),syn_z:e.round(a.exac.all.n_syn,1),exp_mis:e.round(a.exac.all.exp_mis,1),n_mis:e.round(a.exac.all.n_mis,1),mis_z:e.round(a.exac.all.mis_z,2),exp_lof:e.round(a.exac.all.exp_lof,1),n_lof:e.round(a.exac.all.n_lof,1),p_li:e.round(a.exac.all.p_li,1),link:t.request.responseURL}):e.$emit("show-exac",!1)})).catch((function(){e.$emit("show-exac",!1)}))}}},Fe=Be,Je=(a("2368"),Object(y["a"])(Fe,Le,Ge,!1,null,null,null)),je=Je.exports,Ue=function(){var e=this,t=e.$createElement,a=e._self._c||t;return e.showTable?a("div",{staticClass:"col-xs-12 col-md-10 col-lg-9 table-responsive",attrs:{id:"vue-exac"}},[a("h2",{staticClass:"p-2"},[e._v("ExAC Population Frequencies")]),a("table",{staticClass:"table table-hover"},[e._m(0),a("tbody",[a("tr",[a("th",{attrs:{scope:"row"}},[e._v("South Asian")]),a("td",[e._v(e._s(e.rowData.sas.aC))]),a("td",[e._v(e._s(e.rowData.sas.aN))]),a("td",[e._v(e._s(e.rowData.sas.hom))]),a("td",[e._v(e._s(e.rowData.sas.aF))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("Other")]),a("td",[e._v(e._s(e.rowData.oth.aC))]),a("td",[e._v(e._s(e.rowData.oth.aN))]),a("td",[e._v(e._s(e.rowData.oth.hom))]),a("td",[e._v(e._s(e.rowData.oth.aF))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("Latino")]),a("td",[e._v(e._s(e.rowData.amr.aC))]),a("td",[e._v(e._s(e.rowData.amr.aN))]),a("td",[e._v(e._s(e.rowData.amr.hom))]),a("td",[e._v(e._s(e.rowData.amr.aF))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("European (Non-Finnish)")]),a("td",[e._v(e._s(e.rowData.nfe.aC))]),a("td",[e._v(e._s(e.rowData.nfe.aN))]),a("td",[e._v(e._s(e.rowData.nfe.hom))]),a("td",[e._v(e._s(e.rowData.nfe.aF))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("African")]),a("td",[e._v(e._s(e.rowData.afr.aC))]),a("td",[e._v(e._s(e.rowData.afr.aN))]),a("td",[e._v(e._s(e.rowData.afr.hom))]),a("td",[e._v(e._s(e.rowData.afr.aF))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("East Asian")]),a("td",[e._v(e._s(e.rowData.eas.aC))]),a("td",[e._v(e._s(e.rowData.eas.aN))]),a("td",[e._v(e._s(e.rowData.eas.hom))]),a("td",[e._v(e._s(e.rowData.eas.aF))])]),a("tr",[a("th",{attrs:{scope:"row"}},[e._v("European (Finnish)")]),a("td",[e._v(e._s(e.rowData.fin.aC))]),a("td",[e._v(e._s(e.rowData.fin.aN))]),a("td",[e._v(e._s(e.rowData.fin.hom))]),a("td",[e._v(e._s(e.rowData.fin.aF))])]),a("tr",{staticStyle:{"border-top":"solid black 2px"}},[a("th",{attrs:{scope:"row"}},[e._v("Total")]),a("td",[e._v(" "+e._s(e.rowData.tot.aC)+" ")]),a("td",[e._v(" "+e._s(e.rowData.tot.aN)+" ")]),a("td",[e._v(" "+e._s(e.rowData.tot.hom)+" ")]),a("td",[e._v(" "+e._s(e.rowData.tot.aF)+" ")])])])]),a("div",{attrs:{id:"exac-link"}},[a("a",{staticClass:"glyphicon glyphicon-link",attrs:{href:e.exacID,target:"_blank"}})])]):e._e()},We=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("thead",[a("tr",[a("th",[e._v("Population")]),a("th",[e._v("Allele Count")]),a("th",[e._v("Allele Number")]),a("th",[e._v("Number of Homozygotes")]),a("th",[e._v("Allele Frequency")])])])}],ze={props:{nodeId:{type:String,required:!0}},data:function(){return{rowData:"",exacID:"",showTable:!1,curieMap:{ClinVarVariant:"clinvar.variant_id",dbSNP:"dbsnp.rsid"}}},computed:{nodePrefix:function(){var e=this.nodeId.split(":");return{prefix:e[0],identifier:e[1]}}},mounted:function(){this.nodePrefix.prefix in this.curieMap&&this.hitMyVariant()},methods:{buildRowData:function(e,t,a,n){var r=this,o={},i=0,s=0,l=0;return e.forEach((function(e){var c=t["ac_".concat(e)],d=a["an_".concat(e)],u=n["hom_".concat(e)];i+=c,s+=d,l+=u;var h={aC:c,aN:d,hom:u,aF:r.singleAlleleFrequency(c,d)};o[e]=h})),o.tot={aC:i,aN:s,hom:l,aF:this.round(i/s,4)},o},singleAlleleFrequency:function(e,t){return this.round(e/t,7)},round:function(e,t){var a="";return a=e<1?e.toPrecision(2):Number(Math.round("".concat(e,"e").concat(t))+"e-".concat(t)),a},hitMyVariant:function(){var e=this,t="https://myvariant.info/v1/query";B.a.get(t,{params:{q:"".concat(this.curieMap[this.nodePrefix.prefix],":").concat(this.nodePrefix.identifier),fields:"exac"}}).then((function(t){if(1===t.data.total){var a=t.data.hits[0].exac;if(a){var n=a.ac,r=a.an,o=a.hom,i="https://exac.broadinstitute.org/variant/",s=[a.chrom,a.pos,a.ref,a.alt].join("-");e.exacID="".concat(i).concat(s);var l=["sas","oth","amr","nfe","afr","eas","fin"];e.rowData=e.buildRowData(l,n,r,o),e.showTable=!0}}})).catch((function(e){console.log(e)}))}}},Ke=ze,qe=(a("7ab0"),Object(y["a"])(Ke,Ue,We,!1,null,null,null)),Ve=qe.exports,$e=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"genome-feature"},[e.mygeneData.externalURL?a("div",[a("svg",{attrs:{id:"genome-feature",width:"100%"}})]):e._e(),e.mygeneData.externalURL?a("div",{staticClass:"p-3"},[a("a",{staticClass:"col-12",attrs:{href:e.mygeneData.externalURL,target:"_blank",rel:"noopener noreferrer"}},[e._v(" Browse Genome at "+e._s(e.position.chr)+":"+e._s(e.position.start)+".."+e._s(e.position.end)+" "+e._s(e.position.strand>0?"+":"-")+" ( "+e._s(e.position.end-e.position.start)+" kb) "),a("i",{staticClass:"fa fa-link"})])]):a("div",[a("h6",[e._v("No Position Data Available for gene "+e._s(e.geneInfo.symbol))])])])},Xe=[],Ye=a("7fe9"),Ze=a.n(Ye),Qe={props:{mygeneData:{type:Object,required:!1,default:null}},data:function(){return{geneInfo:this.mygeneData.hits&&this.mygeneData.hits.length>0?this.mygeneData.hits[0]:null,position:this.mygeneData.hits&&this.mygeneData.hits.length>0?this.mygeneData.hits[0].genomic_pos:null}},mounted:function(){this.mygeneData.hits&&1===this.mygeneData.hits.length&&this.generateView(this.mygeneData.hits[0])},methods:{generateView:function(e){var t=e.genomic_pos;if(Object(F["b"])(e.taxid)&&t){var a=Object(F["a"])("NCBITaxon:".concat(e.taxid)),n="?name=".concat(e.symbol);t.ensemblgene&&(n+="&name=".concat(t.ensemblgene));var r={locale:"global",chromosome:t.chr,start:t.start,end:t.end,tracks:[{id:1,genome:a,type:"ISOFORM",url:["https://agr-apollo.berkeleybop.io/apollo/track/","/All%20Genes/",".json".concat(n)],transcriptTypes:["mRNA","ncRNA","piRNA","lincRNA","miRNA","pre_miRNA","snoRNA","lnc_RNA","tRNA","snRNA","rRNA","ARS","antisense_RNA","C_gene_segment","V_gene_segment","pseudogene_attribute","snoRNA_gene"]}]};new Ze.a(r,"#genome-feature",700,400)}}}},et=Qe,tt=(a("e911"),Object(y["a"])(et,$e,Xe,!1,null,null,null)),at=tt.exports,nt=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"reactome-wrapper",attrs:{id:"reactome-wrapper"}},[e.reactomeId?a("div",{staticClass:"row",attrs:{id:"reactome-div"}}):a("div",[a("h6",[e._v("No Reactome ID")])])])},rt=[],ot=null,it="reactome-script",st="reactome-wrapper",lt="reactome-div",ct="https://reactome.org/DiagramJs/diagram/diagram.nocache.js";window.onReactomeDiagramReady=function(){ot.buildDiagram(),ot.loadDiagram()};var dt={props:{reactomeId:{type:String,required:!0,default:null}},data:function(){return{reactomeLoaded:!1,diagram:null,width:null,height:null}},mounted:function(){if(ot=this,this.fit(),document.getElementById(it))this.diagram||this.buildDiagram(),this.loadDiagram();else{var e=document.createElement("script");e.src=ct,e.id=it,document.getElementsByTagName("head")[0].appendChild(e)}window.addEventListener("resize",this.handleResize)},beforeUnmount:function(){window.removeEventListener("resize",this.handleResize),ot=null},methods:{fit:function(){var e=document.getElementById(st);e?(this.width=e.clientWidth,this.height=window.innerHeight-150):(this.width=50,this.height=50)},handleResize:function(){this.fit(),this.diagram&&this.diagram.resize(this.width,this.height)},buildDiagram:function(){document.getElementById(lt);this.fit(),this.diagram=Reactome.Diagram.create({placeHolder:lt,width:this.width,height:this.height})},loadDiagram:function(){this.diagram.loadDiagram(this.reactomeId)}}},ut=dt,ht=(a("b360"),Object(y["a"])(ut,nt,rt,!1,null,null,null)),pt=ht.exports;function vt(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function mt(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){vt(o,n,r,i,s,"next",e)}function s(e){vt(o,n,r,i,s,"throw",e)}i(void 0)}))}}function gt(e){var t=(new DOMParser).parseFromString(e,"text/html");return t.documentElement.textContent}var ft=["phenotype","gene","causal-gene","correlated-gene","variant","model","disease","causal-disease","correlated-disease","pathway","cellline","anatomy","publication","case","function","genotype","homolog","interaction","ortholog-phenotype","ortholog-disease"],bt={anatomy:a("5df7"),cellline:a("a545"),disease:a("9212"),"causal-disease":a("9212"),"correlated-disease":a("9212"),function:a("f415"),gene:a("e7a4"),"causal-gene":a("e7a4"),"correlated-gene":a("e7a4"),genotype:a("0fc2"),case:a("9212"),homolog:a("fc62"),interaction:a("caab"),publication:a("c433"),model:a("443d"),"ortholog-disease":a("034f"),"ortholog-phenotype":a("94b4"),pathway:a("03a1"),phenotype:a("c359"),variant:a("51db")},_t={anatomy:"Anatomy",cellline:"Cell Line",disease:"Disease","causal-disease":"Disease (causal)","correlated-disease":"Disease (correlated)",function:"Function",gene:"Gene","causal-gene":"Gene (causal)","correlated-gene":"Gene (correlated)",genotype:"Genotype",case:"Case",homolog:"Homolog",interaction:"Interaction",publication:"Publication",model:"Model","ortholog-phenotype":"Ortholog Phenotype","ortholog-disease":"Ortholog Disease",pathway:"Pathway",phenotype:"Phenotype",variant:"Variant"},yt={components:{"node-sidebar":he,"assoc-table":He,"exac-gene":je,"exac-variant":Ve,"genome-feature":at,"reactome-viewer":pt,"histo-pheno":C},data:function(){return{jsonld:[{"@context":"http://schema.org","@type":"Organization",url:"https://monarchinitiative.org",email:"info@monarchinitiative.org"},{"@context":"http://schema.org","@type":"WebSite",url:"https://monarchinitiative.org",potentialAction:{"@type":"SearchAction",target:"https://monarchiniative.org/search/{term}","query-input":"required name=term"}}],isFacetsShowing:!1,isNeighborhoodShowing:!1,facetObject:{evidence:{IEA:!0},systems:{"Skeletal system":!0,Limbs:!0,"Nervous system":!0,"Head or neck":!0,"Metabolism/homeostasis":!0,"Cardiovascular system":!0,Integument:!0,"Genitourinary system":!0,Eye:!0,Musculature:!0,Neoplasm:!0,"Digestive system":!0,"Immune System":!0,"Blood and blood-forming tissues":!0,Endocrine:!0,"Respiratory system":!0,Ear:!0,"Connective tissue":!0,"Prenatal development or birth":!0,Growth:!0,Constitutional:!0,"Thoracic cavity":!0,Breast:!0,Voice:!0,Cellular:!0}},isSelected:{phenotypes:!1,genes:!1,models:!1,diseases:!1},taxonCounts:{},node:null,nodeError:null,equivalentClasses:null,superclasses:null,subclasses:null,synonyms:null,references:[],inheritance:null,modifiers:null,contentScript:"",contentBody:"",path:null,icons:bt,labels:_t,nodeId:null,nodeType:null,nodeIcon:null,nodeCategory:null,originalId:null,isGroup:!1,availableCards:ft,nonEmptyCards:[],expandedCard:null,hasGeneExac:!1,entrezResult:null,reactomeId:null,isRedirected:!1,showExac:!1,authoritiveXref:{synopsis:"",patient:""},hasXrefs:!1,counts:{disease:0,phenotype:0,gene:0,variant:0,model:0,pathway:0,publication:0,cellline:0,genotype:0,case:0},histoPhenoData:{},relationshipsColumns:[{label:"Subject",field:"subject.id"},{label:"Property",field:"property.id"},{label:"Object",field:"object.id"},{label:"Source",field:"source"}]}},computed:{},watch:{facetObject:{handler:function(){this.updateCounts()},deep:!0},$route:function(e){if(e.path===this.path||this.isRedirected){var t=e.hash.slice(1);t!==this.cardType&&this.expandCard(t),this.isRedirected=!1}else this.fetchData(),this.originalId=null}},updated:function(){if(this.$refs.titleBar){var e=document.getElementsByClassName("title-bar-wrapper")[0];e.style.height=this.$refs.titleBar.clientHeight+20+"px"}},mounted:function(){this.fetchData()},methods:{overviewSection:function(){return this.node.description||this.inheritance||this.entrezResult||this.modifiers},supportSection:function(){return this.hasXrefs>0&&!this.entrezResult},expandCard:function(e){this.$router.push({hash:e}).catch((function(){})),this.expandedCard=e},hideOverlay:function(){this.isNeighborhoodShowing=!1},toggleNeighborhood:function(){this.isNeighborhoodShowing=!this.isNeighborhoodShowing},buildCounts:function(){this.node.association_counts&&this.updateCounts()},updateCounts:function(){var e=this,t=[];if(this.node.association_counts){var a=this.node.association_counts;this.availableCards.forEach((function(n){var r=a[n],o=r&&r.counts||0;e.counts[n]=o,o>0?t.push(n):"phenotype"!==n||"gene"!==e.nodeType&&"variant"!==e.nodeType&&"disease"!==e.nodeType||t.push(n)}))}this.nonEmptyCards=t},fetchData:function(){var e=this;return mt(p.a.mark((function t(){var a,n,r,o,i,s,l,c,d,u,h,v,m,g,f,b,_,y,w,C,M,D;return p.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:return e.path=e.$route.path,e.nodeId=e.$route.params.id,e.nodeType=e.path.split("/")[1],/_/.test(e.nodeId)&&!/:/.test(e.nodeId)&&(a=e.nodeId.replace("_",":"),e.$router.push(a)),e.node=null,e.nodeError=null,e.entrezResult=null,e.expandedCard=null,e.nonEmptyCards=[],e.isNeighborhoodShowing=!1,e.inheritance=null,e.references={linked:[],static:[]},e.modifiers=null,e.reactomeId=null,e.histoPhenoData={},t.prev=15,t.next=18,S["g"](e.nodeId,e.nodeType);case 18:n=t.sent,t.next=25;break;case 21:return t.prev=21,t.t0=t["catch"](15),e.nodeError=t.t0,t.abrupt("return");case 25:return t.next=27,S["f"](e.nodeId,e.nodeType);case 27:if(r=t.sent,e.nodeId!==n.id&&(e.isRedirected=!0,e.originalId=e.$route.params.id,e.nodeId=n.id,e.$router.push(n.id)),e.node=n,r.synonyms?e.synonyms=r.synonyms:e.synonyms={},"disease"!==e.nodeType){t.next=36;break}return t.next=34,S["j"](e.node.id);case 34:o=t.sent,e.histoPhenoData={categories:o};case 36:if("publication"!==e.nodeType){t.next=41;break}return t.next=39,X([e.nodeId]);case 39:i=t.sent[0],i&&(e.entrezResult=i,s=gt(i.title),e.node.label=s,e.node.uri=i.pubmedURL,e.entrezResult.doi=i.doi||"No doi found",e.entrezResult.abstract=i.abstract||"No abstract found");case 41:if(e.node.label||(e.node.label=e.node.id),e.node.label!==Object(Z["e"])(e.node.label)&&(e.node.label=Object(Z["d"])(e.node.label)),"gene"!==e.nodeType&&"variant"!==e.nodeType){t.next=51;break}return t.next=46,U(e.node.id);case 46:l=t.sent,c=l&&l.hits[0],c&&(n.description=c.summary,n.geneInfo=l),t.next=52;break;case 51:"case"===e.nodeType&&(n.description=e.getCaseDescription());case 52:d="REACT:","pathway"===e.nodeType&&0===e.node.id.indexOf(d)&&(e.reactomeId=e.node.id.slice(d.length)),u="NCBITaxon:",e.node.taxon&&e.node.taxon.id&&0===e.node.taxon.id.indexOf(u)&&(h=e.node.taxon.id.slice(u.length),e.node.taxon.uri="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=info&id=".concat(h)),v=r.nodeMap,m=r.superclasses,g=r.subclasses,f=r.xrefs,e.hasXrefs=f.length>0,e.superclasses=L["a"].map(L["a"].uniq(m),(function(e){return{id:e,label:v[e].lbl}})),e.subclasses=L["a"].map(L["a"].uniq(g),(function(e){return{id:e,label:v[e].lbl}})),e.isGroup=e.subclasses.length>0,b=Object.keys(e.node.association_counts.sources),b=Object(Z["c"])(b),b=new Set([e.nodeId.split(":")[0].toLowerCase()].concat(b)),b.has("impc")&&"phenotype"!==e.nodeType&&"gene"!==e.nodeType&&b.delete("impc"),_=new Set([]),y=new Set([]),e.resetAuthoritiveXref(),f.forEach((function(t){var a=!1,n=t.split(":")[0].toLowerCase();if(Array.from(b).concat([n]).forEach((function(n){var r=Object(Z["a"])(n,t,e.node.label.split(" ")[0]);if(r){if(a=!0,_.has(t)){var o=t.split(":")[1];t="".concat(n.toUpperCase(),":").concat(o)}else _.add(t);if(!y.has(r)){var i={id:t,label:"",uri:r};e.filterAuthority(i)||e.references.linked.push(i),y.add(r)}}})),!a){var r={id:t,label:"",uri:null};e.filterAuthority(r)||e.references.static.push(r)}})),"gene"===e.nodeType&&(w={},C=e.node.id.split(":")[0].toLowerCase(),w.label=C,w.id=e.node.id,w.uri=Object(Z["a"])(C,e.node.id,e.node.label.split(" ")[0]),e.authoritiveXref.gene.taxon=w),e.nodeId.startsWith("HGNC")&&(e.authoritiveXref.gene.clinical.varsome={id:e.nodeId,label:"Varsome",uri:"https://varsome.com/gene/".concat(e.nodeId)}),e.node.inheritance&&(e.inheritance=L["a"].uniq(e.node.inheritance.map((function(e){return e.label}))).join(", ")),e.node.clinical_modifiers&&(e.modifiers=L["a"].uniq(e.node.clinical_modifiers.map((function(e){return e.label}))).join(", ")),e.nodeCategory=e.node.categories?e.node.categories[0].toLowerCase():e.nodeType,e.nodeIcon=e.icons[e.nodeCategory],e.hasGeneExac="gene"===e.nodeType||"variant"===e.nodeType,e.buildCounts(),M=e.$router.currentRoute.hash,M.length>1&&(D=M.split("?")[0].slice(1),e.$nextTick((function(){e.expandCard(D)})));case 82:case"end":return t.stop()}}),t,null,[[15,21]])})))()},getCaseDescription:function(){return'\n    <div class="text">\n      The NIH Undiagnosed Disease Program (UDP) was started in 2008.\n      The program’s goals include making diagnosis and conducting research for individuals\n      and families with illnesses that remain undiagnosed despite an extensive medical\n      workup. In 2013, the program expanded to the Undiagnosed Diseases Network (UDN).\n      The UDN performs similar work, but now includes a number of clinical and research\n      sites across the United States. Persons interested in learning more about the UDN,\n      are invited to visit the UDN website:<br>\n      <a href="https://undiagnosed.hms.harvard.edu/">\n        https://undiagnosed.hms.harvard.edu/\n      </a>\n    </div>\n\n    <div class="text" style="padding-top:10px">\n      This case includes information from a participant in the Undiagnosed Diseases Program\n      (UDP) at the National Institutes of Health. For protection of participant privacy,\n      the case description does not include identifiable information.\n      Features of the participant’s medical condition are presented, along with genes of potential\n      interest. In addition, we provide similarity scores to diseases, other cases, and non-human models\n      based on phenotype profile comparisons computed by OwlSim.\n      If you are a researcher studying a listed gene or clinical feature, or if you have a\n      family member with a similar illness, please consider contacting the UDP. Contacting\n      the UDP will allow a discussion about options for collaboration and/or study participation.\n      Inquiries should be sent to David Adams, MD, PhD at david.adams[at]nih.gov.\n      Some UDN families and individuals have elected to post more extensive information on the web.\n      Those pages can be viewed at the following website:<br>\n      <a href="https://undiagnosed.hms.harvard.edu/updates/participant-pages/">\n        https://undiagnosed.hms.harvard.edu/updates/participant-pages/\n      </a>\n    </div>\n       '},showExacSection:function(e){this.showExac=e},filterAuthority:function(e){if("disease"===this.nodeType){if(e.id.includes("OMIM")||e.id.includes("ORPHA")){if(!this.authoritiveXref.synopsis.id.includes("OMIM"))return e.label=e.id.includes("OMIM")?"OMIM":"Orphanet",this.authoritiveXref.synopsis=e,!0}else if(e.id.includes("GARD")){var t=e.id.replace(/^GARD:(0)*/g,""),a=this.node.label.replace(/\s/g,"-");return e.uri="https://rarediseases.info.nih.gov/diseases/".concat(t,"/").concat(a.toLowerCase()),e.label="GARD",this.authoritiveXref.patient=e,!0}}else if("gene"===this.nodeType){if(e.id.includes("OMIM"))return e.label="OMIM",this.authoritiveXref.gene.clinical.omim=e,!0;if(e.id.includes("ENSEMBL")&&e.uri.includes("ensembl"))return e.label="ENSEMBL",this.authoritiveXref.gene.ensembl=e,!0;if(e.id.includes("REACT"))return e.label="REACTOME",this.authoritiveXref.gene.pathway=e,!0}else if("phenotype"===this.nodeType&&e.id.includes("HP"))return e.label="HPO",this.authoritiveXref.phenotype=e,!0;return!1},resetAuthoritiveXref:function(){this.authoritiveXref={synopsis:{id:"",uri:"",label:""},patient:{id:"",uri:"",label:""},gene:{ensembl:"",taxon:"",clinical:{},pathway:""},phenotype:""}}}},wt=yt,Ct=(a("b44c"),Object(y["a"])(wt,P,H,!1,null,null,null)),Mt=Ct.exports,St=a("733f");function Dt(e){return kt(e)||xt(e)||It(e)||Ot()}function Ot(){throw new TypeError("Invalid attempt to spread non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.")}function It(e,t){if(e){if("string"===typeof e)return At(e,t);var a=Object.prototype.toString.call(e).slice(8,-1);return"Object"===a&&e.constructor&&(a=e.constructor.name),"Map"===a||"Set"===a?Array.from(e):"Arguments"===a||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(a)?At(e,t):void 0}}function xt(e){if("undefined"!==typeof Symbol&&null!=e[Symbol.iterator]||null!=e["@@iterator"])return Array.from(e)}function kt(e){if(Array.isArray(e))return At(e)}function At(e,t){(null==t||t>e.length)&&(t=e.length);for(var a=0,n=new Array(t);a<t;a++)n[a]=e[a];return n}r.a.use(s["a"]);var Et=["anatomy","cellline","case","disease","function","gene","genotype","homolog","interaction","publication","model","ortholog-phenotype","ortholog-disease","pathway","phenotype","variant"],Tt=function(e,t,a){var n=sessionStorage.redirect;n?(console.info({redirect:n}),delete sessionStorage.redirect,a(n)):a()},Rt=new s["a"]({mode:"history",base:"/",routes:[].concat(Dt(Et.map((function(e){return{path:"/".concat(e,"/:id"),name:"/Node".concat(e),component:Mt,beforeEnter:function(e,t,a){if(e.fullPath.startsWith("/disease/Orphanet:")){var n=e.params.id.replace("Orphanet","ORPHA");Rt.push("/disease/".concat(n))}else a()}}}))),[{path:"/",name:"home",component:N,beforeEnter:Tt},{path:"/about/monarch",name:"about-monarch",component:a("92ec").default},{path:"/about/monarch-web-services",name:"monarch-web-services",component:a("709a").default},{path:"/about/team",name:"about-team",component:a("ad30").default},{path:"/about/disclaimer",name:"about-disclaimer",component:a("adad").default},{path:"/documentation/publications",name:"about-publications",component:a("2d78").default},{path:"/help/cite",name:"about-citing",component:a("09ce").default},{path:"/help/linkout",name:"link-to-us",component:a("5f5a").default},{path:"/help/contact",name:"contact-us",component:a("17ca").default},{path:"/tools/other-tools",name:"other-tools",component:a("6b39").default},{path:"/tools/text-annotate",name:"text-annotate",component:function(){return a.e("text-annotate").then(a.bind(null,"ecb7"))}},{path:"/search/:query?",name:"search",component:function(){return a.e("search").then(a.bind(null,"2d3b"))}},{path:"/analyze/phenotypes",name:"analyze-phenotypes",component:function(){return a.e("analyze-phenotypes").then(a.bind(null,"69ad"))}},{path:"/analytics",name:"analytics",component:function(){return a.e("mon-analytics").then(a.bind(null,"41f2"))}},{path:"/about/data-sources",name:"sources",component:function(){return a.e("data-sources").then(a.bind(null,"2e20"))}},{path:"/about/licensing",name:"licensing",component:function(){return a.e("licensing").then(a.bind(null,"4de4"))}},{path:"/about/phenotypes",name:"about-phenotypes",component:a("4e41").default},{path:"/glossary",name:"glossary",component:a("2849").default},{path:"/*",name:"RouteError",component:function(){return a.e("routeerror").then(a.bind(null,"7066"))},beforeEnter:function(e,t,a){var n=e.params.pathMatch;n.startsWith("MONARCH")&&(n=":"+n),Object(S["d"])(n).then((function(e){var t=Object(St["a"])(e.category);t&&Et.includes(t)?Rt.push("/".concat(t,"/").concat(n)):a()})).catch((function(){return a()}))}}]),scrollBehavior:function(e,t,a){var n={x:0,y:0};return a?n=a:e.hash&&(n={selector:e.hash}),n}});function Nt(e){return Object.keys(e.query).length>0}Rt.beforeEach((function(e,t,a){if(!Nt(e)&&Nt(t)){var n=Object.assign({},e,{query:t.query});a(n)}else a()}));var Pt=Rt,Ht=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"page-wrapper",attrs:{id:"app"}},[a("monarch-navbar"),a("div",{staticClass:"content-wrapper"},[a("router-view")],1),a("div",{staticClass:"footer-wrapper"},[a("monarch-footer",{attrs:{compact:e.useCompactFooter}})],1)],1)},Lt=[],Gt=function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("b-navbar",{staticClass:"sticky-top",attrs:{id:"monarchng-navbar",fixed:"true",toggleable:"lg",type:"dark",variant:"info"}},[n("b-navbar-brand",{attrs:{to:"/"}},[n("img",{staticClass:"branding-logo",attrs:{src:a("cce6"),alt:"Monarch Initiative logo",title:"Monarch Initiative front page"}})]),n("b-collapse",{attrs:{id:"nav_collapse","is-nav":""}},[n("b-navbar-nav",[n("b-nav-item-dropdown",{attrs:{text:"Tools"}},[n("b-dropdown-item",{attrs:{to:"/analyze/phenotypes"}},[e._v(" Phenotype Profile Search ")]),n("b-dropdown-item",{attrs:{to:"/tools/text-annotate"}},[e._v(" Text Annotator ")]),n("b-dropdown-item",{attrs:{to:"/tools/other-tools"}},[e._v(" Other Monarch Tools ")])],1),n("b-nav-item-dropdown",{attrs:{text:"About"}},[n("b-dropdown-item",{attrs:{to:"/about/monarch"}},[e._v(" About Monarch ")]),n("b-dropdown-item",{attrs:{to:"/about/team"}},[e._v(" Our Team ")]),n("b-dropdown-item",{attrs:{to:"/about/licensing"}},[e._v(" Licensing ")]),n("b-dropdown-item",{attrs:{to:"/about/licensing#ontologies"}},[e._v(" Ontologies ")]),n("b-dropdown-item",{attrs:{to:"/about/disclaimer"}},[e._v(" Disclaimer ")]),n("b-dropdown-item",{attrs:{to:"/about/phenotypes"}},[e._v(" About Phenotypes ")]),n("b-dropdown-divider"),n("b-dropdown-item",{attrs:{to:"/about/data-sources"}},[e._v(" Data Sources ")]),n("b-dropdown-item",{attrs:{to:"/about/monarch-web-services"}},[e._v(" Monarch Web Services ")]),n("b-dropdown-item",{attrs:{href:"https://archive.monarchinitiative.org/latest/",target:"_blank",rel:"noopener noreferrer"}},[e._v(" Data Downloads ")])],1),n("b-nav-item-dropdown",{attrs:{text:"Documentation"}},[n("b-dropdown-item",{attrs:{to:"/documentation/publications"}},[e._v(" Publications ")]),n("b-dropdown-item",{attrs:{href:"https://github.com/monarch-initiative",target:"_blank",rel:"noopener noreferrer"}},[e._v(" GitHub Organization ")]),n("b-dropdown-item",{attrs:{href:"https://medium.com/@MonarchInit/how-to-annotate-a-patients-phenotypic-profile-afc5773d6cdb",target:"_blank",rel:"noopener noreferrer"}},[e._v(" Phenotype Curation Guidelines ")]),n("b-dropdown-item",{attrs:{href:"https://github.com/obophenotype/upheno",target:"_blank",rel:"noopener noreferrer"}},[e._v(" Phenotype Ontologies Project ")])],1),n("b-nav-item-dropdown",{attrs:{text:"Help"}},[n("b-dropdown-item",{attrs:{to:"/help/contact"}},[e._v(" Contact Us ")]),n("b-dropdown-item",{attrs:{to:"/help/cite"}},[e._v(" Cite Monarch ")]),n("b-dropdown-item",{attrs:{to:"/help/linkout"}},[e._v(" Link to Monarch ")])],1),n("b-nav-item-dropdown",{attrs:{text:"Examples"}},[n("b-dropdown-item",{attrs:{to:"/phenotype/HP:0001994#disease"}},[e._v(" Phenotype: Renal Fanconi syndrome HP:0001994 ")]),n("b-dropdown-item",{attrs:{to:"/disease/MONDO:0007669#phenotype"}},[e._v(" Disease: renal cysts and diabetes syndrome MONDO:0007669 ")]),n("b-dropdown-item",{attrs:{to:"/disease/MONDO:0007947"}},[e._v(" Disease: Marfan syndrome MONDO:0007947 ")]),n("b-dropdown-item",{attrs:{to:"/disease/OMIM:105830"}},[e._v(" Disease: Angelman syndrome OMIM:105830 ")]),n("b-dropdown-item",{attrs:{to:"/disease/MONDO:0007739#gene"}},[e._v(" Disease: Huntington disease MONDO:0007739 ")]),n("b-dropdown-item",{attrs:{to:"/disease/MONDO:0010935"}},[e._v(" Disease: neuronopathy, distal hereditary motor, type 5A MONDO:0010935 ")]),n("b-dropdown-item",{attrs:{to:"/disease/OMIMPS:PS120435"}},[e._v(" Disease: Colorectal cancer, hereditary nonpolyposis OMIMPS:PS120435 (no superclass) ")]),n("b-dropdown-item",{attrs:{to:"/disease/MONDO:0010156"}},[e._v(" Disease: Troyer Syndrome MONDO:0010156 ")]),n("b-dropdown-divider"),n("b-dropdown-item",{attrs:{to:"/gene/MGI:2447362#model"}},[e._v(" Gene: Slc24a4 "),n("em",[e._v("Mus musculus")]),e._v(" MGI:2447362 (models) ")]),n("b-dropdown-item",{attrs:{to:"/gene/HGNC:11773"}},[e._v(" Gene: TGFBR2 "),n("em",[e._v("Homo sapiens")]),e._v(" HGNC:11773 ")]),n("b-dropdown-item",{attrs:{to:"/gene/HGNC:11180"}},[e._v(" Gene: SOD2 "),n("em",[e._v("Homo sapiens")]),e._v(" HGNC:11180 ")]),n("b-dropdown-item",{attrs:{to:"/gene/ZFIN:ZDB-GENE-980526-333"}},[e._v(" Gene: sox3 "),n("em",[e._v("Danio rerio")]),e._v(" ZFIN:ZDB-GENE-980526-333 ")]),n("b-dropdown-item",{attrs:{to:"/gene/NCBIGene:428534"}},[e._v(" Gene: SOX17 "),n("em",[e._v("Gallus gallus")]),e._v(" NCBIGene:428534 ")]),n("b-dropdown-divider"),n("b-dropdown-item",{attrs:{to:"/publication/PMID:28650316"}},[e._v(" Publication: The Na"),n("sup",[e._v("+")]),e._v("/Ca"),n("sup",[e._v("2+")]),e._v(", K"),n("sup",[e._v("+")]),e._v(" exchanger NCKX4 is required for efficient cone-mediated vision. PMID:28650316 ")]),n("b-dropdown-item",{attrs:{to:"/pathway/REACT:R-MMU-425561"}},[e._v(" Pathway: Sodium/Calcium exchangers REACT:R-MMU-425561 ")]),n("b-dropdown-item",{attrs:{to:"/pathway/REACT:R-SSC-381753"}},[e._v(" Pathway: Olfactory Signaling Pathway REACT:R-SSC-381753 ")]),n("b-dropdown-item",{attrs:{to:"/pathway/REACT:R-RNO-1461973"}},[e._v(" Pathway: Defensins REACT:R-RNO-1461973 ")])],1)],1),"/"!==e.$route.path?n("b-navbar-nav",{staticClass:"ml-auto"},[n("div",{staticClass:"nav-ac nav-search"},[n("monarch-autocomplete",{attrs:{"full-width-search":!1}})],1)]):e._e()],1),n("b-navbar-toggle",{attrs:{target:"nav_collapse"}}),"production"!==e.apiVersion?n("div",{directives:[{name:"b-popover",rawName:"v-b-popover.hover.v-danger.bottomleft",value:"You are currently using a beta version of our APIs. The Monarch Initiative is in the process of creating a new experience for you. If you believe you see an issue or want to suggest content please see the footer of this page.",expression:"\n      'You are currently using a beta version of our APIs. The Monarch Initiative is in the process of creating a new experience for you. If you believe you see an issue or want to suggest content please see the footer of this page.'\n    ",modifiers:{hover:!0,"v-danger":!0,bottomleft:!0}}],staticClass:"beta",attrs:{title:"Monarch APIs BETA"}},[e._v(" BETA ")]):e._e(),"production"!==e.apiVersion?n("div",{staticClass:"production"},[n("b-navbar-nav",[n("b-nav-item",{attrs:{href:"https://monarchinitiative.org/",target:"_blank"}},[e._v(" Main Site ")])],1)],1):e._e()],1)},Bt=[],Ft=a("5631"),Jt={name:"MonarchNavbar",components:{MonarchAutocomplete:Ft["a"]},data:function(){return{apiVersion:S["n"]}}},jt=Jt,Ut=(a("16a3"),Object(y["a"])(jt,Gt,Bt,!1,null,null,null)),Wt=Ut.exports,zt=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",[e.compact?a("div",{staticClass:"row footer compact"},[e._m(0),a("div",{staticClass:"col-12 col-lg-4 report"},[a("b-button",{staticClass:"issue-btn",attrs:{target:"_blank",href:"https://github.com/monarch-initiative/helpdesk/issues"}},[e._v(" Report Issue or Suggest Content ")])],1),e._m(1)]):a("div",{staticClass:"row footer"},[e._m(2),e._m(3)])])},Kt=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"col-12 col-lg-4 version"},[a("i",[e._v("The Monarch Initiative 2020")])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"col-12 col-lg-4 copyright"},[a("i",{staticClass:"fa fa-creative-commons fa-fw"}),e._v(" CC-BY 3.0, except where noted. ")])},function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("div",{staticClass:"col-4 col-lg-2"},[n("div",{staticClass:"media"},[n("img",{staticClass:"img-fluid monarch-logo",attrs:{src:a("a35e"),alt:"Monarch logo"}})])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"socials col-8 col-lg-10"},[a("div",{staticClass:"icons"},[a("a",{attrs:{href:"mailto:info@monarchinitiative.org"}},[a("i",{staticClass:"fa fa-envelope fa-fw"})]),a("a",{attrs:{target:"_blank",href:"https://twitter.com/MonarchInit"}},[a("i",{staticClass:"fa fa-twitter fa-fw"})]),a("a",{attrs:{target:"_blank",href:"https://github.com/monarch-initiative/"}},[a("i",{staticClass:"fa fa-github fa-fw"})])])])}],qt={props:{compact:{type:Boolean,required:!1,default:!0}},data:function(){return{isCompact:!0,version:"1.0.5"}}},Vt=qt,$t=(a("53d7"),Object(y["a"])(Vt,zt,Kt,!1,null,null,null)),Xt=$t.exports;r.a.prototype.$sanitizeText=Z["e"],r.a.component("monarch-footer",Xt);var Yt={name:"App",components:{"monarch-navbar":Wt},data:function(){return{useCompactFooter:!1}},watch:{$route:function(e){this.useCompactFooter="/"!==e.path}},mounted:function(){this.useCompactFooter="/"!==this.$route.path}},Zt=Yt,Qt=(a("5c0b"),Object(y["a"])(Zt,Ht,Lt,!1,null,null,null)),ea=Qt.exports;r.a.config.productionTip=!1,r.a.use(o["a"]),"monarchinitiative.org"===window.location.hostname&&r.a.use(i["a"],{config:{id:"UA-41803362-1",params:{anonymize_ip:!0,cookie_flags:"SameSite=None;Secure"}}},Pt),new r.a({router:Pt,mounted:function(){},render:function(e){return e(ea)}}).$mount("#app"),window.MonarchUIVersion="1.0.2"},"58a3":function(e,t,a){e.exports=a.p+"img/Monarch-Architecture2.7b58322f.png"},"5a40":function(e,t,a){},"5aaf":function(e,t,a){"use strict";a("85fb")},"5bdf":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"home-page-section about-monarch-section"},[a("div",{staticClass:"about-block"},[a("h3",[e._v("Connecting phenotypes to genotypes across species")]),a("div",{staticClass:"description"},[e._v(" Semantic-based integrative data analytics."),a("br"),a("br"),a("router-link",{attrs:{to:"/about/monarch"}},[a("b-button",{staticClass:"about-btn",attrs:{squared:""}},[e._v("Learn More"),a("i",{staticClass:"fa fa-caret-right"})])],1)],1)])])])])])},r=[],o=(a("c265"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},"5c0b":function(e,t,a){"use strict";a("9c0c")},"5df7":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_ANATOMY.dfa2358b.png"},"5f20":function(e,t,a){"use strict";a("ce57")},"5f5a":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"container-fluid monarch-view link-to-us"},[a("h2",{staticClass:"page-title"},[e._v("Link To The Monarch Initiative")]),a("p",[e._v("To link to Monarch, please refer to the following tables.")]),a("div",{staticClass:"card"},[a("h4",[e._v("Genes and Features")]),a("table",{attrs:{border:"2"}},[a("thead",[a("tr",[a("th",[e._v("Organization")]),a("th",[e._v("URL")]),a("th",[e._v("Example")])])]),a("tbody",[a("tr",[a("td",[e._v("NCBIGene "),a("strong",[e._v("(clique leader)")]),e._v("*")]),a("td",[e._v("http://monarchinitiative.org/NCBIGene:##")]),a("td",[e._v("https://monarchinitiative.org/NCBIGene:6622")])]),a("tr",[a("td",[e._v("ENSEMBL")]),a("td",[e._v("http://monarchinitiative.org/ENSEMBL:##")]),a("td",[e._v("https://monarchinitiative.org/ENSEMBL:ENSG00000145335")])]),a("tr",[a("td",[e._v("HGNC: HUGO Gene Nomenclature Committee")]),a("td",[e._v("http://monarchinitiative.org/HGNC:##")]),a("td",[e._v("https://monarchinitiative.org/HGNC:11138")])]),a("tr",[a("td",[e._v("KEGG-hsa: Ksyoto Encyclopedia of Genes and Genomes - Human")]),a("td",[e._v("http://monarchinitiative.org/KEGG-hsa:##")]),a("td",[e._v("https://monarchinitiative.org/KEGG-hsa:6622")])]),a("tr",[a("td",[e._v("FlyBase")]),a("td",[e._v("https://monarchinitiative.org/FlyBase:FBgn##")]),a("td",[e._v("https://monarchinitiative.org/FlyBase:FBgn0036212")])]),a("tr",[a("td",[e._v("Xenbase")]),a("td",[e._v("https://monarchinitiative.org/Xenbase:XB-GENE-##")]),a("td",[e._v("https://monarchinitiative.org/Xenbase:XB-GENE-1014191")])]),a("tr",[a("td",[e._v("RGD: Rat Genome Database")]),a("td",[e._v("https://monarchinitiative.org/RGD:##")]),a("td",[e._v("https://monarchinitiative.org/RGD:3729")])]),a("tr",[a("td",[e._v("MGI: Mouse Genome Informatics")]),a("td",[e._v("https://monarchinitiative.org/MGI:##")]),a("td",[e._v("https://monarchinitiative.org/MGI:1277151")])]),a("tr",[a("td",[e._v("ZFIN: Zebrafish Information Network")]),a("td",[e._v("https://monarchinitiative.org/ZFIN:ZDB-GENE-##")]),a("td",[e._v("https://monarchinitiative.org/ZFIN:ZDB-GENE-980526-166")])]),a("tr",[a("td",[e._v("Wormbase")]),a("td",[e._v("https://monarchinitiative.org/WormBase:WBGene##")]),a("td",[e._v("https://monarchinitiative.org/WormBase:WBGene00000415")])])])])]),a("div",{staticClass:"card"},[a("h4",[e._v("Disease")]),a("table",{attrs:{border:"2"}},[a("thead",[a("tr",[a("th",[e._v("Organization")]),a("th",[e._v("URL")]),a("th",[e._v("Example")])])]),a("tbody",[a("tr",[a("td",[e._v(" OMIM: Online Mendelian Inheritance in Man "),a("strong",[e._v("(clique leader)")]),e._v("* ")]),a("td",[e._v("http://monarchinitiative.org/OMIM:##")]),a("td",[e._v("https://monarchinitiative.org/OMIM:602404")])]),a("tr",[a("td",[e._v("Orphanet: Rare Diseases and Orphan Drugs")]),a("td",[e._v("http://monarchinitiative.org/Orphanet:##")]),a("td",[e._v("https://monarchinitiative.org/Orphanet:2828")])]),a("tr",[a("td",[e._v("UMLS: Unified Medical Language System")]),a("td",[e._v("http://monarchinitiative.org/UMLS:##")]),a("td",[e._v("https://monarchinitiative.org/UMLS:CN202824")])]),a("tr",[a("td",[e._v("MESH: Medical Subject Headings")]),a("td",[e._v("http://monarchinitiative.org/MESH:##")]),a("td",[e._v("https://monarchinitiative.org/MESH:D010300")])]),a("tr",[a("td",[e._v("Disease Ontology ID")]),a("td",[e._v("http://monarchinitiative.org/DOID:##")]),a("td",[e._v("https://monarchinitiative.org/DOID:14330")])])])])]),a("div",{staticClass:"card"},[a("h4",[e._v("Phenotype")]),a("table",{attrs:{border:"2"}},[a("thead",[a("tr",[a("th",[e._v("Ontology")]),a("th",[e._v("URL")]),a("th",[e._v("Example")])])]),a("tbody",[a("tr",[a("td",[e._v("HP: Human Phenotype Ontology")]),a("td",[e._v("http://monarchinitiative.org/HP:##")]),a("td",[e._v("https://monarchinitiative.org/HP:0400004")])]),a("tr",[a("td",[e._v("MP: Mammalian Phenotype Ontology")]),a("td",[e._v("http://monarchinitiative.org/MP:##")]),a("td",[e._v("https://monarchinitiative.org/MP:0003878")])]),a("tr",[a("td",[e._v("Uberpheno: phenotype ontology")]),a("td",[e._v("https://monarchinitiative.org/UPHENO:##")]),a("td",[e._v("https://monarchinitiative.org/UPHENO:0001001")])]),a("tr",[a("td",[e._v("UBERON: anatomy ontology")]),a("td",[e._v("http://monarchinitiative.org/UBERON:##PHENOTYPE")]),a("td",[e._v("https://monarchinitiative.org/UBERON:0001690PHENOTYPE")])])])])])])}],o=(a("9f01"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},6246:function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"home-page-section news-section"},[a("div",{staticClass:"row"},[a("div",{staticClass:"col-sm offset-lg-1 col-lg-6"},[e._m(0),a("ul",{staticClass:"list-inline"},e._l(e.newsItems.slice(0,7),(function(t,n){return a("li",{key:n},[a("span",{staticClass:"news-date"},[e._v(e._s(t.date))]),a("a",{staticClass:"news-title",attrs:{href:t.url,target:"_blank"}},[e._v(" "+e._s(t.title)+" ")])])})),0)]),e._m(1)])])])])])},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("h5",{staticClass:"news"},[a("i",{staticClass:"fa fa-newspaper-o fa-fw"}),e._v(" News from "),a("a",{attrs:{href:"https://medium.com/@MonarchInit",target:"__blank"}},[e._v(" @MonarchInit ")])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"col-sm col-lg-4"},[a("i",{staticClass:"fa fa-twitter fa-fw"}),e._v(" Tweets from "),a("a",{attrs:{href:"https://twitter.com/MonarchInit"}},[e._v("@MonarchInit")]),a("a",{staticClass:"twitter-timeline",attrs:{href:"https://twitter.com/MonarchInit","data-preview":"","data-chrome":"noheader nofooter","data-height":"350"}},[e._v(" @MonarchInit ")])])}],o=a("a34a"),i=a.n(o),s=a("bc3a"),l=a.n(s);function c(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function d(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){c(o,n,r,i,s,"next",e)}function s(e){c(o,n,r,i,s,"throw",e)}i(void 0)}))}}function u(){return h.apply(this,arguments)}function h(){return h=d(i.a.mark((function e(){var t,a,n;return i.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return t="https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/@monarchinit",a=[],e.prev=2,e.next=5,l.a.get(t);case 5:n=e.sent,a=n.data.items.map((function(e){return{url:e.link,date:new Date(e.pubDate.replace(/-/g,"/")).toDateString().slice(4),title:e.title}})),e.next=11;break;case 9:e.prev=9,e.t0=e["catch"](2);case 11:return e.abrupt("return",a);case 12:case"end":return e.stop()}}),e,null,[[2,9]])}))),h.apply(this,arguments)}function p(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function v(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){p(o,n,r,i,s,"next",e)}function s(e){p(o,n,r,i,s,"throw",e)}i(void 0)}))}}var m={data:function(){return{newsItems:[]}},mounted:function(){var e=this;return v(i.a.mark((function t(){return i.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:return t.next=2,u();case 2:e.newsItems=t.sent,function(e,t,a){var n,r=e.getElementsByTagName(t)[0],o=window.twttr||{};e.getElementById(a)?o.widgets.load():(n=e.createElement(t),n.id=a,n.src="https://platform.twitter.com/widgets.js",r.parentNode.insertBefore(n,r),o._e=[],o.ready=function(e){o._e.push(e)})}(document,"script","twitter-wjs");case 4:case"end":return t.stop()}}),t)})))()}},g=m,f=(a("b83f"),a("2877")),b=Object(f["a"])(g,n,r,!1,null,null,null);t["default"]=b.exports},"626f":function(e,t,a){e.exports=a.p+"img/monarch-logo-white-stacked.bd2e2757.png"},6271:function(e,t,a){"use strict";a("006b")},6391:function(e,t,a){"use strict";a("55b4")},"63e8":function(e,t,a){"use strict";a.d(t,"b",(function(){return u})),a.d(t,"a",(function(){return p}));var n=a("a34a"),r=a.n(n),o=a("bc3a"),i=a.n(o),s=a("651e"),l=a.n(s);function c(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function d(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){c(o,n,r,i,s,"next",e)}function s(e){c(o,n,r,i,s,"throw",e)}i(void 0)}))}}function u(){return h.apply(this,arguments)}function h(){return h=d(r.a.mark((function e(){var t,a,n,o,s;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return t="".concat("/","team.yaml"),e.next=3,i.a.get(t);case 3:return a=e.sent,n=null,e.prev=5,e.next=8,l.a.safeLoad(a.data,"utf8");case 8:o=e.sent,s=o.institutions,s.forEach((function(e){e.logo="".concat("/").concat(e.logo);var t=e.people,a=t.filter((function(e){return!e.alumni})).map((function(e){return e.name}));e.peopleNames=a,t.forEach((function(e){e.picture="".concat("/").concat(e.picture)}))})),n=o,e.next=16;break;case 14:e.prev=14,e.t0=e["catch"](5);case 16:return e.abrupt("return",n);case 17:case"end":return e.stop()}}),e,null,[[5,14]])}))),h.apply(this,arguments)}function p(){return v.apply(this,arguments)}function v(){return v=d(r.a.mark((function e(){var t,a,n,o,s;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return t="".concat("/","mondo_ids.txt"),e.next=3,i.a.get(t);case 3:return a=e.sent,e.prev=4,n=[],o=new Date,s=o.getMonth(),a.data.split("\n").forEach((function(e){var t=e.split("\t"),a=t[0],r=t[1],o=new Date(r).getUTCMonth();o===s&&(e={},e.mondo=a,e.date=Date.parse(r),n.push(e))})),e.abrupt("return",n);case 12:return e.prev=12,e.t0=e["catch"](4),e.abrupt("return",[]);case 15:case"end":return e.stop()}}),e,null,[[4,12]])}))),v.apply(this,arguments)}},"64bd":function(e,t,a){e.exports=a.p+"img/jackson.7b1eed8a.png"},6533:function(e,t,a){},6732:function(e,t,a){e.exports=a.p+"img/lbnl.29e333c2.jpeg"},"6aa4":function(e,t,a){},"6b39":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view other-tools px-5"},[a("h2",{staticClass:"page-title"},[e._v("Other Resources")]),a("div",{staticClass:"card"},[a("h4",[e._v("Exomiser")]),a("p",[a("a",{attrs:{target:"_blank",href:"https://exomiser.monarchinitiative.org/exomiser/"}},[e._v("Exomiser")]),e._v(" is a Java program that functionally annotates variants from whole-exome sequencing data starting from a VCF file (version 4) and a set of phenotypes encoded using the "),a("a",{attrs:{target:"_blank",href:"https://hpo.jax.org/app/"}},[e._v("Human Phenotype Ontology")]),e._v(". The functional annotation of variants is based on Jannovar and uses UCSC KnownGene or RefSeq transcript definitions and hg19 or hg38 genomic coordinates. Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity, quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data is extracted from the dbNSFP resource. Cross-species phenotype comparisons are powered by the OWLSim algorithm and use human, mouse, and fish phenotypes. Exomiser is available for download "),a("a",{attrs:{target:"_blank",href:"https://github.com/exomiser/Exomiser"}},[e._v(" here")]),e._v(". A paper describing Exomiser is available "),a("a",{attrs:{target:"_blank",href:"http://genome.cshlp.org/content/24/2/340.long"}},[e._v("here")]),e._v(". ")])]),a("div",{staticClass:"card"},[a("h4",[e._v("LIRICAL")]),a("p",[a("a",{attrs:{target:"_blank",href:"https://lirical.readthedocs.io/en/latest/"}},[e._v("LIRICAL")]),e._v(" is Java software that provides prioritization of candidate diseases using a likelihood ratio statistical framework. LIRICAL takes as input a set of phenotypic abnormalities encoded using the "),a("a",{attrs:{target:"_blank",href:"https://hpo.jax.org/app/"}},[e._v("Human Phenotype Ontology")]),e._v(" and, optionally, a set of genome variant calls from a VCF file. For each candidate disease, the likelihood ratio for each phenotype and genotype input by the user is calculated. A ranked list of candidate diseases is output, along with posterior probabilities and likelihood ratios for each phenotype and genotype, as well as a post-test probability for each candidate disease. ")])]),a("div",{staticClass:"card"},[a("h4",[e._v("PhenIX")]),a("p",[e._v(" Phenotypic Interpretation of eXomes is a pipeline for prioritizing candidate genes in exomes or NGS panels with comprehensive coverage of human Mendelian disease genes. It ranks genes based on predicted variant pathogenicity as well as phenotypic similarity of diseases associated with the genes harboring these variants to the phenotypic profile of the individual being investigated. PhenIX requires a VCF file mapped to hg19/Gchr37, as well as a list of HPO terms representing the phenotype observed in the patient. PhenIX is no longer under active development, and most of its functionality has been merged into Exomiser. ")])]),a("div",{staticClass:"card"},[a("h4",[e._v("ExomeWalker")]),a("p",[e._v(" ExomeWalker is a computational method to prioritize a set of candidates in exome sequencing projects that aim to identify novel Mendelian disease genes. This approach involves filtering a Variant Call Format (VCF) file according to a number of user-definable criteria."),a("br"),e._v(" Genes are prioritized according to a variant score (predicted pathogenicity, rarity, pattern of variants compatible with the assumed mode of inheritance) and to their vicinity to other genes that belong to the same phenotypic disease family within a protein protein interaction (PPI) network, using the Random-Walk method as described in "),a("a",{attrs:{target:"_blank",href:"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2427257/"}},[e._v("Köhler")]),e._v(" et al. (2008) to determine similarity within the PPI network on the basis of the global characteristics of the network. ExomeWalker is no longer under active development, and most of its functionality has been merged into Exomiser. ")])]),a("div",{staticClass:"card"},[a("h4",[e._v("Phenogrid")]),a("p",[a("a",{attrs:{target:"_blank",href:"https://github.com/monarch-initiative/phenogrid"}},[e._v("Phenogrid")]),e._v(" is a Javascript component that visualizes semantic similarity calculations provided by OWLSim, as provided through APIs from the Monarch Initiative. Given an input list of phenotypes and parameters specified in config/phenogrid_config.js indicating desired source of matching models (humans, model organisms, etc.), the phenogrid will call the Monarch API to get OWLSim results and render them in your web browser in data visualization. ")])]),a("div",{staticClass:"card"},[a("h4",[e._v("Dipper")]),a("p",[a("a",{attrs:{target:"_blank",href:"https://github.com/monarch-initiative/dipper"}},[e._v("Dipper")]),e._v(" is a Python package to generate RDF triples from common scientific resources. Dipper includes subpackages and modules to create graphical models of this data. Data generated from this pipeline can be used in a variety of ways downstream. We recommend loading the data into a triple store or graph database that is optimized for use with ontologies, such as BlazeGraph. We also maintain SciGraph, an application that loads RDF and OWL into Neo4J. Smaller files can be loaded into an ontology editor like Protege. ")])]),a("div",{staticClass:"card"},[a("h4",[e._v("HPO Case Annotator")]),a("p",[a("a",{attrs:{target:"_blank",href:"https://github.com/monarch-initiative/HpoCaseAnnotator"}},[e._v("The HPO Case Annotator (HCA)")]),e._v(" GUI is a Java app created with aim to make the biocuration of published pathogenic human variants easier. ")])])])])])])}],o=(a("9d6e"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},"6df1":function(e,t,a){"use strict";a("5a40")},"709a":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("div",{staticClass:"vue-markdown"},[n("html",[n("head"),n("body",[n("div",{staticClass:"container-fluid monarch-view services"},[n("h2",{staticClass:"page-title"},[e._v("Monarch Web Services")]),e._m(0),n("div",{staticClass:"row biolink"},[n("div",{staticClass:"offset-1 col-10 offset-desktop-2 col-desktop-7 banner"},[n("div",{staticClass:"logo-wrapper"},[n("img",{staticClass:"logo",attrs:{src:a("e3e1")}}),n("br"),n("br"),n("b-button",{staticClass:"documentation",attrs:{href:"https://api.monarchinitiative.org/api/",target:"_blank",variant:"outline-light"}},[e._v("Documentation")])],1),e._m(1)])])])])])])},r=[function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("div",{staticClass:"row architecture"},[n("div",{staticClass:"col-12 col-lg-6 offset-desktop-1 col-desktop-4 image"},[n("img",{staticClass:"services",attrs:{src:a("58a3")}})]),n("div",{staticClass:"col-12 col-lg-6 text"},[n("h5",[n("strong",[e._v("Architecture")])]),n("p",[e._v(" Structured and unstructured data sources are loaded into SciGraph via dipper (1) , our data ingest pipeline. A variety of ontologies are processed in the Monarch Integration Layer and are also loaded into SciGraph, resulting in a combined knowledge and data graph (2). Data is disseminated via SciGraph Services, wrapped around by the BioLink API (3). This lightweight API makes few assumptions, acting as a data access layer for all of Monarch’s Services. Users can interrogate Monarch using our tools for text annotation, navigating the ontologies, and matching phenotype profiles (4). ")])])])},function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"description"},[a("p",[e._v(" An API providing access to biologically and biomedically relevant entities, and the relationships between them. ")]),a("span",{staticClass:"entities"},[a("ul",[a("li",[e._v("genes, gene products, proteins")]),a("li",[e._v("diseases, phenotypes, traits, and clinical measurements")]),a("li",[e._v("pathways, biological process")]),a("li",[e._v("substances: small molecules, drugs, chemical entities")]),a("li",[e._v("biological and molecular roles and activities")]),a("li",[e._v(" genotypes, alleles, sequence variants; for plants, germplasms ")])])]),a("span",{staticClass:"entities"},[a("ul",[a("li",[e._v("environmental contexts and exposures")]),a("li",[e._v("individual organisms: patients, cohorts, model organisms")]),a("li",[e._v("cell lines and cell types")]),a("li",[e._v(" investigations: experiments, clinical trials, and 'natural experiments' ")]),a("li",[e._v("genomic features, phylogenies")]),a("li",[e._v("publications, ontology terms, database metadata")])])])])}],o=(a("543d"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},"733f":function(e,t,a){"use strict";a.d(t,"a",(function(){return r})),a.d(t,"b",(function(){return o}));var n={gene:"gene",variant:"variant","sequence feature":"variant",genotype:"genotype",phenotype:"phenotype",disease:"disease",pathway:"pathway","anatomical entity":"anatomy",publication:"publication",case:"case","sequence featurevariant":"variant"};function r(e){var t=["cell line","organism"],a=e.reduce((function(e,t){var a=n[t];return a&&(e[a]=a),e}),{}),r=Object.keys(a)[0],o=!1;return o=r?a.gene||a.variant||a[r]:e.filter((function(e){return t.includes(e)})).length>0?"model":e[0],o}function o(e){return n[e]}},"7ab0":function(e,t,a){"use strict";a("90a5")},"7b3b":function(e,t,a){},"7dd7":function(e,t){function a(e){var t=e.renderer.rules.link_open||function(e,t,a,n,r){return r.renderToken(e,t,a)},a=e.renderer.rules.link_close||function(e,t,a,n,r){return r.renderToken(e,t,a)},n=!1;e.renderer.rules.link_open=function(e,a,r,o,i){var s="_blank",l="noopener noreferrer",c=null,d=e[a].attrIndex("href");if(d>=0){var u=e[a].attrs[d][1];u.match(/^#/)?(s="_self",l=""):u.match(/^https?:\/\//)||(n=!0,s="_self",l="",c='\n<router-link\nto="'.concat(u,'">\n'))}var h=e[a].attrIndex("target");h<0?e[a].attrPush(["target",s]):e[a].attrs[h][1]=s;var p=e[a].attrIndex("rel");return p<0?e[a].attrPush(["rel",l]):e[a].attrs[p][1]=l,c||(c=t(e,a,r,o,i)),c},e.renderer.rules.link_close=function(e,t,r,o,i){var s=a(e,t,r,o,i);return n&&(n=!1,s="</router-link>"),s},e.renderer.rules.blockquote_open=function(){return'<blockquote class="blockquote">'}}e.exports={applyLinkHandlers:a}},8322:function(e,t,a){"use strict";a("b8af")},"85fb":function(e,t,a){},"88b5":function(e,t,a){},"90a5":function(e,t,a){},9212:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_DISEASE.bafbacff.png"},9276:function(e,t,a){},"92ec":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("div",{staticClass:"vue-markdown"},[n("html",[n("head"),n("body",[n("div",{staticClass:"container-fluid monarch-view monarch-about-view"},[n("h2",{staticClass:"page-title"},[e._v("The Monarch Initiative")]),n("div",{staticClass:"top-section col-12"},[n("p",[e._v(" The Monarch Initiative is an integrative data and analytic platform connecting phenotypes to genotypes across species, bridging basic and applied research with semantics-based analysis. The correlation of phenotypic outcomes and disease with genetic variation and environmental factors is a core pursuit in biology and biomedicine. We have created or currently contribute to many essential bio-ontologies that together enable sophisticated and semantically integrated computational analysis across gene, genotype, variant, disease, and phenotype data. We have developed algorithms and tools that are in use by multiple communities for tasks including the identification of animal models of human disease through phenotypic similarity, phenotype-driven computational support for differential diagnostics, and translational research. ")])]),n("div",{staticClass:"row"},[n("div",{staticClass:"col-12 col-lg-6"},[n("figure",{staticClass:"cross-species"},[n("img",{attrs:{src:a("3d49")}}),n("figcaption",{staticClass:"phenotype-coverage"},[e._v(" Fig 1. Model organisms provide key insight into phenotypic manifestations of human coding genes. Human coding genes with disease/phenotype-causing mutations are shown on the left; model organism orthologs of human coding genes with associated phenotypes are on the right. By including just 5 species, we can boost coverage by 61%, up to ~82% of human protein coding genes reported by the HUGO Gene Nomenclature Committee (HGNC). There are 33 sources of data integrated within the Monarch Initiative. ")])])]),n("div",{staticClass:"col-12 col-lg-5"},[n("figure",{staticClass:"comparison"},[n("img",{attrs:{src:a("0c1b")}}),n("figcaption",[e._v(" Fig 2. Comparison of PAX6 abnormal phenotypes in human and model organism eyes. (Image from "),n("a",{attrs:{href:"http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000247",target:"__blank"}},[e._v("Washington et al, 2009)")])])])])]),n("div",{staticClass:"goals-banner offset-2 col-8"},[n("h3",{attrs:{id:"our-goals"}},[n("strong",[e._v("Monarch's Vision")])]),n("div",{attrs:{id:"our-goals-list"}},[n("p",[e._v(" Integrate, align, and re-distribute cross-species gene, genotype, variant, disease, and phenotype data ")]),n("p",[e._v("Provide a portal for exploration of phenotype-based similarity")]),n("p",[e._v(" Facilitate identification of animal models of human disease through phenotypic similarity ")]),n("p",[e._v("Enable quantitative comparison of cross-species phenotypes")]),n("p",[e._v("Develop embeddable widgets for data exploration")]),n("p",[e._v("Influence genotype and phenotype reporting standards")]),n("p",[e._v("Improve ontologies to better curate genotype-phenotype data")])])]),n("br"),n("br"),n("p",[e._v(" We lead the development of the "),n("a",{attrs:{href:"https://hpo.jax.org/"}},[e._v("Human Phenotype Ontology")]),e._v(", which is used across the world for genomic diagnostics in genetic disease and other areas. We are a Driver Project for the "),n("a",{attrs:{href:"https://www.ga4gh.org/",target:"__blank"}},[e._v("Global Alliance for Genomics and Health")]),e._v(" (GA4GH), and are major contributors to the development of genomics standards within GA4GH. Additionally, we have developed "),n("a",{attrs:{href:"http://obofoundry.org/ontology/mondo.html",target:"__blank"}},[e._v("Mondo")]),e._v(", a unified disease ontology that represents the most comprehensive integration of disease entities ever achieved. ")]),n("div",{staticClass:"row"},[n("div",{staticClass:"col-12 col-lg-6"},[n("figure",{staticClass:"ecosystems"},[n("img",{attrs:{src:a("50dc")}}),n("figcaption",[e._v(" Fig 3. The "),n("a",{attrs:{href:"https://hpo.jax.org",target:"__blank"}},[e._v("Human Phenotype Ontology")]),e._v(" (HPO) provides a standardized vocabulary of phenotypic abnormalities encountered in human disease. "),n("a",{attrs:{href:"https://pubmed.ncbi.nlm.nih.gov/30476213/",target:"__blank"}},[e._v("More about HPO...")])])])]),n("div",{staticClass:"col-12 col-lg-6"},[n("figure",{staticClass:"ecosystems phenopackets"},[n("img",{attrs:{src:a("a07b")}}),n("figcaption",[e._v(" Fig 4. "),n("a",{attrs:{href:"http://phenopackets.org/",target:"__blank"}},[e._v("Phenopackets")]),e._v(" is a standard exchange format for phenotypes and environmental factors. This packet of phenotype data can be used anywhere, and can be written by anyone. This work is being done in collaboration with the "),n("a",{attrs:{href:"https://www.ga4gh.org/",target:"__blank"}},[e._v("Global Alliance for Genomics and Health (GA4GH)")]),e._v(". ")])])])])])])])])}],o={name:"AboutMonarch",components:{}},i=o,s=(a("5aaf"),a("2877")),l=Object(s["a"])(i,n,r,!1,null,null,null);t["default"]=l.exports},"94b4":function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_ORTHOLOG_PHENOTYPE.b053c4ec.png"},"98eb":function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"intro"},[e._m(0),a("p",[a("embedded-autocomplete",{attrs:{"home-search":!0,"show-search-button":!0,"auto-focus":!0}})],1)])])])])},r=[function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("p",[n("img",{attrs:{src:a("626f"),alt:""}})])}],o=a("5631"),i={name:"Home",components:{"embedded-autocomplete":o["a"]}},s=i,l=(a("8322"),a("2877")),c=Object(l["a"])(s,n,r,!1,null,null,null);t["default"]=c.exports},"9c0c":function(e,t,a){},"9d6e":function(e,t,a){"use strict";a("d711")},"9e25":function(e,t,a){"use strict";a("bb3b")},"9ea1":function(e,t,a){e.exports=a.p+"img/health-ai.f69127f8.png"},"9efd":function(e,t,a){"use strict";a("b9a0")},"9f01":function(e,t,a){"use strict";a("9276")},a07b:function(e,t,a){e.exports=a.p+"img/phenopackets-ecosystem2.129bc61d.png"},a355:function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,n=e._self._c||t;return n("div",{staticClass:"vue-markdown"},[n("html",[n("head"),n("body",[n("div",{staticClass:"home-page-section partners-section"},[n("h5",{attrs:{id:"the-monarch-initiative-is-a-collaboration-between"}},[e._v("The "),n("strong",[e._v("Monarch Initiative")]),e._v(" is a collaboration between: "),n("a",{staticClass:"header-anchor",attrs:{href:"#the-monarch-initiative-is-a-collaboration-between",target:"_self",rel:""}},[e._v("🔗")])]),n("br"),n("div",{staticClass:"row"},[n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://oregonstate.edu/",target:"__blank"}},[n("img",{attrs:{src:a("c457"),alt:"osu"}})])]),n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://www.jax.org/",target:"__blank"}},[n("img",{attrs:{src:a("64bd"),alt:"jax"}})])]),n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://www.lbl.gov/",target:"__blank"}},[n("img",{attrs:{src:a("6732"),alt:"lbl"}})])]),n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://medschool.cuanschutz.edu/ai",target:"__blank"}},[n("img",{attrs:{src:a("9ea1"),alt:"Center for Health AI"}})])]),n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://www.qmul.ac.uk/smd/",target:"__blank"}},[n("img",{attrs:{src:a("fbd3"),alt:"qmul"}})])]),n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://www.ebi.ac.uk/",target:"__blank"}},[n("img",{attrs:{src:a("0150"),alt:"ebi"}})])]),n("div",{staticClass:"col-sm-6 col-md-4 collab"},[n("a",{attrs:{href:"https://renci.org/",target:"__blank"}},[n("img",{attrs:{src:a("1231"),alt:"renci"}})])])]),n("p",[e._v(" Monarch is supported generously by a NIH Office of the Director Grant #5R24OD011883, as well as by NIH-UDP: HHSN268201350036C, HHSN268201400093P, NCI/Leidos #15X143. We are grateful to the many "),n("a",{attrs:{href:"/about/data-sources"}},[e._v("original sources of our data")]),e._v(" for allowing Monarch to integrate them in this way. For more information, see "),n("a",{attrs:{href:"/about/licensing"}},[e._v("About Monarch Licensing")]),e._v(". ")])])])])])}],o=(a("04b5"),a("2877")),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},a35e:function(e,t,a){e.exports=a.p+"img/monarch-logo-white.418cd8cf.png"},a545:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_CELL_LINE.81a6ad77.png"},a9b9:function(e,t,a){"use strict";a("ed50")},ac4d:function(e,t,a){},ad30:function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view monarch-team-view"},[e._m(0),a("div",{staticClass:"card p-3 bg-light"},[a("h3",[e._v("Participating Institutions")]),a("div",{staticClass:"teamtoc"},[a("dl",[e._l(e.institutions,(function(t){return[a("dt",[a("a",{attrs:{href:"#"+t.id}},[e._v(e._s(t.name))])]),a("dd",[e._v(e._s(t.peopleNames.join(", ")))])]}))],2)])]),e._l(e.institutions,(function(t){return a("div",{staticClass:"institution-target",attrs:{id:t.id}},[a("h3",[a("a",{attrs:{href:t.website,target:"_blank",rel:"noreferrer"}},[a("img",{staticClass:"teamlogo",attrs:{src:t.logo}}),e._v(" "+e._s(t.name)+" ")])]),e._l(t.people,(function(t){return a("div",{staticClass:"teammember"},[t.alumni?[a("div",{staticClass:"teammember"},[a("div",{staticClass:"membername"},[e._v(" "+e._s(t.name)+" (alumni "+e._s(t.title)+") ")])])]:[a("div",{staticClass:"memberhead"},[a("div",{staticClass:"membername"},[e._v(e._s(t.name))]),a("div",{staticClass:"membertitle"},[e._v(e._s(t.title))])]),a("img",{staticClass:"memberpicture",attrs:{src:t.picture}}),a("div",{staticClass:"clearfix"}),t.bio?a(e.markdownToComponent(t.bio),{tag:"div",staticClass:"memberbio"}):e._e()]],2)}))],2)}))],2)])])])},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("h2",{attrs:{id:"monarch-team-and-collaborators"}},[e._v("Monarch Team and Collaborators "),a("a",{staticClass:"header-anchor",attrs:{href:"#monarch-team-and-collaborators",target:"_self",rel:""}},[e._v("🔗")])])}],o=a("a34a"),i=a.n(o),s=a("63e8"),l=a("d4cd"),c=a.n(l),d=a("7dd7");function u(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function h(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){u(o,n,r,i,s,"next",e)}function s(e){u(o,n,r,i,s,"throw",e)}i(void 0)}))}}var p={components:{},data:function(){return{institutions:[],markdown:null,inRouterLink:!1}},created:function(){var e=new c.a;this.markdown=e,Object(d["applyLinkHandlers"])(e)},mounted:function(){var e=this;return h(i.a.mark((function t(){return i.a.wrap((function(t){while(1)switch(t.prev=t.next){case 0:return t.next=2,Object(s["b"])();case 2:e.institutions=t.sent.institutions;case 3:case"end":return t.stop()}}),t)})))()},methods:{markdownToComponent:function(e){var t=this.markdown.render(e||"");return{template:"<div>".concat(t,"</div>")}}}},v=p,m=(a("c014"),a("2877")),g=Object(m["a"])(v,n,r,!1,null,null,null);t["default"]=g.exports},adad:function(e,t,a){"use strict";a.r(t);var n=function(){var e=this,t=e.$createElement;e._self._c;return e._m(0)},r=[function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("div",{staticClass:"vue-markdown"},[a("html",[a("head"),a("body",[a("div",{staticClass:"container-fluid monarch-view monarch-disclaimer"},[a("h2",{staticClass:"page-title"},[e._v("Monarch Disclaimer")]),a("h4",[e._v("Introduction")]),a("p",[e._v(' You are here identified as a recipient of one or more Monarch Initiative software and/or data collection tools, including but not limited to the Exomiser, (hereby referred to as “Monarch Tools”) developed by Oregon Health and Science University ("OHSU"). OHSU provides you the Monarch Tools under the following limitations. ')]),a("h4",[e._v("Use of Hotjar")]),a("p",[e._v(" This web application tracks user behavior such as navigation and clicks. We do this through the use of Hotjar, a background script and service that takes significant steps to remove identifying information from the data it collects to ensure anonymity. To learn more about how Hotjar maintains privacy and complies with relevant privacy laws, "),a("a",{attrs:{href:"https://www.hotjar.com/legal/compliance/gdpr-commitment/"}},[e._v("see their explanation here")]),e._v(". ")]),a("p",[e._v(" Furthermore, the Monarch Initiative pledges to never inspect or disclose any private, identifying information collected by Hotjar, and to only use it for the purposes of improving the Monarch web app. ")]),a("h4",[e._v("Disclaimer")]),a("p",[e._v(' YOU ACKNOWLEDGE THAT THE MONARCH TOOLS, INCLUDING WITHOUT LIMITATION, THE DATABASE OF INFORMATION CONTAINED THEREIN, IS EXPERIMENTAL AND ACADEMIC IN NATURE, AND IS NOT LICENSED BY THE U.S. FOOD AND DRUG ADMINISTRATION OR ANY OTHER REGULATORY BODY. THE MONARCH TOOLS ARE PROVIDED ON AN "AS IS" AND “AS AVAILABLE” BASIS WITH ALL FAULTS, WITHOUT WARRANTY OR REPRESENTATION OF ANY KIND, EXPRESS OR IMPLIED. OHSU DISCLAIMS ALL WARRANTIES AND REPRESENTATIONS, EXPRESS OR IMPLIED, WITH RESPECT TO THE MONARCH TOOLS, INCLUDING WITHOUT LIMITATION, ANY WARRANTIES OF TITLE, MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT OF THIRD PARTY RIGHTS. THIS DISCLAIMER APPLIES TO THE FULLEST EXTENT POSSIBLE. OHSU DOES NOT WARRANT THAT THE MONARCH TOOLS WILL MEET YOUR REQUIREMENTS OR THAT THE USE OF THE MONARCH TOOLS WILL BE FREE OF INFECTION OR VIRUSES, ERROR-FREE, UNINTERRUPTED, SECURE OR WILL PRODUCE ACCURATE RESULTS. YOU SHALL BEAR TOTAL AND EXCLUSIVE RESPONSIBILITY AND RISK FOR THE USE OF THE MONARCH TOOLS. YOU SHALL BE SOLELY RESPONSIBLE FOR ANY RESULTING DAMAGE TO YOUR COMPUTER SYSTEMS OR LOSS OF DATA. IF YOU ARE DISSATISFIED WITH THE MONARCH TOOLS, OR WITH ANY OF THESE TERMS, YOUR SOLE AND EXCLUSIVE REMEDY IS TO DISCONTINUE USING THE MONARCH TOOLS. ')]),a("h4",[e._v("Limitation of Liability")]),a("p",[e._v(' IN NO EVENT SHALL OHSU, OR THEIR RESPECTIVE DIRECTORS, OFFICERS, EMPLOYEES, AGENTS, LICENSEES, LICENSORS, CONTRIBUTORS, FOUNDERS OR ADVISORS, OR ANY OTHER PERSON WHO HAS BEEN INVOLVED IN THE CREATION OR PRODUCTION OF THE MONARCH TOOLS (EACH A “RELEASED PARTY” AND COLLECTIVELY, "RELEASED PARTIES") BE LIABLE FOR ANY CLAIMS, DEMANDS, LIABILITIES, COSTS, LOSSES, DAMAGES OR EXPENSES (INCLUDING WITHOUT LIMITATION LEGAL COSTS AND ATTORNEYS’ FEES) CAUSED TO OR SUFFERED BY ANY PERSONS OR ENTITIES (INCLUDING WITHOUT LIMITATION, YOU), THAT DIRECTLY OR INDIRECTLY ARISE OUT OF OR RESULT FROM THE USE OF, THE INABILITY TO USE, OR ERRORS, OMISSIONS, INTERRUPTIONS OR OTHER INACCURACIES IN, THE MONARCH TOOLS OR VIOLATION OF ANY RIGHTS OF ANY THIRD PARTY. WITHOUT LIMITING THE FOREGOING, THE RELEASED PARTIES SHALL NOT BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, CONSEQUENTIAL, SPECIAL, PUNITIVE OR EXEMPLARY DAMAGES, OR ANY OTHER DAMAGES OF ANY KIND, INCLUDING WITHOUT LIMITATION, DAMAGES FOR LOSS OF BUSINESS, LOSS OF DATA, LOST PROFITS OR BUSINESS INTERRUPTION, UNDER ANY CONTRACT, NEGLIGENCE, STRICT LIABILITY OR OTHER THEORY, EVEN IF A RELEASED PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES. THIS LIMITATION OF LIABILITY APPLIES TO THE FULLEST EXTENT POSSIBLE. ')]),a("h4",[e._v("Intended use")]),a("p",[e._v(" MonarchInitiative.org is not intended for direct diagnostic use or medical decision-making. If you have any health questions about the information contained on this website, please contact a medical professional. ")]),a("h4",[e._v("Variant interpretation")]),a("p",[e._v(" The Monarch Initiative sources variant interpretations as provided by the original source, with no consolidation of results, and always has a link out to the original reference. While we display the variants and interpretations as originally reported in the variant databases, some variants in the same genes have been reported on different transcripts. Whenever possible, Monarch will attempt to map all variants in a particular gene to a common transcript to aid in viewing and organizing the variants. However, to aid in variant searches, we will also continue to report the original mapping information. All interpretations were performed by the original submitters and have not been modified in any way. As such, there might be discordant interpretations for the same variant from different archives. For further information or to request a review of the variant, please contact the submitter of the original reference. Variant interpretations are not static as they are based on the current understanding of the variant at the time it was observed in a patient or in the literature. Any reported interpretation may not be up-to-date, and some information may not be reported. Many interpretations will change over time as more information about the variant becomes available. If you have any questions about a variant interpretation or would like use any variant interpretations for publication purposes, please contact the submitter of the original reference. As always, we value your feedback. Please email us if you have any problems, concerns, or suggestions. ")])])])])])}],o=a("2877"),i={},s=Object(o["a"])(i,n,r,!1,null,null,null);t["default"]=s.exports},af03:function(e,t,a){},b035:function(e,t,a){"use strict";a("6533")},b279:function(e,t,a){"use strict";a.d(t,"b",(function(){return p})),a.d(t,"c",(function(){return v})),a.d(t,"a",(function(){return m})),a.d(t,"e",(function(){return g})),a.d(t,"d",(function(){return f}));var n=a("c46f"),r=a("5e22"),o=a.n(r),i={animalqtldb:{AQTLTrait:"http://identifiers.org/animalqtltrait/[reference]",horseQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/EC/qdetails?QTL_ID=[reference]",cattleQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/BT/qdetails?QTL_ID=[reference]",catfishQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/IP/qdetails?QTL_ID=[reference]",chickenQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/GG/qdetails?QTL_ID=[reference]",pigQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/SS/qdetails?QTL_ID=[reference]",rainbow_troutQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/OM/qdetails?QTL_ID=[reference]",sheepQTL:"https://www.animalgenome.org/cgi-bin/QTLdb/OA/qdetails?QTL_ID=[reference]",AQTLPub:"https://www.animalgenome.org/cgi-bin/QTLdb/BT/qabstract?PUBMED_ID=[reference]"},bfo:{BFO:"http://purl.obolibrary.org/obo/BFO_[reference]"},cito:{cito:"http://purl.org/spar/cito/[reference]"},chebi:{CHEBI:"http://purl.obolibrary.org/obo/CHEBI_[reference]"},chr:{CHR:"http://purl.obolibrary.org/obo/CHR_[reference]"},cl:{CL:"http://purl.obolibrary.org/obo/CL_[reference]"},clo:{CLO:"http://purl.obolibrary.org/obo/CLO_[reference]"},cmo:{CMO:"http://purl.obolibrary.org/obo/CMO_[reference]"},"edam-data":{"EDAM-DATA":"http://edamontology.org/data_[reference]"},dc_cl:{DC_CL:"http://purl.obolibrary.org/obo/DC_CL[reference]"},decipher:{DECIPHER:"https://decipher.sanger.ac.uk/syndrome/[reference]"},doid:{DOID:"http://purl.obolibrary.org/obo/DOID_[reference]"},eco:{ECO:"http://purl.obolibrary.org/obo/ECO_[reference]"},efo:{EFO:"http://www.ebi.ac.uk/efo/EFO_[reference]"},ensembl:{ENSEMBL:"http://ensembl.org/id/[reference]",EnsemblGenome:"http://www.ensemblgenomes.org/id/[reference]"},envo:{ENVO:"http://purl.obolibrary.org/obo/ENVO_[reference]"},ero:{ERO:"http://purl.obolibrary.org/obo/ERO_[reference]"},faldo:{faldo:"http://biohackathon.org/resource/faldo#[reference]"},fbcv:{FBcv:"http://purl.obolibrary.org/obo/FBcv_[reference]"},fbbt:{FBbt:"http://purl.obolibrary.org/obo/FBbt_[reference]"},fbdv:{FBdv:"http://purl.obolibrary.org/obo/FBdv_[reference]"},geno:{GENO:"http://purl.obolibrary.org/obo/GENO_[reference]"},go:{GO:"http://amigo.geneontology.org/amigo/term/GO:[reference]",GO_REF:"http://www.geneontology.org/cgi-bin/references.cgi#GO_REF:[reference]",PAINT_REF:"http://www.geneontology.org/gene-associations/submission/paint/[reference]"},hpoa:{HP:"https://hpo.jax.org/app/browse/term/HP:[reference]",OMIM:"http://omim.org/entry/[reference]",OMIMPS:"http://www.omim.org/phenotypicSeries/[reference]",ORPHA:"https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=[reference]"},iao:{IAO:"http://purl.obolibrary.org/obo/IAO_[reference]"},kegg:{"KEGG-ds":"http://purl.obolibrary.org/KEGG-ds_[reference]","KEGG-ko":"http://www.kegg.jp/dbget-bin/www_bget?ko:[reference]","KEGG-path":"http://www.kegg.jp/dbget-bin/www_bget?path:[reference]","KEGG-img":"http://www.genome.jp/kegg/pathway/map/map[reference]","KEGG-hsa":"http://www.kegg.jp/dbget-bin/www_bget?hsa:[reference]"},lpt:{LPT:"http://purl.obolibrary.org/obo/LPT_[reference]"},ma:{MA:"http://purl.obolibrary.org/obo/MA_[reference]"},medgen:{MedGen:"http://www.ncbi.nlm.nih.gov/medgen/[reference]"},mpath:{MPATH:"http://purl.obolibrary.org/obo/MPATH_[reference]"},nbo:{NBO:"http://purl.obolibrary.org/obo/NBO_[reference]"},oba:{OBA:"http://purl.obolibrary.org/obo/OBA_[reference]"},omia:{OMIA:"https://omia.org/OMIA[reference]","OMIA-breed":"https://monarchinitiative.org/model/OMIA-breed:[reference]"},omim:{OMIM:"http://omim.org/entry/[reference]",OMIMPS:"http://www.omim.org/phenotypicSeries/[reference]"},orphanet:{ORPHA:"https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=[reference]",Orphanet:"https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=[reference]"},pato:{PATO:"http://purl.obolibrary.org/obo/PATO_[reference]"},pco:{PCO:"http://purl.obolibrary.org/obo/PCO_[reference]"},pr:{PR:"http://purl.obolibrary.org/obo/PR_[reference]"},pw:{PW:"http://purl.obolibrary.org/obo/PW_[reference]"},ro:{RO:"http://purl.obolibrary.org/obo/RO_[reference]"},sio:{SIO:"http://semanticscience.org/resource/SIO_[reference]"},snomed:{SNOMED:"http://purl.obolibrary.org/obo/SNOMED_[reference]"},so:{SO:"http://purl.obolibrary.org/obo/SO_[reference]"},stato:{STATO:"http://purl.obolibrary.org/obo/STATO_[reference]"},uberon:{UBERON:"http://purl.obolibrary.org/obo/UBERON_[reference]"},umls:{UMLS:"http://linkedlifedata.com/resource/umls/id/[reference]"},uo:{UO:"http://purl.obolibrary.org/obo/UO_[reference]"},vt:{VT:"http://purl.obolibrary.org/obo/VT_[reference]"},wbphenotype:{WBPhenotype:"http://purl.obolibrary.org/obo/WBPhenotype_[reference]"},xco:{XCO:"http://purl.obolibrary.org/obo/XCO_[reference]"},zfa:{ZFA:"http://purl.obolibrary.org/obo/ZFA_[reference]"},zfs:{ZFS:"http://purl.obolibrary.org/obo/ZFS_[reference]"},zp:{ZP:"http://purl.obolibrary.org/obo/ZP_[reference]"},wbbt:{WBbt:"http://purl.obolibrary.org/obo/WBbt_[reference]"},emapa:{EMAPA:"http://purl.obolibrary.org/obo/EMAPA_[reference]"},xao:{XAO:"http://purl.obolibrary.org/obo/XAO_[reference]"},ncit:{NCIT:"http://purl.obolibrary.org/obo/NCIT_[reference]"},sepio:{SEPIO:"http://purl.obolibrary.org/obo/SEPIO_[reference]"},vivo:{VIVO:"http://vivoweb.org/ontology/core#[reference]"},vfb:{vfb:"http://virtualflybrain.org/reports/[reference]"},eom:{EOM:"https://elementsofmorphology.nih.gov/index.cgi?tid=[reference]"},doi:{DOI:"http://dx.doi.org/[reference]"},genereviews:{GeneReviews:"http://www.ncbi.nlm.nih.gov/books/[reference]"},isbn:{ISBN:"https://monarchinitiative.org/ISBN_[reference]"},"isbn-10":{"ISBN-10":"https://monarchinitiative.org/ISBN10_[reference]"},"isbn-13":{"ISBN-13":"https://monarchinitiative.org/ISBN13_[reference]"},mpd:{MPD:"https://phenome.jax.org/[reference]","MPD-assay":"https://phenome.jax.org/db/qp?rtn=views/catlines&keymeas=[reference]","MPD-strain":"http://phenome.jax.org/db/q?rtn=strains/details&strainid=[reference]"},pmid:{PMID:"http://www.ncbi.nlm.nih.gov/pubmed/[reference]"},pmcid:{PMCID:"http://www.ncbi.nlm.nih.gov/pmc/[reference]"},aspgd_ref:{AspGD_REF:"http://www.aspergillusgenome.org/cgi-bin/reference/reference.pl?dbid=[reference]"},apo:{APO:"http://purl.obolibrary.org/obo/APO_[reference]"},apb:{APB:"http://pb.apf.edu.au/phenbank/strain.html?id=[reference]"},cmmr:{CMMR:"http://www.cmmr.ca/order.php?t=m&id=[reference]"},coriell:{Coriell:"https://catalog.coriell.org/0/Sections/Search/Sample_Detail.aspx?Ref=[reference]",CoriellCollection:"https://catalog.coriell.org/1/[reference]",CoriellFamily:"https://catalog.coriell.org/0/Sections/BrowseCatalog/FamilyTypeSubDetail.aspx?fam=[reference]",CoriellIndividual:"https://catalog.coriell.org/Search?q=[reference]"},dbsnpindividual:{dbSNPIndividual:"http://www.ncbi.nlm.nih.gov/SNP/snp_ind.cgi?ind_id=[reference]"},emma:{EMMA:"https://www.infrafrontier.eu/search?keyword=EM:[reference]"},jax:{JAX:"http://jaxmice.jax.org/strain/[reference]"},mmrrc:{MMRRC:"https://www.mmrrc.org/catalog/sds.php?mmrrc_id=[reference]"},mugen:{MUGEN:"http://bioit.fleming.gr/mugen/Controller?workflow=ViewModel&expand_all=true&name_begins=model.block&eid=[reference]"},ncimr:{NCIMR:"https://mouse.ncifcrf.gov/available_details.asp?ID=[reference]"},rbrc:{RBRC:"http://www2.brc.riken.jp/lab/animal/detail.php?brc_no=RBRC[reference]"},ncbiassembly:{NCBIAssembly:"https://www.ncbi.nlm.nih.gov/assembly?term=[reference]"},ncbigenome:{NCBIGenome:"https://www.ncbi.nlm.nih.gov/genome/[reference]"},ncbitaxon:{NCBITaxon:"http://purl.obolibrary.org/obo/NCBITaxon_[reference]"},ucscbands:{UCSC:"ftp://hgdownload.cse.ucsc.edu/goldenPath/[reference]",UCSCBuild:"http://genome.ucsc.edu/cgi-bin/hgGateway?db=[reference]"},homologene:{HOMOLOGENE:"http://www.ncbi.nlm.nih.gov/homologene/[reference]"},panther:{PANTHER:"http://www.pantherdb.org/panther/family.do?clsAccession=[reference]"},clinvar:{ClinVar:"http://www.ncbi.nlm.nih.gov/clinvar/[reference]",ClinVarVariant:"http://www.ncbi.nlm.nih.gov/clinvar/variation/[reference]",ClinVarSubmitters:"http://www.ncbi.nlm.nih.gov/clinvar/submitters/[reference]"},cosmic:{COSMIC:"http://cancer.sanger.ac.uk/cosmic/mutation/overview?id=[reference]"},hgmd:{HGMD:"http://www.hgmd.cf.ac.uk/ac/gene.php?gene=[reference]"},dbsnp:{dbSNP:"http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=[reference]"},dbvar:{dbVar:"http://www.ncbi.nlm.nih.gov/dbvar/[reference]"},gwascatalog:{GWAS:"https://www.ebi.ac.uk/gwas/variants/[reference]",dbSNP:"https://www.ebi.ac.uk/gwas/variants/[reference]",ENSEMBL:"https://www.ebi.ac.uk/gwas/genes/[label]"},react:{REACT:"http://www.reactome.org/PathwayBrowser/#/[reference]"},smpdb:{SMPDB:"http://smpdb.ca/view/[reference]"},aspgd:{AspGD:"http://www.aspergillusgenome.org/cgi-bin/locus.pl?dbid=[reference]"},biogrid:{BIOGRID:"http://thebiogrid.org/[reference]"},string:{home:"https://string-db.org"},ccds:{CCDS:"http://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi?REQUEST=CCDS&DATA=[reference]"},cgnc:{CGNC:"http://birdgenenames.org/cgnc/GeneReport?id=[reference]"},dictybase:{dictyBase:"http://dictybase.org/gene/[reference]"},ecogene:{EcoGene:"http://ecogene.org/gene/[reference]"},flybase:{FlyBase:"http://flybase.org/reports/[reference]"},genatlas:{Genatlas:"http://genatlas.medecine.univ-paris5.fr/fiche.php?symbol=[reference]"},genbank:{GenBank:"http://www.ncbi.nlm.nih.gov/nuccore/[reference]"},hgnc:{HGNC:"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:[reference]"},mgi:{MGI:"http://www.informatics.jax.org/accession/MGI:[reference]",MP:"http://www.informatics.jax.org/vocab/mp_ontology/MP:[reference]",J:"http://www.informatics.jax.org/reference/J:[reference]"},impc:{IMPC:"http://www.mousephenotype.org/data/genes/[reference]",MGI:"https://www.mousephenotype.org/data/genes/MGI:[reference]",MP:"https://www.mousephenotype.org/data/phenotypes/MP:[reference]","IMPRESS-procedure":"https://www.mousephenotype.org/impress/procedures/[reference]","IMPRESS-protocol":"https://www.mousephenotype.org/impress/protocol/[reference]","IMPRESS-parameter":"https://www.mousephenotype.org/impress/parameterontologies/[reference]"},mirbase:{miRBase:"http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=[reference]"},ncbigene:{NCBIGene:"https://www.ncbi.nlm.nih.gov/gene/[reference]"},pombase:{PomBase:"https://www.pombase.org/spombe/result/[reference]"},refseq:{RefSeq:"http://www.ncbi.nlm.nih.gov/refseq/?term=[reference]"},rgd:{RGD:"http://rgd.mcw.edu/rgdweb/report/gene/main.html?id=[reference]",RGDRef:"http://rgd.mcw.edu/rgdweb/report/reference/main.html?id=[reference]"},sgd:{SGD:"https://www.yeastgenome.org/locus/[reference]",SGD_REF:"https://www.yeastgenome.org/reference/[reference]"},tair:{TAIR:"https://www.arabidopsis.org/servlets/TairObject?type=locus&id=[reference]"},vgnc:{VGNC:"https://vertebrate.genenames.org/data/gene-symbol-report/#!/vgnc_id/[reference]"},wormbase:{WormBase:"https://www.wormbase.org/get?name=[reference]"},xenbase:{Xenbase:"http://www.xenbase.org/gene/showgene.do?method=display&geneId=[reference]"},zfin:{ZFIN:"http://zfin.org/[reference]"},ec:{EC:"https://www.enzyme-database.org/query.php?ec=[reference]"},hprd:{HPRD:"http://www.hprd.org/protein/[reference]"},ncbiprotein:{NCBIProtein:"http://www.ncbi.nlm.nih.gov/protein/[reference]"},pdb:{PDB:"http://www.ebi.ac.uk/pdbsum/[reference]"},swissprot:{SwissProt:"http://identifiers.org/SwissProt:[reference]"},trembl:{TrEMBL:"http://purl.uniprot.org/uniprot/[reference]"},uniprotkb:{UniProtKB:"http://identifiers.org/uniprot/[reference]"},iuphar:{IUPHAR:"http://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=[reference]"},cid:{CID:"http://pubchem.ncbi.nlm.nih.gov/compound/[reference]"},drugbank:{DrugBank:"http://www.drugbank.ca/drugs/[reference]"},oae:{OAE:"http://purl.obolibrary.org/obo/OAE_[reference]"},rxcui:{RXCUI:"http://purl.bioontology.org/ontology/RXNORM/[reference]"},meddra:{MEDDRA:"http://purl.bioontology.org/ontology/MEDDRA/[reference]"},fdadrug:{FDADrug:"http://www.fda.gov/Drugs/InformationOnDrugs/[reference]"},bt:{BT:"http://c.biothings.io/#[reference]"},unii:{UNII:"http://fdasis.nlm.nih.gov/srs/unii/[reference]"},ginas:{GINAS:"http://tripod.nih.gov/ginas/app/substance#[reference]"},hmdb:{HMDB:"http://www.hmdb.ca/metabolites/[reference]"},wikidata:{WD_Prop:"https://www.wikidata.org/wiki/Property:[reference]",WD_Entity:"https://www.wikidata.org/wiki/[reference]"}};function s(e,t){return h(e)||u(e,t)||c(e,t)||l()}function l(){throw new TypeError("Invalid attempt to destructure non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.")}function c(e,t){if(e){if("string"===typeof e)return d(e,t);var a=Object.prototype.toString.call(e).slice(8,-1);return"Object"===a&&e.constructor&&(a=e.constructor.name),"Map"===a||"Set"===a?Array.from(e):"Arguments"===a||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(a)?d(e,t):void 0}}function d(e,t){(null==t||t>e.length)&&(t=e.length);for(var a=0,n=new Array(t);a<t;a++)n[a]=e[a];return n}function u(e,t){var a=null==e?null:"undefined"!==typeof Symbol&&e[Symbol.iterator]||e["@@iterator"];if(null!=a){var n,r,o=[],i=!0,s=!1;try{for(a=a.call(e);!(i=(n=a.next()).done);i=!0)if(o.push(n.value),t&&o.length===t)break}catch(l){s=!0,r=l}finally{try{i||null==a["return"]||a["return"]()}finally{if(s)throw r}}return o}}function h(e){if(Array.isArray(e))return e}function p(e){return e.filter((function(e){return!e.id.startsWith("MONDO")})).map((function(e){var t="";if(e.id.startsWith("PMID")){var a=e.id.split(":")[1];t="http://www.ncbi.nlm.nih.gov/pubmed/".concat(a)}else t="/publication/".concat(e.id);return{id:e.id,label:e.label||e.id,url:t}}))}function v(e){return null===e&&(e=["http://purl.obolibrary.org/obo/mondo.owl"]),e=n["a"].uniq(e.map((function(e){return e.replace(/_?slim/,"")}))),e.map((function(e){return e.split("/").pop().replace("#","").split(".")[0].toLowerCase().replace("monarch","omia")}))}function m(e,t,a){var n=null,r=t.split(":"),o=s(r,2),l=o[0],c=o[1];return"OMIM"===l&&(c=c.replace(".","#")),e in i&&l in i[e]&&(n=i[e][l],n=n.replace("[reference]",c),n=n.replace("[label]",a)),n}function g(e){var t=o()(e,{allowedTags:o.a.defaults.allowedTags.concat(["img","sup"])});return t}function f(e){var t=new RegExp("<(.*?)>","g");return e.replace(t,"<sup>$1</sup>")}},b360:function(e,t,a){"use strict";a("1e02")},b44c:function(e,t,a){"use strict";a("88b5")},b83f:function(e,t,a){"use strict";a("ac4d")},b8af:function(e,t,a){},b9a0:function(e,t,a){},bb3b:function(e,t,a){},c014:function(e,t,a){"use strict";a("3763")},c1be:function(e,t,a){"use strict";a.d(t,"n",(function(){return I})),a.d(t,"b",(function(){return x})),a.d(t,"g",(function(){return k})),a.d(t,"d",(function(){return E})),a.d(t,"f",(function(){return P})),a.d(t,"m",(function(){return B})),a.d(t,"k",(function(){return J})),a.d(t,"l",(function(){return U})),a.d(t,"h",(function(){return K})),a.d(t,"i",(function(){return V})),a.d(t,"c",(function(){return X})),a.d(t,"a",(function(){return Y})),a.d(t,"e",(function(){return Q})),a.d(t,"j",(function(){return te}));var n=a("a34a"),r=a.n(n),o=a("bc3a"),i=a.n(o),s=a("c46f"),l=a("522a");function c(){var e=[{sourceDisplayName:"Mouse Genome Informatics",sourceDescription:"MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease.",monarchUsage:"We list genotype-phenotype associations and asserted disease-models. We use MGI identifiers as the main hook into mouse data. Additionally, we utilize the Mouse Phenotype Ontology (MP) in our cross-species ontology, to link all mouse phenotype annotations from various sources, which is developed by MGI.",vocabulary:"ECO, MA, MP, SO",summaryIRI:"MonarchArchive:#mgi",logoUrl:"/img/sources/source-mgi.png",reusableDataUrl:"http://reusabledata.org/mgi.html"},{sourceDisplayName:"Zebrafish Information Network",sourceDescription:"The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.",monarchUsage:"",vocabulary:"We integrate the curated genotype-phenotype data, including experimentally derived fish (such as via application of morpholinos), and links to the literature as evidence.",summaryIRI:"MonarchArchive:#zfin",logoUrl:"/img/sources/source-zfin.png",reusableDataUrl:"http://reusabledata.org/zfin.html"},{sourceDisplayName:"Zebrafish Information Network (ZFIN Slim)",sourceDescription:"The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.",monarchUsage:"Zfin model only containing Gene to phenotype associations using this file: https://zfin.org/downloads/phenoGeneCleanData_fish.txt",vocabulary:"",summaryIRI:"MonarchArchive:#zfinslim",logoUrl:"/img/sources/source-zfin.png",reusableDataUrl:"http://reusabledata.org/zfin.html"},{sourceDisplayName:"WormBase database of nematode biology",sourceDescription:"WormBase is an international consortium dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of C. elegans and related nematodes.",monarchUsage:"Wormbase curates variant (allele)-phenotype associations. The variants are both genetic (intrinsic) and induced through application of reagents such as RNAi (extrinsic). We list the variant-phenotype associations. Some data is pulled from WormBase directly, other data is routed via WormMine.",vocabulary:"WBbt, WBls, WBPhenotype",summaryIRI:"MonarchArchive:#wormbase",logoUrl:"/img/sources/source-wormbase.png",reusableDataUrl:"http://reusabledata.org/wormbase.html"},{sourceDisplayName:"FlyBase",sourceDescription:"FlyBase is the model organism database providing integrated genetic, genomic, phenomic, and biological data for Drosophila melanogaster.",monarchUsage:"We integrate the genotype-phenotype associations.",vocabulary:"FBbt, FBcv, FBdv",summaryIRI:"MonarchArchive:#flybase",logoUrl:"/img/sources/source-flybase.png",reusableDataUrl:"http://reusabledata.org/flybase.html"},{sourceDisplayName:"International Mouse Phenotyping Consortium",sourceDescription:"The International Mouse Phenotyping Consortium (IMPC) is generating a knockout mouse strain for every protein coding gene by using the embryonic stem cell resource generated by the International Knockout Mouse Consortium (IKMC). Systematic broad-based phenotyping is performed by each IMPC center using standardized procedures found within the International Mouse Phenotyping Resource of Standardised Screens (IMPReSS) resource. Gene-to-phenotype associations are made by a versioned statistical analysis.",monarchUsage:"We use the allele-phenotype associations recorded by the consortium. In addition, we map their allele+zygosity+background to MGI genotype identifiers. Where they do not map to MGI genotype identifiers, we create temporary identifiers for navigation purposes.",vocabulary:"MP",summaryIRI:"MonarchArchive:#impc",logoUrl:"/img/sources/source-impc.png",reusableDataUrl:"http://reusabledata.org/impc.html"},{sourceDisplayName:"Mouse Phenome Database",sourceDescription:"The Mouse Phenome Database is a collaborative standardized collection of measured data on laboratory mouse strains, and includes: baseline phenotype data sets, studies of drug, diet, disease and aging effect, protocols, projects, and publications, and SNP, variation and gene expression studies. MPD collects data for classical inbred strains, other fixed-genotype strains, derived lines and populations that are openly acquirable (strain panel examples). Strains can be from JAX-Mice or from any other vendor that's a recognized breeding source.",monarchUsage:"genotype (strain)-phenotype association",vocabulary:"MP",summaryIRI:"MonarchArchive:#mpd",logoUrl:"/img/sources/source-mpd.png",reusableDataUrl:"http://reusabledata.org/mpd.html"},{sourceDisplayName:"Online Mendelian Inheritance in Man",sourceDescription:"OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes with full-text, referenced overviews that contains information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.",monarchUsage:"We use curated disease-gene, disease-locus, and variant-disease associations, together with their annotated references. Most OMIM diseases are further curated by the HPO group. Most OMIM diseases are integrated into the Disease Ontology.",vocabulary:"OMIM",summaryIRI:"MonarchArchive:#omim",logoUrl:"/img/sources/source-omim.png",reusableDataUrl:"http://reusabledata.org/impc.html"},{sourceDisplayName:"Orphanet",sourceDescription:"Orphanet provides reference information on rare diseases and orphan drugs to help improve the diagnosis, care and treatment of patients with rare diseases.",monarchUsage:"We use the Orphanet disease-gene associations.",vocabulary:"ORPHA",summaryIRI:"MonarchArchive:#orphanet",logoUrl:"/img/sources/source-orphanet.png",reusableDataUrl:"http://reusabledata.org/orphanet-open.html"},{sourceDisplayName:"PANTHER",sourceDescription:"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) according to evolutionary family/subfamily, molecular function, biological process, and pathway. The PANTHER Classifications are the result of human curation as well as sophisticated bioinformatics algorithms",monarchUsage:"We currently utilize the 12 RefGenome species, as well as HUGO HCOP species, to seed the orthology calls. Species currently include: arabidopsis, budding yeast, chicken, chimp, dog, fission yeast, fruitfly, green lizard, horse, human, macaque, mouse, opossum, pig, platypus, rat, slime mold, worms, zebrafish. We use the orthology calls to populate the orthologs tabs for genes, as well as to infer disease-model associations via homology.",vocabulary:"RO",summaryIRI:"MonarchArchive:#panther",logoUrl:"/img/sources/source-panther.png",reusableDataUrl:"http://reusabledata.org/panther.html"},{sourceDisplayName:"Coriell Institute for Medical Research",sourceDescription:"The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures. These collections, supported by funds from the National Institutes of Health (NIH) and several foundations, are extensively utilized by research scientists around the world. NINDS and NIGMS cell line catalog. NIGMS samples represent a variety of disease states, chromosomal abnormalities, apparently healthy individuals and many distinct human populations. NINDS samples are drawn from subjects with cerebrovascular disease, epilepsy, motor neuron disease, Parkinsonism and Tourette Syndrome, as well as controls.",monarchUsage:"We link pertinent cell lines to any diseases for which they are asserted models.",vocabulary:"OMIM",logoUrl:"/img/sources/source-coriell.png",summaryIRI:"MonarchArchive:#coriell"},{sourceDisplayName:"Comparative Toxicogenomics Database",sourceDescription:"CTD promotes understanding about the effects of environmental chemicals on human health by integrating data from curated scientific literature to describe chemical interactions with genes and proteins, and associations between diseases and chemicals, and diseases and genes/proteins.",monarchUsage:"We integrate the asserted (curated) disease-gene associations, and their evidence.",vocabulary:"MESH, OMIM",summaryIRI:"MonarchArchive:#ctd",logoUrl:"/img/sources/source-ctd.png",reusableDataUrl:"http://reusabledata.org/ctd.html"},{sourceDisplayName:"Human Phenotype Ontology",sourceDescription:"A curated database of human hereditary syndromes from OMIM, Orphanet, and DECIPHER mapped to classes of the human phenotype ontology. Various meta-attributes such as frequency, references and negations are associated with each annotation. These are presently limited to rare mendelian diseases.",monarchUsage:"We use the HPO disease-phenotype annotations as the primary atomic description of a disease, and list them on the disease pages, together with their references. The Human Phenotype Ontology is integrated into our cross-species phenotype ontology.",vocabulary:"ECO, HPO",summaryIRI:"MonarchArchive:#hpoa",logoUrl:"/img/sources/source-hpo.png",reusableDataUrl:"http://reusabledata.org/hpo.html"},{sourceDisplayName:"Kyoto Encyclopedia of Genes and Genomes",sourceDescription:"KEGG is an integrated database resource consisting of the seventeen main databases including systems, genomic, chemical, and health information.",monarchUsage:"We list disease-gene associations, and gene-pathway associations. We utilize the KEGG Ortholog (KO) gene-pathway associations, and infer a specific-organisms' participation in that pathway based on the gene-KO links.",vocabulary:"MESH, OMIM",summaryIRI:"MonarchArchive:#kegg",logoUrl:"/img/sources/source-kegg.png",reusableDataUrl:"http://reusabledata.org/kegg-ftp.html"},{sourceDisplayName:"National Center for Biotechnology Information",sourceDescription:"Gene integrates information from a wide range of species, and includes nomenclature, Reference Sequences (RefSeqs), maps, pathways, variations, phenotypes, and links to genome-, phenotype-, and locus-specific resources worldwide. Taxon lists the taxinomic organization of organisms. Pub2Gene serves links between genes and pubmed identifiers where they are mentioned.",monarchUsage:"We use NCBIGene ids and symbols as the primary identifier and label for human genes in our system and NCBITaxon identifiers and scientific name for species-specific labeling. For any given gene, we also list the annotated pmids from Pub2Gene.",vocabulary:"",summaryIRI:"MonarchArchive:#ncbigene",logoUrl:"/img/sources/source-ncbi.png",reusableDataUrl:"https://reusabledata.org/ncbi-gene.html"},{sourceDisplayName:"BioGRID",sourceDescription:"BioGRID is an curated gene and protein interaction repository with for major model organism species.",monarchUsage:"Monarch indicates gene-gene/protein-protein interactions on gene pages. We also use many of the id mappings to resolve ids in our own site.",vocabulary:"MI",summaryIRI:"MonarchArchive:#biogrid",logoUrl:"/img/sources/source-biogrid.png",reusableDataUrl:"http://reusabledata.org/biogrid.html"},{sourceDisplayName:"GWAS Catalog",sourceDescription:"The NHGRI-EBI Catalog of published genome-wide association studies.",monarchUsage:"Monarch links the variants recorded here to the curated EFO-classes",vocabulary:"RO, EFO, ECO",summaryIRI:"MonarchArchive:#gwascatalog",logoUrl:"/img/sources/source-gwascatalog.png"},{sourceDisplayName:"AnimalQTLdb",sourceDescription:"The Animal Quantitative Trait Loci (QTL) database (Animal QTLdb) is designed to house publicly all available QTL and single-nucleotide polymorphism/gene association data on livestock animal species.",monarchUsage:"Monarch uses the QTL genetic maps and their computed genomic locations to create associations between the QTLs and their traits. The traits come in their internal Animal Trait ontology vocabulary, which they further map to [Vertebrate Trait](http://bioportal.bioontology.org/ontologies/VT), Product Trait, and Clinical Measurement Ontology vocabularies.",vocabulary:"RO, ECO",summaryIRI:"MonarchArchive:#animalqtldb",logoUrl:"/img/sources/source-animalqtldb.png"},{sourceDisplayName:"Ensembl database of automatically annotated genomic data",sourceDescription:"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation.",monarchUsage:"Monarch obtains equivalencies between Ensembl gene IDs and NCBI gene IDs",vocabulary:"",logoUrl:"/img/sources/source-ensembl.png",summaryIRI:"MonarchArchive:#ensembl"},{sourceDisplayName:"Gene Ontology Database",sourceDescription:"The GO defines concepts/classes used to describe gene function, and relationships between these concepts.",monarchUsage:"Monarch processes gene-process/function/subcellular/location associations.",vocabulary:"RO",summaryIRI:"MonarchArchive:#go",logoUrl:"/img/sources/source-geneontology.png",reusableDataUrl:"https://reusabledata.org/go-ann.html"},{sourceDisplayName:"HUGO Gene Nomenclature Committee",sourceDescription:"A curated online repository of HGNC-approved gene nomenclature, gene families and associated resources.",monarchUsage:"Monarch creates equivalences between HGNC identifiers and ENSEMBL and NCBIGene. We also add the links to cytogenic locations for the gene features.",vocabulary:"",summaryIRI:"MonarchArchive:#hgnc",logoUrl:"/img/sources/source-hgnc.png"},{sourceDisplayName:"Mutant Mouse Resource and Research Centers",sourceDescription:"A repository of mouse stocks and ES cell line collections serving the world-wide genetics and biomedical research community for the benefit of human health.",monarchUsage:"Centers A repository of mouse stocks and ES cell line collections serving the world-wide genetics and biomedical research community for the benefit of human health.\tMonarch processes the Mutant Mouse Resource and Research Center strain data, which includes: strains and their mutant alleles, phenotypes of the alleles, and descriptions of the research uses of the strains.",vocabulary:"MP",summaryIRI:"MonarchArchive:#mmrrc",logoUrl:"/img/sources/source-mmrrc.png"},{sourceDisplayName:"Reactome - a curated knowledgebase of biological pathways",sourceDescription:"A repository of mouse stocks and ES cell line collections serving the world-wide genetics and biomedical research community for the benefit of human health.",monarchUsage:"Monarch processes ensembl gene to pathway associations",vocabulary:"RO",summaryIRI:"MonarchArchive:#reactome",logoUrl:"/img/sources/source-reactome.png",reusableDataUrl:"http://reusabledata.org/reactome.html"},{sourceDisplayName:"Gene expression data in animals",sourceDescription:"Bgee is a database to retrieve and compare gene expression patterns in multiple animal species",monarchUsage:"Monarch stores the top 20 ranked tissues where a gene is over expressed based on differential expression measures across evidence lines",vocabulary:"Uberon",summaryIRI:"MonarchArchive:#bgee",logoUrl:"/img/sources/source-bgee.png",reusableDataUrl:"http://reusabledata.org/bgee.html"},{sourceDisplayName:"STRING",sourceDescription:"STRING is a database of known and predicted protein-protein interactions.",monarchUsage:"Monarch stores protein protein interactions with a confidence >.7",vocabulary:"RO",summaryIRI:"MonarchArchive:#string",logoUrl:"/img/sources/source-string.png",reusableDataUrl:"http://reusabledata.org/string.html"},{sourceDisplayName:"Rat Genome Database",sourceDescription:"The Rat Genome Database (RGD)is the premier site for genetic, genomic, phenotype, and disease data generated from rat research",monarchUsage:"Monarch stores gene to phenotype associations",vocabulary:"MP, RO, RGD References, RGD genes",summaryIRI:"MonarchArchive:#rgd",logoUrl:"/img/sources/source-rgd.png",reusableDataUrl:"http://reusabledata.org/rgd.html"},{sourceDisplayName:"Saccharomyces Genome Database",sourceDescription:"The Saccharomyces Genome Database (SGD) provides comprehensive integrated biological information for the budding yeast Saccharomyces cerevisiae along with search and analysis tools to explore these data, enabling the discovery of functional relationships between sequence and gene products in fungi and higher organisms.",monarchUsage:"Monarch stores gene to phenotype associations and gene function data",vocabulary:"APO, RO",summaryIRI:"MonarchArchive:#sgd",logoUrl:"/img/sources/source-sgd.png",reusableDataUrl:"http://reusabledata.org/sgd.html"},{sourceDisplayName:"Elements of Morphology",sourceDescription:"Elements of Morphology is a resource from NHGRI that has definitions of morphological abnormalities, together with image depictions.",monarchUsage:"We pull those relationships, as well as our local mapping of equivalences between EOM and HP terminologies.",vocabulary:"",summaryIRI:"MonarchArchive:#eom",logoUrl:"/img/sources/source-eom.png"},{sourceDisplayName:"Mouse Genome Informatics (MGI Slim)",sourceDescription:"See 'Mouse Genome Informatics'",monarchUsage:"Slim MGI model only containing Gene to phenotype associations. This uses mousemine: http://www.mousemine.org/mousemine/begin.do",vocabulary:"",summaryIRI:"MonarchArchive:#mgi_slim",logoUrl:"/img/sources/source-mgi.png",reusableDataUrl:"http://reusabledata.org/mgi.html"},{sourceDisplayName:"Monarch Initiative",sourceDescription:"This is the parser for data curated by the Monarch Initiative.",monarchUsage:"",vocabulary:"",summaryIRI:"MonarchArchive:#monarch",logoUrl:"/img/sources/source-monarch.png",reusableDataUrl:"http://reusabledata.org/monarch.html"},{sourceDisplayName:"Monochrom",sourceDescription:"Ingested data represents major structural pieces of chromosomes which are often universally referenced, using physical properties/observations that remain constant over different genome builds (such as banding patterns and arms). The idea is to create a scaffold upon which we can hang build-specific chromosomal coordinates, and reason across them.",monarchUsage:"We leverage the GENO ontology and modeling patterns to build an ontology of chromosomes for any species.We currently compute bands for human, mouse, zebrafish, and rat but will be expanding in the future as needed.",vocabulary:"",summaryIRI:"MonarchArchive:#monochrom",logoUrl:"/img/sources/source-monochrom.png",reusableDataUrl:"http://reusabledata.org/monochrom.html"},{sourceDisplayName:"MyChem",sourceDescription:"Chemical and drug annotation data, aggregated from many popular data resources.",monarchUsage:"",vocabulary:"",summaryIRI:"MonarchArchive:#mychem",logoUrl:"/img/sources/source-mychem.png"},{sourceDisplayName:"UCSC Bands",sourceDescription:"UCSC definitions of cytogenic bands",monarchUsage:"We create nested structures to enable overlap and containment queries. We use Monochrom.py to create the OWL-classes of the chromosomal parts. Here, we simply worry about the instance-level values for particular genomebuilds.",vocabulary:"",summaryIRI:"MonarchArchive:#ucscbands",logoUrl:"/img/sources/source-ucscbands.png"},{sourceDisplayName:"Online Mendelian Inheritance in Animals",sourceDescription:"Online Mendelian Inheritance in Animals (OMIA) is a catalogue/compendium of inherited disorders, other (single-locus) traits, and genes in 215 (non-model) animal species.",monarchUsage:"gene-disease association",vocabulary:"OMIA, OMIM",summaryIRI:"MonarchArchive:#omia",logoUrl:"/img/sources/source-omia.png",reusableDataUrl:"http://reusabledata.org/omia.html"},{sourceDisplayName:"ClinVar",sourceDescription:"ClinVar archives and aggregates information about relationships among variation and human health. ClinVar collects reports of variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data.",monarchUsage:"disease-gene association, variant-disease association, variant definitions",vocabulary:"UMLS",sourceFiles:[{fileUrl:"ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz",retrievedOn:""},{fileUrl:"ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/gene_condition_source_id",retrievedOn:""}],logoUrl:"/img/sources/source-clinvar.png",summaryIRI:"MonarchArchive:#clinvar",reusableDataUrl:"http://reusabledata.org/clinvar.html",ingestDate:"",rdfDownloadUrl:"https://archive.monarchinitiative.org/latest/rdf/clinvar.nt"},{sourceDisplayName:"Undiagnosed Diseases Program (UDP)",sourceDescription:"The National Institutes of Health (NIH) Undiagnosed Diseases Program (UDP) is part of the Undiagnosed Disease Network (UDN), an NIH Common Fund initiative that focuses on the most puzzling medical cases referred to the NIH Clinical Center in Bethesda, Maryland.",monarchUsage:"Monarch stores phenotypes for each case and variants of interest",vocabulary:"RO",logoUrl:"/img/sources/source-udp.png",summaryIRI:"MonarchArchive:#udp",ingestDate:"",rdfDownloadUrl:"https://archive.monarchinitiative.org/latest/rdf/udp.ttl"}],t=e.reduce((function(e,t){return e[t.summaryIRI]=t,e}),{});return t}var d={distribution:"dcat:distribution",source:"dc:source",retrievedOn:"http://purl.org/pav/retrievedOn",created:"http://purl.org/dc/terms/created",downloadUrl:"dcat:downloadURL"},u={MonarchArchive:"https://archive.monarchinitiative.org/",CoriellCollection:"https://catalog.coriell.org/1/",OBO:"http://purl.obolibrary.org/obo/",ZFIN:"http://zfin.org/"};function h(e){return{_summary_iri:e._summary_iri,_version_iri:e._version_iri,sourceDisplayName:e._version_iri,sourceDescription:"",monarchUsage:"",vocabulary:"",logoUrl:"",ingestDate:"",rdfDownloadUrl:"",sourceFiles:[]}}function p(e,t,a){var n=a.get_edges_by_subject(e),r=s["a"].chain(n).filter((function(e){return e._predicate_id===t})).pluck("_object_id").value();return r}function v(e,t,a){return s["a"].chain(p(e,t,a)).first().value()}function m(e,t){return v(e,d.distribution,t)}function g(e,t){var a=[];return e=s["a"].map(e,(function(e){if(a.push(e._summary_iri),Object.prototype.hasOwnProperty.call(e,"_summary_iri")){var n=t[e._summary_iri];return Object.assign({},e,n)}return e})),s["a"].each(t,(function(t){if(!a.includes(t.summaryIRI)){var n=h({_summary_iri:t.summaryIRI});Object.assign(n,t),e.push(n)}})),e}function f(e,t){for(var a=0;a<e.length;a++){var n=p(e[a]._version_iri,d.source,t);e[a].sourceFiles=s["a"].chain(n).map((function(e){var a=t.get_node(e),n="";return Object.prototype.hasOwnProperty.call(a._metadata,d.retrievedOn)&&(n=a._metadata[d.retrievedOn][0]),{fileUrl:e,retrievedOn:n}})).map((function(e){return s["a"].each(u,(function(t,a){e.fileUrl=e.fileUrl.replace(a+":",t)})),e})).value()}}function b(e,t){for(var a=0;a<e.length;a++)e[a].ingestDate=t.get_node(e[a]._version_iri)._metadata[d.created][0]}function _(e,t){for(var a=0;a<e.length;a++){var n=m(e[a]._version_iri,t),r=v(n,d.downloadUrl,t);void 0!==r&&(e[a].rdfDownloadUrl=r.replace("MonarchArchive:",u.MonarchArchive))}}var y=a("09b1");function w(e){return w="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e},w(e)}function C(e,t,a,n,r,o,i){try{var s=e[o](i),l=s.value}catch(c){return void a(c)}s.done?t(l):Promise.resolve(l).then(n,r)}function M(e){return function(){var t=this,a=arguments;return new Promise((function(n,r){var o=e.apply(t,a);function i(e){C(o,n,r,i,s,"next",e)}function s(e){C(o,n,r,i,s,"throw",e)}i(void 0)}))}}var S={production:"https://api.monarch-gc-balanced.ddns.net/api/",beta:"https://api-dev.monarchinitiative.org/api/"},D="production",O=new URLSearchParams(document.location.search.substring(1)).get("api"),I=O&&S[O]?O:D,x=S[I];function k(e,t){return A.apply(this,arguments)}function A(){return A=M(r.a.mark((function e(t,a){var n,o,s;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return n="".concat(x,"bioentity/").concat(a,"/").concat(t),"function"===a&&(n="".concat(x,"bioentity/").concat(t)),o={fetch_objects:!1,unselect_evidence:!0,exclude_automatic_assertions:!0,use_compact_associations:!1,get_association_counts:!0,rows:1},e.next=5,i.a.get(n,{params:o}).then((function(e){var t=e.data;return t.xrefs||(t.xrefs=[]),t.description||(t.description=""),t.type=a,t.uri=t.iri,t})).catch((function(e){throw new Error(e.response.data.error.message)}));case 5:return s=e.sent,console.log(s),e.abrupt("return",s);case 8:case"end":return e.stop()}}),e)}))),A.apply(this,arguments)}function E(e){return T.apply(this,arguments)}function T(){return T=M(r.a.mark((function e(t){var a;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return a="".concat(x,"bioentity/").concat(t),e.abrupt("return",new Promise((function(e,t){i.a.get(a).then((function(a){var n=a.data;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))})));case 2:case"end":return e.stop()}}),e)}))),T.apply(this,arguments)}function R(e,t){var a=!0;return"disease"===t?a="MONDO:0000001"!==e:"anatomy"===t?a="UBERON:0001062"!==e:"phenotype"===t?a="UPHENO:0001001"!==e:"function"===t?a="GO:0003674"!==e:"pathway"===t&&(a="GO:0008150"!==e),a}var N=["disease","phenotype","anatomy","function"];function P(e,t){return H.apply(this,arguments)}function H(){return H=M(r.a.mark((function e(t,a){var n,o,l,c,d,u,h,p,v,m,g,f,b;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return n="".concat(x,"graph/edges/from/").concat(t),o={},l=[],c=[],d=[],u=[],h={},p={"Exact Synonym":"http://www.geneontology.org/formats/oboInOwl#hasExactSynonym","Narrow Synonym":"http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym","Broad Synonym":"http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym","Related Synonym":"http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym"},v="http://www.geneontology.org/formats/oboInOwl#hasDbXref",m=new RegExp(/MONDO|:?MONARCH|PHENOTYPE$/),g=new URLSearchParams,N.includes(a)?(g.append("relationship_type","equivalentClass"),g.append("relationship_type","subClassOf"),g.append("relationship_type","BFO:0000050")):g.append("relationship_type","equivalentClass"),e.next=14,i.a.get(n,{params:g});case 14:return f=e.sent,b=f.data,b.nodes&&b.nodes.forEach((function(e){v in e.meta||(e.meta[v]=[]),o[e.id]=e})),u=o[t].meta[v].map((function(e){return e.startsWith("Orphanet")?e=e.replace("Orphanet","ORPHA"):/^OMIMPS:\d/.test(e)&&(e=e.replace("PS:","PS:PS")),e})),Object.keys(p).forEach((function(e){p[e]in o[t].meta?h[e]=o[t].meta[p[e]].map((function(e){return e.replace(",","")})):h[e]=[]})),m.test(t)||(u=u.concat([t])),b.edges&&(b.edges.forEach((function(e){if("subClassOf"===e.pred||"BFO:0000050"===e.pred){if(e.sub===t)R(t,a,e.obj)&&c.push(e.obj);else if(e.obj===t){if("UPHENO:0001001"===t&&"isDefinedBy"in e.meta&&!e.meta.isDefinedBy.includes("http://purl.obolibrary.org/obo/upheno/monarch.owl"))return;d.push(e.sub)}}else if("equivalentClass"===e.pred)if(e.sub===t){l.push(e.obj);var n=e.obj.replace("OBO:","").replace("_",":");u=u.concat([n],o[e.obj].meta[v]),Object.keys(h).forEach((function(t){p[t]in o[e.obj].meta&&(h[t]=h[t].concat(o[e.obj].meta[p[t]].map((function(e){return e.replace(",","")}))))}))}else{l.push(e.sub);var r=e.sub.replace("OBO:","").replace("_",":");u=u.concat([r],o[e.sub].meta[v]),Object.keys(h).forEach((function(t){p[t]in o[e.sub].meta&&(h[t]=h[t].concat(o[e.sub].meta[p[t]].map((function(e){return e.replace(",","")}))))}))}})),u=s["a"].uniq(u),Object.keys(h).forEach((function(e){h[e]=s["a"].uniq(h[e])}))),e.abrupt("return",{nodeMap:o,equivalentClasses:l,superclasses:c,subclasses:d,xrefs:u,synonyms:h});case 22:case"end":return e.stop()}}),e)}))),H.apply(this,arguments)}var L=["gene","variant","genotype","phenotype","disease","goterm","pathway","anatomy","substance","individual","case","publication","model","anatomical entity"];function G(e){var t=e.facet_counts.category;Object.keys(t).forEach((function(e){-1===L.indexOf(e)&&delete t[e]}))}function B(){return F.apply(this,arguments)}function F(){return F=M(r.a.mark((function e(){var t,a,n,o,d,u;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return t=new l["graph"],e.prev=1,a="".concat(x,"metadata/datasets"),n=new URLSearchParams,e.next=6,i.a.get(a,{params:n});case 6:o=e.sent,t.load_base_json(o.data),e.next=12;break;case 10:e.prev=10,e.t0=e["catch"](1);case 12:return d=s["a"].chain(t.all_edges()).filter((function(e){return"dc:isVersionOf"===e._predicate_id})).map((function(e){return{_version_iri:e._subject_id,_summary_iri:e._object_id}})).map(h).value(),b(d,t),_(d,t),f(d,t),u=c(),d=g(d,u),e.abrupt("return",d);case 19:case"end":return e.stop()}}),e,null,[[1,10]])}))),F.apply(this,arguments)}function J(e,t,a,n,r){return j.apply(this,arguments)}function j(){return j=M(r.a.mark((function e(t,a,n,o,s){var l,c,d,u;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return l="".concat(x,"search/entity/").concat(t),c=new URLSearchParams,c.append("start",a),c.append("rows",n),c.append("highlight_class","hilite"),c.append("boost_q","category:disease^5"),c.append("boost_q","category:phenotype^5"),c.append("boost_q","category:genotype^-10"),c.append("boost_q","category:variant^-35"),c.append("boost_q","category:publication^-10"),c.append("prefix","-OMIA"),c.append("min_match","67%"),d=o,d&&0!==d.length||(d=L),d.forEach((function(e){c.append("category",e)})),s&&s.length>0&&s.forEach((function(e){var t=e.startsWith("NCBITaxon")?e:Object(y["e"])(e);c.append("taxon",t)})),e.next=18,i.a.get(l,{params:c});case 18:return u=e.sent,G(u.data),e.abrupt("return",u.data);case 21:case"end":return e.stop()}}),e)}))),j.apply(this,arguments)}function U(e,t,a,n){return W.apply(this,arguments)}function W(){return W=M(r.a.mark((function e(t,a,n,o){var s,l,c;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return s="".concat(x,"search/entity/autocomplete/"),l="".concat(s).concat(t),c=new URLSearchParams,c.append("rows",10),c.append("start",0),c.append("highlight_class","hilite"),c.append("boost_q","category:disease^5"),c.append("boost_q","category:phenotype^5"),c.append("boost_q","category:genotype^-10"),c.append("boost_q","category:variant^-35"),c.append("boost_q","category:publication^-10"),c.append("prefix","-OMIA"),c.append("min_match","50%"),c.append("boost_fx","pow(edges,0.1)"),n&&n.length&&n.forEach((function(e){c.append("prefix",e)})),a&&"all"!==a?Array.isArray(a)||(a=[a]):a=L,a.forEach((function(e){c.append("category",e)})),1===a.length&&("gene"===a[0]&&c.append("boost_fx","pow(edges,0.334)"),"variant"!==a[0]&&"genotype"!==a[0]||c.append("minimal_tokenizer",!0)),o&&o.forEach((function(){c.append("fq",o)})),e.abrupt("return",new Promise((function(e,t){i.a.get(l,{params:c}).then((function(a){var n=a.data;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))})));case 20:case"end":return e.stop()}}),e)}))),W.apply(this,arguments)}function z(e){var t="".concat(e,"s");return"anatomy"===e?t="expression/anatomy":"ortholog-phenotype"===e?t="ortholog/phenotypes":"ortholog-disease"===e?t="ortholog/diseases":"causal-disease"===e||"correlated-disease"===e?t="diseases":"causal-gene"===e||"correlated-gene"===e?t="genes":"function"===e&&(t=e),t}function K(e,t,a,n,r){return q.apply(this,arguments)}function q(){return q=M(r.a.mark((function e(t,a,n,o,s){var l,c,d,u,h,p,v;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return l="".concat(x,"bioentity/"),c=z(n),d="".concat(t,"/").concat(a,"/").concat(c),u="".concat(l).concat(d),h=o&&o.length>0&&Object(y["d"])(n),h&&(s.start=0,s.rows=1e4),p=new URLSearchParams,Object.keys(s).forEach((function(e){p.append(e,s[e])})),p.append("unselect_evidence",!0),"function"===n&&(u="".concat(x,"association/type/gene_function"),p.append("subject",a)),n.startsWith("causal")?p.append("association_type","causal"):n.startsWith("correlated")&&p.append("association_type","non_causal"),Object(y["d"])(n)&&(p.append("facet",!0),Object(y["c"])(n)?p.append("facet_fields","subject_taxon"):p.append("facet_fields","object_taxon"),null!=o&&-1!==o&&(o.length>1?o.forEach((function(e){p.append("taxon",e)})):p.append("taxon",o[0]),p.append("direct_taxon",!0))),e.next=14,i.a.get(u,{params:p});case 14:return v=e.sent,h&&(v.data.associations=v.data.associations.filter((function(e){var t=e.subject.taxon,a=e.object.taxon,n=!1;return null!==t.id&&o.indexOf(t.id)>=0&&(n=!0),null!==a.id&&o.indexOf(a.id)>=0&&(n=!0),n})),v.data.numFound=v.data.associations.length),e.abrupt("return",v);case 17:case"end":return e.stop()}}),e)}))),q.apply(this,arguments)}function V(e){return $.apply(this,arguments)}function $(){return $=M(r.a.mark((function e(t){var a,n;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return a="".concat(x,"bioentity/").concat(t),n={fetch_objects:!0,rows:100},e.abrupt("return",new Promise((function(e,t){i.a.get(a,{params:n}).then((function(a){var n=a;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))})));case 3:case"end":return e.stop()}}),e)}))),$.apply(this,arguments)}function X(e,t,a){var n;if("search"===a){var r="".concat(x,"sim/search"),o=new URLSearchParams;if(e.forEach((function(e){return o.append("id",e.id)})),1===t.length&&(o.append("taxon",t[0].groupId),n=new Promise((function(e,t){i.a.get(r,{params:o}).then((function(a){var n=a;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))}))),t.length>1){var s=[];t.forEach((function(e){o.append("taxon",e.groupId),s.push(i.a.get(r+"?"+o.toString())),o.delete("taxon")})),n=new Promise((function(e,t){i.a.all(s).then(i.a.spread((function(){for(var a={data:{matches:[]}},n=arguments.length,r=new Array(n),o=0;o<n;o++)r[o]=arguments[o];r.forEach((function(e){a.data.matches.push(e.data.matches)})),a.data.matches=a.data.matches.flat(),"object"!==w(a)?t(a):e(a)}))).catch((function(e){t(e)}))}))}}else{var l="".concat(x,"sim/compare"),c=t.filter((function(e){return e.includes("HP:")})),d=c.length>0;e=e.map((function(e){return e.id}));var u={is_feature_set:d,reference_ids:e,query_ids:t};n=new Promise((function(e,t){i.a.post(l,u).then((function(a){var n=a;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))}))}return n}function Y(e){return Z.apply(this,arguments)}function Z(){return Z=M(r.a.mark((function e(t){var a,n,o,s=arguments;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return a=!(s.length>1&&void 0!==s[1])||s[1],n="".concat(x,"nlp/annotate/"),o=new URLSearchParams,o.append("content",t),o.append("longestOnly",a),e.abrupt("return",new Promise((function(e,t){i.a.post(n,o,{headers:{"content-type":"application/x-www-form-urlencoded"}}).then((function(a){var n=a;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))})));case 6:case"end":return e.stop()}}),e)}))),Z.apply(this,arguments)}function Q(e,t){return ee.apply(this,arguments)}function ee(){return ee=M(r.a.mark((function e(t,a){var n,o;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return n="".concat(x,"evidence/graph/").concat(t,"/table"),o={},"publication"===a&&(o.is_publication=!0),e.abrupt("return",new Promise((function(e,t){i.a.get(n,{params:o}).then((function(a){var n=a;"object"!==w(n)?t(n):e(n)})).catch((function(e){t(e)}))})));case 4:case"end":return e.stop()}}),e)}))),ee.apply(this,arguments)}function te(e){return ae.apply(this,arguments)}function ae(){return ae=M(r.a.mark((function e(t){var a,n,o;return r.a.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return a="".concat(x,"bioentity/disease/").concat(t,"/phenotypes"),n=new URLSearchParams,n.append("rows",0),n.append("facet",!0),n.append("unselect_evidence",!1),n.append("exclude_automatic_assertions",!1),n.append("fetch_objects",!1),n.append("use_compact_associations",!1),n.append("direct",!1),n.append("direct_taxon",!1),e.next=12,i.a.get(a,{params:n});case 12:if(o=e.sent,!o.data.facet_counts.closure_bin){e.next=15;break}return e.abrupt("return",o.data.facet_counts.closure_bin);case 15:return e.abrupt("return",{});case 16:case"end":return e.stop()}}),e)}))),ae.apply(this,arguments)}},c265:function(e,t,a){"use strict";a("0fdb")},c359:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_PHENOTYPE.8fd1edd1.png"},c433:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_PUBLICATION.160b5af0.png"},c457:function(e,t,a){e.exports=a.p+"img/osu.7096fff0.png"},caab:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_INTERACTIONS.7aad1b54.png"},cb74:function(e,t){e.exports={}},cce6:function(e,t,a){e.exports=a.p+"img/monarch-logo-white.c0b31d8d.svg"},cd8c:function(e,t,a){},ce57:function(e,t,a){},ce90:function(e,t,a){},d711:function(e,t,a){},e3e1:function(e,t,a){e.exports=a.p+"img/biolink_logo_white.80a28b97.png"},e7a4:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_GENE.86a0b6ff.png"},e911:function(e,t,a){"use strict";a("4b21")},eba9:function(e,t,a){},ed50:function(e,t,a){},ef71:function(e,t,a){"use strict";a("ce90")},f415:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_FUNCTION.14e9b5fe.png"},fbd3:function(e,t,a){e.exports=a.p+"img/qmul.6d87eef5.png"},fc62:function(e,t,a){e.exports=a.p+"img/monarch-ui-icon_HOMOLOG.ce2c8924.png"}});
//# sourceMappingURL=app.f2c93058.js.map