Glossary of terms and acronyms we often need to reference; this is a work in progress. For convenience, curation of the living document is not done here but rather [here](https://docs.google.com/spreadsheets/d/1B0f1rE5SAkPfOsE0M22QWTDx-z3_gkLvJn6AyVltco0/edit#gid=1364170751).

Abbreviation|Full name|Link
-----|-----|----
AACR|American Association for Cancer Research|
ACMG||
ACT|Accrual to Clinical Trials|
AMIA|The American Medical Informatics Association|
Animal QTLdb||
API||
BD2K|Big Data to Knowledge|
BgeeDB||http://bgee.org/
BioCADDIE|Biomedical and Healthcare Data Discovery Index Ecosystem|
BioGRID|Biological General Repository for Interaction Datasets.|https://thebiogrid.org/
caDSR|Cancer Data Standards Registry and Repository|
CCH|Center for Community Health|
CCR|Center for Clinical Research|
CDE|Common Data Elements|
CDM|Common Data Model|
CDSI|Center for Data Science and Informatics,|
CECD|Center for Education and Career Development,|
CEDAR|Center for Expanded Data Annotation and Retrieval|
ChEBI||
Chiro|ChEBl Integrated Role Ontology|https://github.com/obophenotype/chiro
CI|Collaborative Innovation|
CIELO|Collaborative Informatics Environment for Learning on healthcare Outcomes|
CIOs|Chief Information Officers|
CL|Cell Ontology|http://obofoundry.org/ontology/cl.html
ClinVar|ClinVar|https://www.ncbi.nlm.nih.gov/clinvar/
CNVs||
CQI|Continuous Quality Improvement|
CRediT|Contributor Roles Taxonomy|
CRO|Contributor Role Ontology|
CRU|Clinical Research Unit,|
CSCW|Computer Supported Collaborative Work|
CSER|Clinical Sequencing Exploratory Research|
CTD|Clinical Toxicology Database|http://ctdbase.org/
CTS|Clinical and Translational Science|
CTSA|Clinical and Translational Science Awards|
D&I|Diversity and Inclusion|
DREAM|Dialogue for Reverse Engineering Assessments and Methods|
DSD||
DTF|Domain Task Force|
EAC|External Advisory Committee|
EC|Entrepreneurship and Commercialization|
ECO|Evidence and Conclusions Ontology|http://obofoundry.org/ontology/eco.html
ECTO|Environmental Conditions, Treatments and Exposures Ontology|https://github.com/EnvironmentOntology/environmental-exposure-ontology)
EDM|Electronic Data Methods|
EdX||
EFO|Experimental Factor Ontology|
EGEMS|Generating Evidence & Methods to improve patient outcomes|
EHR|electronic health record|
eMERGE|Electronic Medical Records and Genomics Network|
EMRAM|Electronic Medical Record Adoption Model|
ePHI|electronic protected health information|
ESC|External Scientific Consultants|
ETL|Extract, Transform, Load|
ExAC|Exome Aggregation Consortium|http://exac.broadinstitute.org/
FAIR|Findable, Accessible, Interoperable, Reusable|
FAIR-TLC|Findable, Accessible, Interoperable, Reusable - Traceable, Licensed, Connected|
FDA|Food and Drug Administration|
FHIR|Fast Healthcare Interoperability Resource|
FISMA|Federal Information Security Management Act|
FlyBase|FlyBase|http://flybase.org
FORCE11|The Future of Research Communications and e-Scholarship|
ForSTI|Foresight for Science, Technology and Innovation|
FSM|Feinberg School of Medicine|
GA4GH|Global Alliance for Genomics and Health|https://www.ga4gh.org/
GARD|Genetics and Rare Diseases Information Center|
GeneNetwork|Gene Network|http://genenetwork.org
GENIE|Genomics, Evidence, Neoplasia, Information, Exchange|
GENO|Genotype Ontology|https://github.com/monarch-initiative/GENO-ontology/
GERA|Genetic Epidemiology Research on Adult Health and Aging|
GO|Gene Ontology|http://geneontology.org
GRID|Global Research Identifier Database|
GWAS Catalog|genome-wide association studies|https://www.ebi.ac.uk/gwas/
HCLS||https://www.w3.org/TR/2015/NOTE-hcls-dataset-20150514/
HGNC||
HGVS||
HIMSS|Healthcare Information and Management Systems Society|
HIPAA|Health Insurance Portability and Accountability Act|
HIPPO|(Does it stand for anything?)|hippo.monarchinitiative.org
HL7|Health Level Seven International|
HPO|Human Phenotype Ontology|http://human-phenotype-ontology.org/
I2b2|Informatics for Integrating Biology & the Bedside|
I2I|Idea to Implementation|
ICBO||
ICD11|International Classification of Diseases, Eleventh Revision|
IEEE|Institute of Electrical and Electronics Engineers|
IMPC||
IP|Intellectual Property|
IRB|Institutional Review Board|
IT|Information Technology|
JSON||
JSON-LD|JavaScript Object Notation for Linked Data|
KEGG|Kyoto Encyclopedia of Genes and Genomes|http://www.kegg.jp/
KL2|Transition to Independence|
LCH|Ann and Robert H. Lurie Childrenâ€™s Hospital of Chicago|
LoS|Letter of Support|
MAxO|Medical Action Ontology|https://github.com/monarch-initiative/MAxO
MESA||
MGI|Mouse Genome Informatics|http://www.informatics.jax.org/
MIM|Mendelian Inheritance in Man|
MME|Matchmaker Exchange|
MMRRC|Mutant Mouse Resource and Research Centers|
MonDO|Monarch Merged Disease Ontology|https://github.com/monarch-initiative/monarch-disease-ontology/
MP|Mammalian Phenotype Ontology|http://obofoundry.org/ontology/mp.html
MPD|Mouse phenome database|http://phenome.jax.org/
MyGene|MyGene|http://mygene.info
NASA/JPL|National Aeronautics and Space Administration/Jet Propulsion Laboratory|
NBO||
NCATS||
NCIt||
Neo4J||
NGS||
NHLBI|National Heart Lung and Blood Institute|
NIH|National Institutes of Health|
NITRO|Northwestern Integrated Translational Research Optimizer|
NLM|National Library of Medicine|
NLP|Natural Language Processing|
NMHC|Northwestern Memorial HealthCare|
NSIGHT|Newborn Sequencing in Genomic Medicine and Public Health|
NU|Northwestern University|
NUCATS|Northwestern University Clinical and Translational Sciences Institute|
OBO||
ODK|Ontology Development Kit|
OHDSI|Observational Health Data Sciences and Informatics|
OMIA|Online Mendelian Inheritance in Animals|http://omia.angis.org.au/home/
OMIM|Online Mendelian Inheritance in Man|http://omim.org
ONC|Office of the National Coordinator|
ORCiD|Open Researcher and Contributor ID|
Orphanet|Portal for rare diseases and orphan drugs|http://www.orpha.net
OSI|Open Source Initiative|
OWL|Web Ontology Language|
Panther|PantherDB|http://pantherdb.org
PATO|Phenotype And Trait Ontology|
PCORI|Patient-Centered Outcomes Research Institute|
PCORnet|National Patient-Centered Clinical Research Network|
PDs|Program Directors|
PhenX|Phenotypes and eXposures|
PHI|Protected Health Information|
PI|Principal Investigator|
POTATO|Phenotype Ontologies: Traversing All The Organisms|
PROMIS|Patient-Reported Outcomes Measurement Information System|
QA|Quality assurance|
QC|Quality control|
QTLs||
RBA|Results-Based Accountability|
RCS|Revision Control System|
RDF|Resource Description Framework |
REDCap|Research Electronic Data Capture|
RFA|Request for Applications|
RGD|Rat Genome Database|
RIC|Recruitment Innovation Center|
RO|Relation Ontology|http://obofoundry.org/ontology/ro.html
ROO|Research Outputs Ontology|
SCCS|Source Code Control System|
SciENcv|Science Experts Network Curriculum Vitae|
SciTS|Science of Team Science|
SDK|Software Development Kit|
SEPIO|Scientific Evidence and Provenance Information Ontology|https://github.com/monarch-initiative/SEPIO-ontology/
SGD|Saccharomyces Genome Database|
SHARPn|Strategic Health IT Advanced Research Projects|
SIO||
SMC-DNA|Somatic Mutation Calling-Deoxyribonucleic Acid|
SNOMED-CT|Systematized Nomenclature of Medicine -- Clinical Terms|
SNPs||
SNVs||
SO|Sequence Ontology|http://www.sequenceontology.org/
Solr||
SOLVE-RD||
SPARQL|Protocol and RDF Query Language|
STRING||
TIC|Trial Innovation Center|
TIN|Trials Innovation Network|
TNF|Tumor Necrosis Factor|
TOADS|Tools Ontologies, Algorithms, Data, and Standards|
TSB|Translational Science Base|
TSC|Team Science and Collaboration|
TSV||
Uberon|Uber-anatomy ontology|http://uberon.org
UCSF|University of California San Francisco|
UDN|Undiagnosed Diseases Network|
UMLS||
uPheno|Unified Phenotype Ontology|https://github.com/obophenotype/upheno/
UW|University of Washington|
VICC||https://cancervariants.org/
WormBase|WormBase|http://wormbase.org
XPO|Xenopus Phenotype Ontology|
ZFIN|Zebrafish Information Resource|http://zfin.org
